diff --git a/code/modules/farming/chaff.dm b/code/modules/farming/chaff.dm
--- a/code/modules/farming/chaff.dm
+++ b/code/modules/farming/chaff.dm
@@ -1,18 +1,18 @@
 /obj/item/natural/chaff
 	icon = 'icons/roguetown/items/produce.dmi'
 	var/foodextracted = null
-	name = "chaff"
+	name = "谷壳秆束"
 	icon_state = "chaff1"
-	desc = "A farmer's chaff." //english is not my native language, upon searching "chaff" i didn't even get what this is.
+	desc = "农人收下来的秸秆与谷壳。" //english is not my native language, upon searching "chaff" i didn't even get what this is.
 	var/canthresh = TRUE
 	//dropshrink = 0.75
 
 /obj/item/natural/chaff/attack_right(mob/user)
 	if(foodextracted && !user.get_active_held_item())
-		to_chat(user, span_warning("I start to shuck [src]..."))
+		to_chat(user, span_warning("我开始剥开[src]..."))
 		if(move_after(user,40, target = src)) //ROGTODO make this based on farming skill and speed
-			user.visible_message(span_notice("[user] shucks [src]."), \
-								span_notice("I shuck [src]."))
+			user.visible_message(span_notice("[user]剥开了[src]。"), \
+								span_notice("我剥开了[src]。"))
 			testing("1")
 			var/obj/item/G = new foodextracted(get_turf(src))
 			user.put_in_active_hand(G)
@@ -39,9 +39,9 @@
 							W.forked += R
 							stuff++
 					if(stuff)
-						to_chat(user, span_notice("I pick up the stalks with the pitchfork."))
+						to_chat(user, span_notice("我用草叉挑起了这些秸秆。"))
 					else
-						to_chat(user, span_warning("I'm carrying enough with the pitchfork."))
+						to_chat(user, span_warning("草叉上已经挑得够多了。"))
 					W.update_icon()
 					return
 
@@ -53,8 +53,8 @@
 				continue
 			if(prob(threshchance))
 				C.thresh()
-		user.visible_message(span_notice("[user] threshes the stalks!"), \
-							span_notice("I thresh the stalks."))
+		user.visible_message(span_notice("[user]正在给秸秆脱粒！"), \
+							span_notice("我给这些秸秆脱粒。"))
 		user.changeNext_move(CLICK_CD_MELEE)
 		playsound(loc,"plantcross", 100, FALSE)
 		playsound(loc,"smashlimb", 50, FALSE)
@@ -64,15 +64,15 @@
 
 /obj/item/natural/chaff/wheat
 	icon_state = "wheatchaff"
-	name = "wheat stalks"
+	name = "小麦秸秆"
 	foodextracted = /obj/item/reagent_containers/food/snacks/grown/wheat
 
 /obj/item/natural/chaff/oat
-	name = "oat stalks"
+	name = "燕麦秸秆"
 	icon_state = "oatchaff"
 	foodextracted = /obj/item/reagent_containers/food/snacks/grown/oat
 
 /obj/item/natural/chaff/rice
-	name = "rice stalks"
+	name = "稻秆"
 	icon_state = "ricechaff"
 	foodextracted = /obj/item/reagent_containers/food/snacks/grown/rice
