From b2189470fd01160feb9dc80310405af8b6e2af58 Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Sat, 30 May 2026 13:00:00 +0800
Subject: [PATCH] 翻译 code/modules/mob/dead/new_player/sprite_accessory/_sprite_accessory.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

diff --git a/code/modules/mob/dead/new_player/sprite_accessory/_sprite_accessory.dm b/code/modules/mob/dead/new_player/sprite_accessory/_sprite_accessory.dm
index 75aa98aa50b3..5eea18b31d92 100644
--- a/code/modules/mob/dead/new_player/sprite_accessory/_sprite_accessory.dm
+++ b/code/modules/mob/dead/new_player/sprite_accessory/_sprite_accessory.dm
@@ -19,7 +19,7 @@
 	/// Amount of color keys this accessory uses.
 	var/color_keys = 1
 	/// Color key name to describe a single customizable color key.
-	var/color_key_name = "Accessory"
+	var/color_key_name = "配饰"
 	/// List of names for color keys, required if you use more than 1. This is to present the user with how every color will affect the accessory.
 	var/list/color_key_names
 	/// List of defines for determining which color to use for which key as a default.
@@ -311,7 +311,7 @@
 
 /// None state which just means no appearance. Exists for easier manipulation so we dont have to write cases for customizng into a null type.
 /datum/sprite_accessory/none
-	name = "None"
+	name = "无"
 	color_keys = 0
 	icon = null
 	icon_state = null
-- 
Translation Patch Toolkit
