From dd0aa767ca177d7ee767a58ecb894d2a12134363 Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Sat, 30 May 2026 12:00:00 +0800
Subject: [PATCH] 翻译 code/modules/mob/dead/new_player/sprite_accessory/antennas.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

diff --git a/code/modules/mob/dead/new_player/sprite_accessory/antennas.dm b/code/modules/mob/dead/new_player/sprite_accessory/antennas.dm
index a9a338b78ae1..c03982963ac8 100644
--- a/code/modules/mob/dead/new_player/sprite_accessory/antennas.dm
+++ b/code/modules/mob/dead/new_player/sprite_accessory/antennas.dm
@@ -1,6 +1,6 @@
 /datum/sprite_accessory/antenna
 	abstract_type = /datum/sprite_accessory/antenna
-	color_key_name = "Antennae"
+	color_key_name = "触角"
 	relevant_layers = list(BODY_ADJ_LAYER)
 
 /datum/sprite_accessory/antenna/is_visible(obj/item/organ/organ, obj/item/bodypart/bodypart, mob/living/carbon/owner)
@@ -16,69 +16,69 @@
 	default_colors = list("#FFFFFF")
 
 /datum/sprite_accessory/antenna/moth/plain
-	name = "Plain"
+	name = "普通"
 	icon_state = "plain"
 
 /datum/sprite_accessory/antenna/moth/reddish
-	name = "Reddish"
+	name = "泛红"
 	icon_state = "reddish"
 
 /datum/sprite_accessory/antenna/moth/royal
-	name = "Royal"
+	name = "皇家"
 	icon_state = "royal"
 
 /datum/sprite_accessory/antenna/moth/gothic
-	name = "Gothic"
+	name = "哥特"
 	icon_state = "gothic"
 
 /datum/sprite_accessory/antenna/moth/whitefly
-	name = "White Fly"
+	name = "白粉虱"
 	icon_state = "whitefly"
 
 /datum/sprite_accessory/antenna/moth/lovers
-	name = "Lovers"
+	name = "恋人"
 	icon_state = "lovers"
 
 /datum/sprite_accessory/antenna/moth/burnt_off
-	name = "Burnt Off"
+	name = "烧焦残翼"
 	icon_state = "burnt_off"
 
 /datum/sprite_accessory/antenna/moth/firewatch
-	name = "Firewatch"
+	name = "火焰守望"
 	icon_state = "firewatch"
 
 /datum/sprite_accessory/antenna/moth/deathhead
-	name = "Deathshead"
+	name = "骷髅纹"
 	icon_state = "deathhead"
 
 /datum/sprite_accessory/antenna/moth/poison
-	name = "Poison"
+	name = "毒纹"
 	icon_state = "poison"
 
 /datum/sprite_accessory/antenna/moth/ragged
-	name = "Ragged"
+	name = "破损"
 	icon_state = "ragged"
 
 /datum/sprite_accessory/antenna/moth/moonfly
-	name = "Moon Fly"
+	name = "月蛾"
 	icon_state = "moonfly"
 
 /datum/sprite_accessory/antenna/moth/oakworm
-	name = "Oak Worm"
+	name = "栎蚕蛾"
 	icon_state = "oakworm"
 
 /datum/sprite_accessory/antenna/moth/jungle
-	name = "Jungle"
+	name = "丛林"
 	icon_state = "jungle"
 
 /datum/sprite_accessory/antenna/moth/witchwing
-	name = "Witch Wing"
+	name = "女巫之翼"
 	icon_state = "witchwing"
 
 /datum/sprite_accessory/antenna/moth/regal
-	name = "Regal"
+	name = "华贵"
 	icon_state = "regal"
 
 /datum/sprite_accessory/antenna/moth/mothra
-	name = "Mothra"
+	name = "魔斯拉"
 	icon_state = "mothra"
-- 
Translation Patch Toolkit
