From 0e7f0fbd9307517b18850c00e757d1e88e681f8d Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Sat, 30 May 2026 12:00:00 +0800
Subject: [PATCH] 翻译 code/modules/mob/dead/new_player/sprite_accessory/eyes.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

diff --git a/code/modules/mob/dead/new_player/sprite_accessory/eyes.dm b/code/modules/mob/dead/new_player/sprite_accessory/eyes.dm
index cae996161e02..d4f44f0008d1 100644
--- a/code/modules/mob/dead/new_player/sprite_accessory/eyes.dm
+++ b/code/modules/mob/dead/new_player/sprite_accessory/eyes.dm
@@ -1,7 +1,7 @@
 /datum/sprite_accessory/eyes
 	abstract_type = /datum/sprite_accessory/eyes
 	color_keys = 2
-	color_key_names = list("First Eye", "Second Eye")
+	color_key_names = list("第一只眼", "第二只眼")
 	icon = 'icons/mob/sprite_accessory/eyes/eyes.dmi'
 
 /datum/sprite_accessory/eyes/is_visible(obj/item/organ/organ, obj/item/bodypart/bodypart, mob/living/carbon/owner)
@@ -11,17 +11,17 @@
 	generic_gender_feature_adjust(appearance_list, organ, bodypart, owner, OFFSET_FACE, OFFSET_FACE_F)
 
 /datum/sprite_accessory/eyes/moth
-	name = "Fluvian Eyes"
+	name = "Fluvian 眼睛"
 	icon_state = "moth"
 
 /datum/sprite_accessory/eyes/humanoid
-	name = "Humanoid Eyes"
+	name = "类人眼睛"
 	icon_state = "human"
 
 /datum/sprite_accessory/eyes/humanoid_glow
-	name = "Humanoid Glowing"
+	name = "类人发光眼"
 	icon_state = "human_glow"
 
 /datum/sprite_accessory/eyes/humanoid_cyber
-	name = "Humanoid Cyber"
+	name = "类人义眼"
 	icon_state = "human_cyber"
-- 
Translation Patch Toolkit
