From 9750555acaa2d916dec8204ccfaa3bcbdae1a5f6 Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Sat, 30 May 2026 12:00:00 +0800
Subject: [PATCH] 翻译 code/modules/mob/dead/new_player/sprite_accessory/misc.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

diff --git a/code/modules/mob/dead/new_player/sprite_accessory/misc.dm b/code/modules/mob/dead/new_player/sprite_accessory/misc.dm
index 9854dd6918d7..483d4a534c17 100644
--- a/code/modules/mob/dead/new_player/sprite_accessory/misc.dm
+++ b/code/modules/mob/dead/new_player/sprite_accessory/misc.dm
@@ -11,7 +11,7 @@
 	generic_gender_feature_adjust(appearance_list, organ, bodypart, owner, OFFSET_FACE, OFFSET_FACE_F)
 
 /datum/sprite_accessory/face_detail/brows
-	name = "Thick Eyebrows"
+	name = "浓眉"
 	icon_state = "brows"
 	layer = BODY_LAYER
 	default_colors =  null
@@ -19,96 +19,96 @@
 	color_disabled = FALSE
 
 /datum/sprite_accessory/face_detail/brows/dark
-	name = "Dark Eyebrows"
+	name = "深色眉毛"
 	icon_state = "darkbrows"
 	layer = BODY_LAYER
 
 /datum/sprite_accessory/face_detail/scar
-	name = "Scar"
+	name = "伤疤"
 	icon_state = "scar"
 	layer = BODY_LAYER
 
 /datum/sprite_accessory/face_detail/scart
-	name = "Scar2"
+	name = "伤疤 2"
 	layer = BODY_LAYER
 	icon_state = "scar2"
 
 /datum/sprite_accessory/face_detail/slashedeye_r
-	name = "Slashed Eye (r)"
+	name = "刀伤眼（右）"
 	icon_state = "slashedeye_r"
 	layer = BODY_LAYER
 
 /datum/sprite_accessory/face_detail/slashedeye_r
-	name = "Slashed Eye (r)"
+	name = "刀伤眼（右）"
 	icon_state = "slashedeye_r"
 	layer = BODY_LAYER
 
 /datum/sprite_accessory/face_detail/slashedeye_l
-	name = "Slashed Eye (l)"
+	name = "刀伤眼（左）"
 	icon_state = "slashedeye_l"
 	layer = BODY_LAYER
 
 /datum/sprite_accessory/face_detail/mangled
-	name = "Mangled Jaw"
+	name = "破损下颌"
 	icon_state = "mangled"
 	layer = BODY_LAYER
 
 /datum/sprite_accessory/face_detail/tattoo_lips
-	name = "Tattoo (Lips)"
+	name = "纹身（嘴唇）"
 	icon_state = "tattoo_lips"
 	layer = BODY_LAYER
 	color_disabled = FALSE
 
 /datum/sprite_accessory/face_detail/tattoo_eye_r
-	name = "Tattoo (r Eye)"
+	name = "纹身（右眼）"
 	icon_state = "tattoo_eye_r"
 	layer = BODY_LAYER
 	color_disabled = FALSE
 
 /datum/sprite_accessory/face_detail/tattoo_eye_l
-	name = "Tattoo (l Eye)"
+	name = "纹身（左眼）"
 	icon_state = "tattoo_eye_l"
 	layer = BODY_LAYER
 	color_disabled = FALSE
 
 /datum/sprite_accessory/face_detail/tattoo_eye_both
-	name = "Tattoo (Both Eyes)"
+	name = "纹身（双眼）"
 	icon_state = "tattoo_eye_both"
 	layer = BODY_LAYER
 	color_disabled = FALSE
 
 /datum/sprite_accessory/face_detail/burnface_r
-	name = "Burns (r)"
+	name = "烧伤（右）"
 	icon_state = "burnface_r"
 	layer = BODY_LAYER
 
 /datum/sprite_accessory/face_detail/burnface_l
-	name = "Burns (l)"
+	name = "烧伤（左）"
 	icon_state = "burnface_l"
 	layer = BODY_LAYER
 
 /datum/sprite_accessory/face_detail/burneye_r
-	name = "Burned Eye (r)"
+	name = "烧伤眼（右）"
 	icon_state = "burneye_r"
 	layer = BODY_LAYER
 
 /datum/sprite_accessory/face_detail/burneye_l
-	name = "Burned Eye (l)"
+	name = "烧伤眼（左）"
 	icon_state = "burneye_l"
 	layer = BODY_LAYER
 
 /datum/sprite_accessory/face_detail/deadeye_r
-	name = "Dead Eye (r)"
+	name = "死眼（右）"
 	icon_state = "deadeye_r"
 	layer = BODY_LAYER
 
 /datum/sprite_accessory/face_detail/deadeye_l
-	name = "Dead Eye (l)"
+	name = "死眼（左）"
 	icon_state = "deadeye_l"
 	layer = BODY_LAYER
 
 /datum/sprite_accessory/face_detail/scarhead
-	name = "Scarred Head"
+	name = "疤痕头部"
 	icon_state = "scarhead"
 	layer = BODY_LAYER
 
@@ -124,50 +124,50 @@
 	generic_gender_feature_adjust(appearance_list, organ, bodypart, owner, OFFSET_FACE, OFFSET_FACE_F)
 
 /datum/sprite_accessory/accessory/earrings
-	name = "Earrings (G)"
+	name = "耳环（G）"
 	icon_state = "earrings"
 	layer = BODY_FRONT_LAYER
 
 /datum/sprite_accessory/accessory/earrings/sil
-	name = "Earrings (Colorable)"
+	name = "耳环（可染色）"
 	icon_state = "earrings_sil"
 	layer = BODY_FRONT_LAYER
 	color_disabled = FALSE
 
 /datum/sprite_accessory/accessory/earrings/em
-	name = "Earrings (E)"
+	name = "耳环（E）"
 	icon_state = "earrings_em"
 	layer = BODY_FRONT_LAYER
 
 /datum/sprite_accessory/accessory/eyepierce
-	name = "Pierced Brow (L)"
+	name = "眉钉（左）"
 	icon_state = "eyepierce"
 	layer = BODY_FRONT_LAYER
 	color_disabled = FALSE
 
 /datum/sprite_accessory/accessory/eyepierce/alt
-	name = "Pierced Brow (R)"
+	name = "眉钉（右）"
 	icon_state = "eyepiercealt"
 	layer = BODY_LAYER
 	color_disabled = FALSE
 
 /datum/sprite_accessory/accessory/choker
-	name = "Neckband"
+	name = "颈环"
 	icon_state = "choker"
 	layer = BODY_LAYER
 
 /datum/sprite_accessory/accessory/chokere
-	name = "Neckband (E)"
+	name = "颈环（E）"
 	icon_state = "chokere"
 	layer = BODY_LAYER
 
 /datum/sprite_accessory/accessory/harlequin
-	name = "Harlequin"
+	name = "丑角"
 	icon_state = "harlequin"
 	layer = BODY_LAYER
 
 /datum/sprite_accessory/accessory/warpaint
-	name = "Warpaint"
+	name = "战纹"
 	icon_state = "warpaint"
 	layer = BODY_FRONT_LAYER
 	color_disabled = FALSE
-- 
Translation Patch Toolkit
