diff --git a/code/modules/jobs/job_types/roguetown/youngfolk/vagabond/unraveled.dm b/code/modules/jobs/job_types/roguetown/youngfolk/vagabond/unraveled.dm
--- a/code/modules/jobs/job_types/roguetown/youngfolk/vagabond/unraveled.dm
+++ b/code/modules/jobs/job_types/roguetown/youngfolk/vagabond/unraveled.dm
@@ -1,7 +1,7 @@
 /datum/advclass/vagabond_unraveled
-	name = "The Unraveled"
-	examine_name = "Beggar"
-	tutorial = "Once you sought to understand the mind’s decay — now you live within it, a wandering physician bound to his own affliction."
+	name = "失序者"
+	examine_name = "乞丐"
+	tutorial = "你曾试图理解心智如何崩坏，如今却亲自活在那崩坏之中，成了一个被自身病症缠住的流浪医者。"
 	allowed_sexes = list(MALE, FEMALE)
 	allowed_races = RACES_NO_CONSTRUCT
 	outfit = /datum/outfit/job/roguetown/vagabond/unraveled
@@ -16,7 +16,7 @@
 		/datum/skill/misc/medicine = SKILL_LEVEL_EXPERT,
 		/datum/skill/misc/climbing = SKILL_LEVEL_EXPERT
 	)
-	extra_context = "Constantly hallucinates."
+	extra_context = "会持续产生幻觉。"
 
 /datum/outfit/job/roguetown/vagabond/unraveled/pre_equip(mob/living/carbon/human/human)
 	..()
