diff --git a/code/modules/roguetown/roguemachine/steward/steward.dm b/code/modules/roguetown/roguemachine/steward/steward.dm
--- a/code/modules/roguetown/roguemachine/steward/steward.dm
+++ b/code/modules/roguetown/roguemachine/steward/steward.dm
@@ -122,7 +122,7 @@
 		SStreasury.log_to_steward("-[amt] imported [D.name]")
 		record_round_statistic(STATS_STOCKPILE_IMPORTS_VALUE, amt)
 		if(amt >= 100) //Only announce big spending.
-			scom_announce("Rotwood Vale imports [D.name] for [amt] mammon.", )
+			scom_announce("谷地以 [amt] 枚 Mammon 的价格进口了 [D.name]。", )
 		D.raise_demand()
 		addtimer(CALLBACK(src, PROC_REF(do_import), D.type), 10 SECONDS)
 	if(href_list["export"])
