diff --git a/code/modules/jobs/job_types/roguetown/other/goblin.dm b/code/modules/jobs/job_types/roguetown/other/goblin.dm
--- a/code/modules/jobs/job_types/roguetown/other/goblin.dm
+++ b/code/modules/jobs/job_types/roguetown/other/goblin.dm
@@ -1,5 +1,6 @@
 /datum/job/roguetown/goblin
 	title = "Goblin"
+	display_title = "地精"
 	flag = GOBLIN
 	department_flag = SLOP
 	faction = "Station"
@@ -66,8 +67,8 @@
 			QDEL_NULL(H.charflaw)
 		H.update_body()
 		H.faction = list("orcs")
-		H.name = "goblin"
-		H.real_name = "goblin"
+		H.name = "地精"
+		H.real_name = "地精"
 		ADD_TRAIT(H, TRAIT_NOMOOD, TRAIT_GENERIC)
 		ADD_TRAIT(H, TRAIT_NOHUNGER, TRAIT_GENERIC)
 		ADD_TRAIT(H, TRAIT_CRITICAL_WEAKNESS, TRAIT_GENERIC)
