diff --git a/code/modules/jobs/job_types/roguetown/other/greater_skeleton.dm b/code/modules/jobs/job_types/roguetown/other/greater_skeleton.dm
--- a/code/modules/jobs/job_types/roguetown/other/greater_skeleton.dm
+++ b/code/modules/jobs/job_types/roguetown/other/greater_skeleton.dm
@@ -1,5 +1,6 @@
 /datum/job/roguetown/greater_skeleton
 	title = "Greater Skeleton"
+	display_title = "强骸"
 	flag = SKELETON
 	department_flag = SLOP
 	faction = "Station"
@@ -11,7 +12,7 @@
 
 	advclass_cat_rolls = list(CTAG_NSKELETON = 20)
 
-	tutorial = "You are bygone. A wandering has-been. But maybe your luck has not run out, yet.."
+	tutorial = "你早已属于过去，是一具在世间游荡的残余亡骸。不过，也许你的运气还没有彻底耗尽……"
 
 	outfit = /datum/outfit/job/roguetown/greater_skeleton/necro
 	show_in_credits = FALSE
@@ -60,8 +61,8 @@
 	shoes = /obj/item/clothing/shoes/roguetown/boots
 
 /datum/advclass/greater_skeleton/necro/shambler
-	name = "Decrepit Shambler"
-	tutorial = "One of many to come. One of many to go."
+	name = "腐朽蹒骸"
+	tutorial = "来者无数，去者亦无数。你不过是其中之一。"
 	outfit = /datum/outfit/job/roguetown/greater_skeleton/necro/shambler
 
 	category_tags = list(CTAG_NSKELETON)
