diff --git a/code/modules/jobs/job_types/roguetown/other/vampire_spawn.dm b/code/modules/jobs/job_types/roguetown/other/vampire_spawn.dm
--- a/code/modules/jobs/job_types/roguetown/other/vampire_spawn.dm
+++ b/code/modules/jobs/job_types/roguetown/other/vampire_spawn.dm
@@ -1,6 +1,7 @@
 #define CTAG_VAMPIRE_SPAWN "ctag_vspawn"
 /datum/job/roguetown/vampire_spawn
 	title = "Vampire Spawn"
+	display_title = "吸血鬼子嗣"
 	flag = VAMPIRE_SERVANT
 	department_flag = SLOP
 	faction = "Station"
@@ -23,7 +24,7 @@
 	return ..()
 
 /datum/advclass/vampire_spawn
-	name = "Vampire Spawn"
+	name = "吸血鬼子嗣"
 	outfit = /datum/outfit/job/roguetown/vampire_spawn
 
 	category_tags = list(CTAG_VAMPIRE_SPAWN)
@@ -64,47 +65,47 @@
 
 	if(H.mind)
 		var/weapons = list(
-			"Longsword + Crossbow",
-			"Billhook + Recurve Bow",
-			"Grand Mace + Longbow", 
-			"Sabre + Recurve Bow",
-			"Claymore",
-			"Great Mace",
-			"Battle Axe",
-			"Greataxe",
+			"长剑与十字弩",
+			"钩镰枪与反曲弓",
+			"巨型权杖与长弓", 
+			"军刀与反曲弓",
+			"双手大剑",
+			"重型权杖",
+			"战斧",
+			"双刃巨斧",
 			"Estoc",
 			"Lucerne",
 			"Partizan",
 		)
-		var/weapon_choice = input(H, "Choose your weapon.", "TAKE UP ARMS") as anything in weapons
+		var/weapon_choice = input(H, "选择你的武器。", "整备武装") as anything in weapons
 		switch(weapon_choice)
-			if("Longsword + Crossbow")
+			if("长剑与十字弩")
 				beltl = /obj/item/rogueweapon/scabbard/sword
 				r_hand = /obj/item/rogueweapon/sword/long
 				beltr = /obj/item/quiver/bolts
 				backl = /obj/item/gun/ballistic/revolver/grenadelauncher/crossbow
-			if("Billhook + Recurve Bow")
+			if("钩镰枪与反曲弓")
 				r_hand = /obj/item/rogueweapon/spear/billhook
 				backl = /obj/item/rogueweapon/scabbard/gwstrap
 				beltr = /obj/item/quiver/arrows
 				beltl = /obj/item/gun/ballistic/revolver/grenadelauncher/bow/recurve
-			if("Grand Mace + Longbow")
+			if("巨型权杖与长弓")
 				backl = /obj/item/gun/ballistic/revolver/grenadelauncher/bow/longbow
 				beltr = /obj/item/quiver/arrows
 				beltl = /obj/item/rogueweapon/mace/goden/steel
-			if("Sabre + Recurve Bow")
+			if("军刀与反曲弓")
 				l_hand = /obj/item/rogueweapon/scabbard/sword
 				r_hand = /obj/item/rogueweapon/sword/sabre
 				beltr = /obj/item/quiver/arrows
 				beltl = /obj/item/gun/ballistic/revolver/grenadelauncher/bow/recurve
-			if("Claymore")
+			if("双手大剑")
 				r_hand = /obj/item/rogueweapon/greatsword/zwei
 				backl = /obj/item/rogueweapon/scabbard/gwstrap
-			if("Great Mace")
+			if("重型权杖")
 				r_hand = /obj/item/rogueweapon/mace/goden/steel
-			if("Battle Axe")
+			if("战斧")
 				r_hand = /obj/item/rogueweapon/stoneaxe/battle
-			if("Greataxe")
+			if("双刃巨斧")
 				r_hand = /obj/item/rogueweapon/greataxe/steel
 				backl = /obj/item/rogueweapon/scabbard/gwstrap
 			if("Estoc")
@@ -122,29 +123,29 @@
 
 	if(H.mind)
 		var/helmets = list(
-			"Pigface Bascinet" 	= /obj/item/clothing/head/roguetown/helmet/bascinet/pigface,
-			"Guard Helmet"		= /obj/item/clothing/head/roguetown/helmet/heavy/guard,
-			"Barred Helmet"		= /obj/item/clothing/head/roguetown/helmet/heavy/sheriff,
-			"Bucket Helmet"		= /obj/item/clothing/head/roguetown/helmet/heavy/bucket,
-			"Knight Helmet"		= /obj/item/clothing/head/roguetown/helmet/heavy/knight,
-			"Visored Sallet"	= /obj/item/clothing/head/roguetown/helmet/sallet/visored,
+			"猪面巴西内盔" 	= /obj/item/clothing/head/roguetown/helmet/bascinet/pigface,
+			"卫兵头盔"		= /obj/item/clothing/head/roguetown/helmet/heavy/guard,
+			"栅面头盔"		= /obj/item/clothing/head/roguetown/helmet/heavy/sheriff,
+			"桶盔"		= /obj/item/clothing/head/roguetown/helmet/heavy/bucket,
+			"骑士头盔"		= /obj/item/clothing/head/roguetown/helmet/heavy/knight,
+			"带面甲萨莱特盔"	= /obj/item/clothing/head/roguetown/helmet/sallet/visored,
 			"Armet"				= /obj/item/clothing/head/roguetown/helmet/heavy/knight/armet,
-			"Hounskull Bascinet" = /obj/item/clothing/head/roguetown/helmet/bascinet/pigface/hounskull,
-			"Etruscan Bascinet" = /obj/item/clothing/head/roguetown/helmet/bascinet/etruscan,
-			"Slitted Kettle"	= /obj/item/clothing/head/roguetown/helmet/heavy/knight/skettle,
-			"None"
+			"犬首巴西内盔" = /obj/item/clothing/head/roguetown/helmet/bascinet/pigface/hounskull,
+			"Etrusca 巴西内盔" = /obj/item/clothing/head/roguetown/helmet/bascinet/etruscan,
+			"裂缝锅盔"	= /obj/item/clothing/head/roguetown/helmet/heavy/knight/skettle,
+			"无"
 		)
-		var/helmchoice = input(H, "Choose your Helm.", "TAKE UP HELMS") as anything in helmets
-		if(helmchoice != "None")
+		var/helmchoice = input(H, "选择你的头盔。", "整备盔甲") as anything in helmets
+		if(helmchoice != "无")
 			head = helmets[helmchoice]
 
 		var/armors = list(
-			"Brigandine"		= /obj/item/clothing/suit/roguetown/armor/brigandine/retinue,
-			"Coat of Plates"	= /obj/item/clothing/suit/roguetown/armor/brigandine/coatplates,
-			"Steel Cuirass"		= /obj/item/clothing/suit/roguetown/armor/plate/half,
-			"Fluted Cuirass"	= /obj/item/clothing/suit/roguetown/armor/plate/half/fluted,
+			"布面甲"		= /obj/item/clothing/suit/roguetown/armor/brigandine/retinue,
+			"札甲罩衣"	= /obj/item/clothing/suit/roguetown/armor/brigandine/coatplates,
+			"钢胸甲"		= /obj/item/clothing/suit/roguetown/armor/plate/half,
+			"纵槽胸甲"	= /obj/item/clothing/suit/roguetown/armor/plate/half/fluted,
 		)
-		var/armorchoice = input(H, "Choose your armor.", "TAKE UP ARMOR") as anything in armors
+		var/armorchoice = input(H, "选择你的护甲。", "整备护甲") as anything in armors
 		armor = armors[armorchoice]
 
 	backpack_contents = list(
