diff --git a/code/modules/jobs/job_types/roguetown/other/vampire_servant.dm b/code/modules/jobs/job_types/roguetown/other/vampire_servant.dm
--- a/code/modules/jobs/job_types/roguetown/other/vampire_servant.dm
+++ b/code/modules/jobs/job_types/roguetown/other/vampire_servant.dm
@@ -1,5 +1,6 @@
 /datum/job/roguetown/vampire_servant
 	title = "Vampire Servant"
+	display_title = "吸血鬼侍从"
 	flag = VAMPIRE_SERVANT
 	department_flag = SLOP
 	faction = "Station"
@@ -28,7 +29,7 @@
 	for(var/datum/advclass/CHECKS in SSrole_class_handler.sorted_class_categories[CTAG_LICKER_WRETCH])
 		possible_classes += CHECKS
 
-	var/datum/advclass/C = input(H.client, "What is my class?", "Adventure") as null|anything in possible_classes
+	var/datum/advclass/C = input(H.client, "我属于哪一类？", "冒险职业") as null|anything in possible_classes
 	C.equipme(H)
 
 	H.adjust_skillrank_up_to(/datum/skill/magic/blood, 3, TRUE)
