diff --git a/code/modules/jobs/job_types/roguetown/other/vampire_guard.dm b/code/modules/jobs/job_types/roguetown/other/vampire_guard.dm
--- a/code/modules/jobs/job_types/roguetown/other/vampire_guard.dm
+++ b/code/modules/jobs/job_types/roguetown/other/vampire_guard.dm
@@ -1,5 +1,6 @@
 /datum/job/roguetown/vampire_guard
 	title = "Vampire Guard"
+	display_title = "吸血鬼护卫"
 	flag = VAMPIRE_GUARD
 	department_flag = SLOP
 	faction = "Station"
@@ -31,7 +32,7 @@
 
 		possible_classes += CHECKS
 
-	var/datum/advclass/C = input(H.client, "What is my class?", "Adventure") as null|anything in possible_classes
+	var/datum/advclass/C = input(H.client, "我属于哪一类？", "冒险职业") as null|anything in possible_classes
 	C.equipme(H)
 
 	H.adjust_skillrank_up_to(/datum/skill/magic/blood, 3, TRUE)
