From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Sat, 30 May 2026 13:30:00 +0800
Subject: [PATCH] 翻译 code/modules/mob/mob.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
diff --git a/code/modules/mob/mob.dm b/code/modules/mob/mob.dm
--- a/code/modules/mob/mob.dm
+++ b/code/modules/mob/mob.dm
@@ -277,22 +277,22 @@
 		var/strz
 		var/strdir
 		if(L.STAPER <= 8 && !(L in viewers(world.view, src)))
-			to_chat(L, span_warning("You hear something... somewhere!"))
+			to_chat(L, span_warning("你听见了什么……在某个地方！"))
 			continue
 		if(L.z != src.z)
 			var/zdiff = abs(L.z - src.z)
 			if(L.z > src.z)
 				switch(zdiff)
 					if(1)
-						strz = "below"
+						strz = "下方"
 					if(2 to 999)
-						strz = "far below"
+						strz = "更下方"
 			if(L.z < src.z)
 				switch(zdiff)
 					if(1)
-						strz = "above"
+						strz = "上方"
 					if(2 to 999)
-						strz = "far above"
+						strz = "更上方"
 		if(directional)
 			var/dir = get_dir(L, src)
 			strdir = dir2text(dir)
@@ -300,7 +300,7 @@
 		//If there's a z difference: "A rock can be heard falling" + "from" + "below/above/etc" + "and" + "northeast"
 		//No z difference: "A rock can be heard falling" + "from" + "northeast"
 		//No dir difference:"A rock can be heard falling" + "from" + "above"
-		var/fullmsg = span_warning("[message] from [strz ? "<b>[strz]</b>" : ""][strdir ? "[strz ? " and " : ""]<b>[strdir]</b>" : ""].")
+		var/fullmsg = span_warning("[message][strz || strdir ? "，来自" : ""][strz ? "<b>[strz]</b>" : ""][strdir ? "[strz ? "的" : ""]<b>[strdir]</b>" : ""]。")
 		to_chat(L, fullmsg)
 /**
  * Show a message to all mobs in earshot of this one
@@ -374,7 +374,7 @@
 			qdel(W)
 		else
 			if(!disable_warning)
-				to_chat(src, "<span class='warning'>I couldn't equip that.</span>")
+				to_chat(src, "<span class='warning'>我没法装备那个。</span>")
 		return FALSE
 	equip_to_slot(W, slot, redraw_mob, initial) //This proc should not ever fail.
 	update_a_intents()
@@ -482,7 +482,7 @@
  * for why this isn't atom/verb/examine()
  */
 /mob/verb/examinate(atom/A as mob|obj|turf in view()) //It used to be oview(12), but I can't really say why
-	set name = "Examine"
+	set name = "检查"
 	set category = "IC"
 	set hidden = 1
 
@@ -491,20 +491,20 @@
 		return
 
 	if(is_blind(src))
-		to_chat(src, span_warning("Something is there but I can't see it!"))
+		to_chat(src, span_warning("那里有东西，但我看不见！"))
 		return
 
 	if(isliving(src) && src.m_intent != MOVE_INTENT_SNEAK && src.stat != DEAD)
 		var/target = "\the [A]"
-		var/message = "[src] looks at"
+		var/message = "[src]看向了"
 		if(!isturf(A))
 			if(A == src)
-				message = "[src] looks over"
-				target = "themselves"
+				message = "[src]打量着"
+				target = "自己"
 			if(A.loc == src)
 				target = "[src.p_their()] [A.name]"
 			if(A.loc.loc == src)
-				message = "[src] looks into"
+				message = "[src]查看了"
 				target = "[src.p_their()] [A.loc.name]"
 			if(isliving(A))
 				var/mob/living/T = A
@@ -584,7 +584,7 @@
  * Calls attack self on the item and updates the inventory hud for hands
  */
 /mob/verb/mode()
-	set name = "Activate Held Object"
+	set name = "使用手持物品"
 	set hidden = 1
 	set src = usr
 
@@ -602,7 +602,7 @@
  * This actually gets the mind datums notes
  */
 /mob/verb/memory()
-	set name = "Notes"
+	set name = "笔记"
 	set category = "IC"
 	set desc = ""
 	if(mind)
@@ -614,7 +614,7 @@
  * Add a note to the mind datum
  */
 /mob/verb/add_memory(msg as message)
-	set name = "AddNote"
+	set name = "添加笔记"
 	set category = "IC"
 	if(mind)
 		if (world.time < memory_throttle_time)
@@ -635,7 +635,7 @@
  * Only works if flag/norespawn is allowed in config
  */
 /mob/verb/abandon_mob()
-	set name = "{ABANDON MOB}"
+	set name = "{放弃身体}"
 	set category = "Options"
 	set hidden = 1
 	if(!check_rights(0))
@@ -643,12 +643,12 @@
 	if (CONFIG_GET(flag/norespawn))
 		return
 	if ((stat != DEAD || !( SSticker )))
-		to_chat(usr, "<span class='boldnotice'>I must be dead to use this!</span>")
+		to_chat(usr, "<span class='boldnotice'>我必须死了才能使用这个！</span>")
 		return
 
 	log_game("[key_name(usr)] used abandon mob.")
 
-	to_chat(src, "<span class='info'>Returned to lobby successfully.</span>")
+	to_chat(src, "<span class='info'>已成功返回大厅。</span>")
 
 	if(!client)
 		log_game("[key_name(usr)] AM failed due to disconnect.")
@@ -674,7 +674,7 @@
  * Sometimes helps if the user is stuck in another perspective or camera
  */
 /mob/verb/cancel_camera()
-	set name = "{RESET CAMERA}"
+	set name = "{重置视角}"
 	set hidden = TRUE
 	set category = null
 	reset_perspective(null)
@@ -780,44 +780,44 @@
 	..()
 	// && check_rights(R_ADMIN,0)
 	var/time_left = SSgamemode.round_ends_at - world.time
-	var/days = "TWILIGHT"
+	var/days = "薄暮"
 	switch(GLOB.dayspassed)
 		if(1)
-			days = "MOON'S DAE"
+			days = "月之日"
 		if(2)
-			days = "TIW'S DAE"
+			days = "提乌之日"
 		if(3)
-			days = "WEDDING'S DAE"
+			days = "婚誓之日"
 		if(4)
-			days = "THULE'S DAE"
+			days = "图勒之日"
 		if(5)
-			days = "FREYJA'S DAE"
+			days = "芙蕾雅之日"
 		if(6)
-			days = "SATURN'S DAE"
+			days = "萨图恩之日"
 		if(7)
-			days = "SUN'S DAE"
+			days = "太阳之日"
 
 	if(client)
-		if(statpanel("RoundInfo"))
-			stat(null, "MAP: [SSmapping.config?.map_name || "Loading..."]")
+		if(statpanel("局内信息"))
+			stat(null, "地图：[SSmapping.config?.map_name || "加载中..."]")
 			var/datum/map_config/cached = SSmapping.next_map_config
 			if(cached)
-				stat(null, "Next Map: [cached.map_name]")
-			stat(null, "ROUND ID: [GLOB.rogue_round_id ? GLOB.rogue_round_id : "NULL"]")
-			stat(null, "ROUND TIME: [time2text(STATION_TIME_PASSED(), "hh:mm:ss", 0)] [world.time - SSticker.round_start_time]")
+				stat(null, "下一张地图：[cached.map_name]")
+			stat(null, "局号：[GLOB.rogue_round_id ? GLOB.rogue_round_id : "NULL"]")
+			stat(null, "局时：[time2text(STATION_TIME_PASSED(), "hh:mm:ss", 0)] [world.time - SSticker.round_start_time]")
 
 			if(SSgamemode.roundvoteend)
-				stat("ROUND END: [DisplayTimeText(time_left)]")
+				stat("结束倒计时：[DisplayTimeText(time_left)]")
 			if(client?.holder)
-				stat(null, "ROUND TrueTime: [worldtime2text()] [world.time]")
-			stat(null, "STORYTELLER: [SSgamemode.storyteller_name]")
-			stat(null, "TIMEOFDAY: [days] ᛉ [uppertext(GLOB.tod)] ᛉ [station_time_timestamp("hh:mm")]")
-			stat(null, "IC Time: [station_time_timestamp()] [station_time()]")
-			stat(null, "PING: [round(client.lastping, 1)]ms (Average: [round(client.avgping, 1)]ms)")
-			stat(null, "TIME DILATION: [round(SStime_track.time_dilation_current,1)]% AVG:([round(SStime_track.time_dilation_avg_fast,1)]%, [round(SStime_track.time_dilation_avg,1)]%, [round(SStime_track.time_dilation_avg_slow,1)]%)")
+				stat(null, "真实时间：[worldtime2text()] [world.time]")
+			stat(null, "故事主宰：[SSgamemode.storyteller_name]")
+			stat(null, "时辰：[days] ᛉ [uppertext(GLOB.tod)] ᛉ [station_time_timestamp("hh:mm")]")
+			stat(null, "IC 时间：[station_time_timestamp()] [station_time()]")
+			stat(null, "延迟：[round(client.lastping, 1)]ms（平均：[round(client.avgping, 1)]ms）")
+			stat(null, "时间膨胀：[round(SStime_track.time_dilation_current,1)]% 均值：([round(SStime_track.time_dilation_avg_fast,1)]%, [round(SStime_track.time_dilation_avg,1)]%, [round(SStime_track.time_dilation_avg_slow,1)]%)")
 			if(check_rights(R_ADMIN,0))
 				stat(null, SSmigrants.get_status_line())
-				stat(null, "Player count: [GLOB.clients.len]") // If someone deletes this again I will slap your balls
+				stat(null, "玩家数：[GLOB.clients.len]") // If someone deletes this again I will slap your balls
 
 	if(client && client.holder && check_rights(R_DEBUG,0))
 		if(statpanel("MC"))
@@ -843,7 +843,7 @@
 					SS.stat_entry()
 		if(length(GLOB.sdql2_queries))
 			if(statpanel("SDQL2"))
-				stat("Access Global SDQL2 List", GLOB.sdql2_vv_statobj)
+				stat("访问全局 SDQL2 列表", GLOB.sdql2_vv_statobj)
 				for(var/i in GLOB.sdql2_queries)
 					var/datum/SDQL2_query/Q = i
 					Q.generate_stat()
@@ -924,7 +924,7 @@
 		return mind.grab_ghost(force = force)
 
 ///Notify a ghost that it's body is being cloned
-/mob/proc/notify_ghost_cloning(message = "Someone is trying to revive you. Re-enter your corpse if you want to be revived!", sound = 'sound/blank.ogg', atom/source = null, flashwindow)
+/mob/proc/notify_ghost_cloning(message = "有人正试图复活你。如果你想被复活，请重新进入你的尸体！", sound = 'sound/blank.ogg', atom/source = null, flashwindow)
 	var/mob/dead/observer/ghost = get_ghost()
 	if(ghost)
 		ghost.notify_cloning(message, sound, source, flashwindow)
@@ -1129,11 +1129,11 @@
 /mob/proc/can_read(obj/O, silent = FALSE)
 	if(is_blind(src) || eye_blurry)
 		if(!silent)
-			to_chat(src, "<span class='warning'>I'm too blind to read.</span>")
+			to_chat(src, "<span class='warning'>我太瞎了，读不了。</span>")
 		return
 	if(!is_literate())
 		if(!silent)
-			to_chat(src, "<span class='warning'>I can't make sense of these verba.</span>")
+			to_chat(src, "<span class='warning'>我无法理解这些文字。</span>")
 		return
 	return TRUE
 
@@ -1154,15 +1154,15 @@
 /mob/vv_get_dropdown()
 	. = ..()
 	VV_DROPDOWN_OPTION("", "---------")
-	VV_DROPDOWN_OPTION(VV_HK_GIB, "Gib")
-	VV_DROPDOWN_OPTION(VV_HK_GIVE_SPELL, "Give Spell")
-	VV_DROPDOWN_OPTION(VV_HK_REMOVE_SPELL, "Remove Spell")
-	VV_DROPDOWN_OPTION(VV_HK_GODMODE, "Toggle Godmode")
-	VV_DROPDOWN_OPTION(VV_HK_DROP_ALL, "Drop Everything")
-	VV_DROPDOWN_OPTION(VV_HK_REGEN_ICONS, "Regenerate Icons")
-	VV_DROPDOWN_OPTION(VV_HK_PLAYER_PANEL, "Show player panel")
-	VV_DROPDOWN_OPTION(VV_HK_DIRECT_CONTROL, "Assume Direct Control")
-	VV_DROPDOWN_OPTION(VV_HK_OFFER_GHOSTS, "Offer Control to Ghosts")
+	VV_DROPDOWN_OPTION(VV_HK_GIB, "爆碎")
+	VV_DROPDOWN_OPTION(VV_HK_GIVE_SPELL, "给予法术")
+	VV_DROPDOWN_OPTION(VV_HK_REMOVE_SPELL, "移除法术")
+	VV_DROPDOWN_OPTION(VV_HK_GODMODE, "切换无敌模式")
+	VV_DROPDOWN_OPTION(VV_HK_DROP_ALL, "丢下全部物品")
+	VV_DROPDOWN_OPTION(VV_HK_REGEN_ICONS, "重建图标")
+	VV_DROPDOWN_OPTION(VV_HK_PLAYER_PANEL, "显示玩家面板")
+	VV_DROPDOWN_OPTION(VV_HK_DIRECT_CONTROL, "直接接管控制")
+	VV_DROPDOWN_OPTION(VV_HK_OFFER_GHOSTS, "向幽魂开放接管")
 
 /mob/vv_do_topic(list/href_list)
 	. = ..()
@@ -1218,7 +1218,7 @@
 
 ///Show the language menu for this mob
 /mob/verb/open_language_menu()
-	set name = "Open Language Menu"
+	set name = "打开语言菜单"
 	set category = "IC"
 	set hidden = 0
 
-- 
Translation Patch Toolkit
