From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Sat, 30 May 2026 13:30:00 +0800
Subject: [PATCH] 翻译 code/modules/mob/point.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
diff --git a/code/modules/mob/point.dm b/code/modules/mob/point.dm
--- a/code/modules/mob/point.dm
+++ b/code/modules/mob/point.dm
@@ -14,7 +14,7 @@
  * overridden here and in /mob/dead/observer for different point span classes and sanity checks
  */
 /mob/verb/pointed(atom/pointed_atom as mob|obj|turf in view())
-	set name = "Point To"
+	set name = "指向"
 	set category = null
 	set hidden = 1
 
@@ -50,9 +50,9 @@
 	lastpoint = world.time
 	var/obj/item/held_item = get_active_held_item()
 	if(held_item)
-		visible_message(span_info("[src] points [held_item] at [pointed_atom]."), span_info("I point [held_item] at [pointed_atom]."))
+		visible_message(span_info("[src]用[held_item]指向了[pointed_atom]。"), span_info("我用[held_item]指向了[pointed_atom]。"))
 	else
-		visible_message(span_info("[src] points at [pointed_atom]."), span_info("I point at [pointed_atom]."))
+		visible_message(span_info("[src]指向了[pointed_atom]。"), span_info("我指向了[pointed_atom]。"))
 
 	return TRUE
 
@@ -97,7 +97,7 @@
 	cut_overlay(thought_bubble)
 
 /obj/effect/temp_visual/point
-	name = "pointer"
+	name = "指示箭头"
 	icon = 'icons/mob/screen_gen.dmi'
 	icon_state = "arrow"
 	plane = POINT_PLANE
-- 
Translation Patch Toolkit
