From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Sat, 30 May 2026 13:30:00 +0800
Subject: [PATCH] 翻译 code/modules/mob/say.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
diff --git a/code/modules/mob/say.dm b/code/modules/mob/say.dm
--- a/code/modules/mob/say.dm
+++ b/code/modules/mob/say.dm
@@ -9,7 +9,7 @@
 	set category = "IC"
 	
 	display_typing_indicator()
-	var/message = input(usr, "", "say") as text|null
+	var/message = input(usr, "", "说话") as text|null
 	// If they don't type anything just drop the message.
 	clear_typing_indicator()
 	if(!length(message))
@@ -17,14 +17,14 @@
 	return say_verb(message)
 
 /mob/verb/say_verb(message as text)
-	set name = "Say"
+	set name = "说话"
 	set category = "IC"
 	set hidden = 1
 
 	if(!length(message))
 		return
 	if(GLOB.say_disabled)	//This is here to try to identify lag problems
-		to_chat(usr, "<span class='danger'>Speech is currently admin-disabled.</span>")
+		to_chat(usr, "<span class='danger'>发言目前被管理员禁用了。</span>")
 		return
 	clear_typing_indicator()		// clear it immediately!
 
@@ -32,12 +32,12 @@
 
 ///Whisper verb
 /mob/verb/whisper_verb(message as text)
-	set name = "Whisper"
+	set name = "低语"
 	set category = "IC"
 	set hidden = 1
 
 	if(GLOB.say_disabled)	//This is here to try to identify lag problems
-		to_chat(usr, span_danger("Speech is currently admin-disabled."))
+		to_chat(usr, span_danger("发言目前被管理员禁用了。"))
 		return
 	whisper(message)
 
@@ -52,7 +52,7 @@
 	set category = "IC"
 
 	display_typing_indicator()
-	var/message = input(usr, "", "me") as text|null
+	var/message = input(usr, "", "动作") as text|null
 	// If they don't type anything just drop the message.
 	clear_typing_indicator()		// clear it immediately!
 	if(!length(message))
@@ -61,7 +61,7 @@
 
 ///The me emote verb
 /mob/verb/me_verb(message as text)
-	set name = "Me"
+	set name = "动作"
 	set category = "IC"
 	set hidden = 1
 #ifndef MATURESERVER
@@ -72,7 +72,7 @@
 	if(!length(message))
 		return
 	if(GLOB.say_disabled)	//This is here to try to identify lag problems
-		to_chat(usr, span_danger("Speech is currently admin-disabled."))
+		to_chat(usr, span_danger("发言目前被管理员禁用了。"))
 		return
 	message = trim(copytext_char(sanitize(message), 1, MAX_MESSAGE_LEN))
 	message = parsemarkdown_basic(message, limited = TRUE, barebones = TRUE)
@@ -86,7 +86,7 @@
 	set hidden = 1
 
 	display_typing_indicator()
-	var/message = input(usr, "", "me") as message|null
+	var/message = input(usr, "", "动作") as message|null
 	// If they don't type anything just drop the message.
 	clear_typing_indicator()
 	if(!length(message))
@@ -95,7 +95,7 @@
 
 ///The me emote verb
 /mob/verb/me_big_verb(message as message)
-	set name = "Me(big)"
+	set name = "动作（大）"
 	set category = "IC"
 	set hidden = 1
 #ifndef MATURESERVER
@@ -106,7 +106,7 @@
 	if(!length(message))
 		return
 	if(GLOB.say_disabled)	//This is here to try to identify lag problems
-		to_chat(usr, span_danger("Speech is currently admin-disabled."))
+		to_chat(usr, span_danger("发言目前被管理员禁用了。"))
 		return
 	message = trim(copytext_char(html_encode(message), 1, MAX_MESSAGE_BIGME))
 	message = parsemarkdown_basic(message, limited = TRUE, barebones = TRUE)
@@ -116,18 +116,18 @@
 
 ///The subtle emote verb
 /mob/verb/subtle_verb()
-	set name = "Subtle"
+	set name = "细微动作"
 	set category = "IC"
 	set hidden = 1
 #ifndef MATURESERVER
 	return
 #endif
-	var/message = input(usr, "", "subtle") as text|null
+	var/message = input(usr, "", "细微动作") as text|null
 	// If they don't type anything just drop the message.
 	if(!length(message))
 		return
 	if(GLOB.say_disabled)
-		to_chat(usr, span_danger("Speech is currently admin-disabled."))
+		to_chat(usr, span_danger("发言目前被管理员禁用了。"))
 		return
 	message = trim(copytext_char(sanitize(message), 1, MAX_MESSAGE_LEN))
 	message = parsemarkdown_basic(message, limited = TRUE, barebones = TRUE)
@@ -137,18 +137,18 @@
 
 ///The subtle emote verb
 /mob/verb/subtle_big_verb()
-	set name = "Subtle(big)"
+	set name = "细微动作（大）"
 	set category = "IC"
 	set hidden = 1
 #ifndef MATURESERVER
 	return
 #endif
-	var/message = input(usr, "", "subtle") as message|null
+	var/message = input(usr, "", "细微动作") as message|null
 	// If they don't type anything just drop the message.
 	if(!length(message))
 		return
 	if(GLOB.say_disabled)
-		to_chat(usr, span_danger("Speech is currently admin-disabled."))
+		to_chat(usr, span_danger("发言目前被管理员禁用了。"))
 		return
 	message = trim(copytext_char(sanitize(message), 1, MAX_MESSAGE_LEN))
 	message = parsemarkdown_basic(message, limited = TRUE, barebones = TRUE)
-- 
Translation Patch Toolkit
