diff --git a/code/modules/mob/living/carbon/human/human.dm b/code/modules/mob/living/carbon/human/human.dm
--- a/code/modules/mob/living/carbon/human/human.dm
+++ b/code/modules/mob/living/carbon/human/human.dm
@@ -8,9 +8,9 @@
 		if(held_item.get_sharpness() && held_item.wlength == WLENGTH_SHORT)
 			if(has_stubble)
 				if(user == src)
-					user.visible_message("<span class='danger'>[user] starts to shave [user.p_their()] stubble with [held_item].</span>")
+					user.visible_message("<span class='danger'>[user]???[held_item]???????</span>")
 				else
-					user.visible_message("<span class='danger'>[user] starts to shave [src]'s stubble with [held_item].</span>")
+					user.visible_message("<span class='danger'>[user]???[held_item]?[src]????</span>")
 				if(do_after(user, 50, needhand = 1, target = src))
 					has_stubble = FALSE
 					update_hair()
@@ -18,9 +18,9 @@
 					held_item.melee_attack_chain(user, src, params)
 			else if(facial_hairstyle != "None")
 				if(user == src)
-					user.visible_message("<span class='danger'>[user] starts to shave [user.p_their()] facehairs with [held_item].</span>")
+					user.visible_message("<span class='danger'>[user]???[held_item]???????</span>")
 				else
-					user.visible_message("<span class='danger'>[user] starts to shave [src]'s facehairs with [held_item].</span>")
+					user.visible_message("<span class='danger'>[user]???[held_item]?[src]????</span>")
 				if(do_after(user, 50, needhand = 1, target = src))
 					facial_hairstyle = "None"
 					update_hair()
@@ -38,7 +38,7 @@
 			if(!underwear)
 				modular_handle_chastity_middleclick_strip(user)
 				return
-			src.visible_message(span_notice("[user] begins to take off [(src==user)?" ":"[src]'s"][underwear]..."))
+			src.visible_message(span_notice("[user]????[(src==user)?"?????":"[src]?"][underwear]??"))
 			if(do_after(user, 30, needhand = 1, target = src))
 				var/obj/item/bodypart/chest = get_bodypart(BODY_ZONE_CHEST)
 				var/obj/item/undies/undies = underwear
@@ -51,7 +51,7 @@
 		if(get_location_accessible(src, BODY_ZONE_PRECISE_GROIN, skipundies = TRUE))
 			if(!legwear_socks)
 				return
-			src.visible_message(span_notice("[user] begins to take off [(src==user)?" ":"[src]'s"][legwear_socks]..."))
+			src.visible_message(span_notice("[user]????[(src==user)?"?????":"[src]?"][legwear_socks]??"))
 			if(do_after(user, 30, needhand = 1, target = src))
 				var/obj/item/bodypart/chest = get_bodypart(BODY_ZONE_CHEST)
 				var/obj/item/legwears/legwears = legwear_socks
@@ -62,7 +62,7 @@
 			return
 	if(HAS_TRAIT(src, TRAIT_PONYGIRL_RIDEABLE))
 		var/mob/living/livinguser = user
-		user.visible_message(span_notice("[livinguser] is trying to mount [src]..."))
+		user.visible_message(span_notice("[livinguser]?????[src]??"))
 		if(!do_after(livinguser, 15, target = src))
 			return
 		if(!istype(livinguser))
@@ -125,18 +125,18 @@
 	switch(rand(1,4))
 		if(1)
 			affecting = get_bodypart(pick(BODY_ZONE_R_LEG, BODY_ZONE_L_LEG))
-			chat_message = span_danger("I fall on my [affecting]!")
+			chat_message = span_danger("??????[affecting]??")
 		if(2)
 			affecting = get_bodypart(pick(BODY_ZONE_R_ARM, BODY_ZONE_L_ARM))
-			chat_message = span_danger("I fall on my arm!")
+			chat_message = span_danger("??????????")
 		if(3)
 			affecting = get_bodypart(BODY_ZONE_CHEST)
-			chat_message = span_danger("I fall flat! I'm winded!")
+			chat_message = span_danger("?????????????????")
 			emote("gasp")
 			adjustOxyLoss(50)
 		if(4)
 			affecting = get_bodypart(BODY_ZONE_HEAD)
-			chat_message = span_danger("I fall on my head!")
+			chat_message = span_danger("????????")
 	if(affecting && apply_damage(dam, BRUTE, affecting, run_armor_check(affecting, "blunt", damage = dam)))
 		update_damage_overlays()
 		if(levels >= 1)
@@ -352,7 +352,7 @@
 					. = 0
 	if(!. && error_msg && user)
 		// Might need re-wording.
-		to_chat(user, span_alert("There is no exposed flesh or thin material [above_neck(target_zone) ? "on [p_their()] head" : "on [p_their()] body"]."))
+		to_chat(user, span_alert("???[above_neck(target_zone) ? "??" : "??"]???????????????"))
 
 //Used for new human mobs created by cloning/constructing/podding
 /mob/living/carbon/human/proc/set_cloned_appearance()
@@ -368,20 +368,20 @@
 	CHECK_DNA_AND_SPECIES(C)
 
 	if(C.stat == DEAD || (HAS_TRAIT(C, TRAIT_FAKEDEATH)))
-		to_chat(src, span_warning("[C.name] is dead!"))
+		to_chat(src, span_warning("[C.name]?????"))
 		return
 	if(is_mouth_covered())
-		to_chat(src, span_warning("Remove your mask first!"))
+		to_chat(src, span_warning("??????"))
 		return 0
 	if(C.is_mouth_covered())
-		to_chat(src, span_warning("Remove [p_their()] mask first!"))
+		to_chat(src, span_warning("???[C.name]????"))
 		return 0
 
 	if(C.cpr_time < world.time + 30)
-		visible_message(span_notice("[src] is trying to perform CPR on [C.name]!"), \
-						span_notice("I try to perform CPR on [C.name]... Hold still!"))
+		visible_message(span_notice("[src]????[C.name]???????"), \
+						span_notice("?????[C.name]????????????"))
 		if(!do_mob(src, C))
-			to_chat(src, span_warning("I fail to perform CPR on [C]!"))
+			to_chat(src, span_warning("????[C]???????"))
 			return 0
 
 		var/they_breathe = !HAS_TRAIT(C, TRAIT_NOBREATH)
@@ -390,7 +390,7 @@
 		if(C.health > C.crit_threshold)
 			return
 
-		src.visible_message(span_notice("[src] performs CPR on [C.name]!"), span_notice("I perform CPR on [C.name]."))
+		src.visible_message(span_notice("[src]?[C.name]????????"), span_notice("??[C.name]????????"))
 		C.cpr_time = world.time
 		log_combat(src, C, "CPRed")
 
@@ -398,11 +398,11 @@
 			var/suff = min(C.getOxyLoss(), 7)
 			C.adjustOxyLoss(-suff)
 			C.updatehealth()
-			to_chat(C, span_unconscious("I feel a breath of fresh air enter your lungs... It feels good..."))
+			to_chat(C, span_unconscious("???????????????????????"))
 		else if(they_breathe && !they_lung)
-			to_chat(C, span_unconscious("I feel a breath of fresh air... but you don't feel any better..."))
+			to_chat(C, span_unconscious("??????????????????????"))
 		else
-			to_chat(C, span_unconscious("I feel a breath of fresh air... which is a sensation you don't recognise..."))
+			to_chat(C, span_unconscious("???????????????????????"))
 
 /mob/living/carbon/human/cuff_resist(obj/item/I)
 	if(..())
@@ -441,13 +441,13 @@
 
 /mob/living/carbon/human/canUseTopic(atom/movable/M, be_close=FALSE, no_dexterity=FALSE, no_tk=FALSE)
 	if(!(mobility_flags & MOBILITY_UI))
-		to_chat(src, span_warning("I can't do that right now!"))
+		to_chat(src, span_warning("???????"))
 		return FALSE
 	if(incapacitated())
-		to_chat(src, span_warning("I can't do that right now!"))
+		to_chat(src, span_warning("???????"))
 		return FALSE
 	if(be_close && !in_range(M, src))
-		to_chat(src, span_warning("I am too far away!"))
+		to_chat(src, span_warning("??????"))
 		return FALSE
 	return TRUE
 
@@ -690,8 +690,8 @@
 /mob/living/carbon/human/vomit(lost_nutrition = 10, blood = 0, stun = 1, distance = 0, message = 1, toxic = 0)
 	if(blood && (NOBLOOD in dna.species.species_traits) && !HAS_TRAIT(src, TRAIT_TOXINLOVER))
 		if(message)
-			visible_message(span_warning("[src] dry heaves!"), \
-							span_danger("I try to throw up, but there's nothing in your stomach!"))
+			visible_message(span_warning("[src]?????"), \
+							span_danger("??????????????????"))
 		if(stun)
 			Immobilize(200)
 		return 1
@@ -817,7 +817,7 @@
 			//If they dragged themselves and we're currently aggressively grabbing them try to piggyback (not on cmode)
 			if(user == target && can_piggyback(target))
 				if(cmode)
-					to_chat(target, span_warning("[src] won't let you on!"))
+					to_chat(target, span_warning("[src]???????"))
 					return FALSE
 				piggyback(target)
 				return TRUE
@@ -858,33 +858,33 @@
 
 	if(can_be_firemanned(target) && !incapacitated(FALSE, TRUE))
 		if(backnotshoulder)
-			visible_message(span_notice("[src] starts lifting [target] onto their back.."))
+			visible_message(span_notice("[src]???[target]????????"))
 		else
-			visible_message(span_notice("[src] starts lifting [target] onto their shoulder.."))
+			visible_message(span_notice("[src]???[target]????????"))
 		if(do_after(src, carrydelay, TRUE, target))
 			//Second check to make sure they're still valid to be carried
 			if(can_be_firemanned(target) && !incapacitated(FALSE, TRUE))
 				buckle_mob(target, TRUE, TRUE, 90, 0, 0)
 				return
-	to_chat(src, span_warning("I fail to carry [target]."))
+	to_chat(src, span_warning("?????[target]?"))
 
 /mob/living/carbon/human/proc/piggyback(mob/living/carbon/target)
 	if(can_piggyback(target))
-		visible_message(span_notice("[target] starts to climb onto [src]..."))
+		visible_message(span_notice("[target]???[src]?????"))
 		if(do_after(target, 15, target = src))
 			if(can_piggyback(target))
 				if(target.incapacitated(FALSE, TRUE) || incapacitated(FALSE, TRUE))
-					to_chat(target, span_warning("I can't piggyback ride [src]."))
+					to_chat(target, span_warning("?????[src]???"))
 					return
 				buckle_mob(target, TRUE, TRUE, FALSE, 0, 0)
 	else
-		to_chat(target, span_warning("I can't piggyback ride [src]."))
+		to_chat(target, span_warning("?????[src]???"))
 
 /mob/living/carbon/human/buckle_mob(mob/living/target, force = FALSE, check_loc = TRUE, lying_buckle = FALSE, hands_needed = 0, target_hands_needed = 0)
 	if(!force)//humans are only meant to be ridden through piggybacking and special cases
 		return
 	if(!is_type_in_typecache(target, can_ride_typecache))
-		target.visible_message(span_warning("[target] really can't seem to mount [src]..."))
+		target.visible_message(span_warning("[target]?????????[src]????"))
 		return
 	buckle_lying = lying_buckle
 	var/datum/component/riding/human/riding_datum = LoadComponent(/datum/component/riding/human)
@@ -901,12 +901,12 @@
 
 	if(hands_needed || target_hands_needed)
 		if(hands_needed && !equipped_hands_self)
-			src.visible_message(span_warning("[src] can't get a grip on [target] because their hands are full!"),
-				span_warning("I can't get a grip on [target] because your hands are full!"))
+			src.visible_message(span_warning("[src]???????????[target]?"),
+				span_warning("???????????[target]?"))
 			return
 		else if(target_hands_needed && !equipped_hands_target)
-			target.visible_message(span_warning("[target] can't get a grip on [src] because their hands are full!"),
-				span_warning("I can't get a grip on [src] because your hands are full!"))
+			target.visible_message(span_warning("[target]???????????[src]?"),
+				span_warning("???????????[src]?"))
 			return
 
 	//stop_pulling()
@@ -926,7 +926,7 @@
 	remove_movespeed_modifier(MOVESPEED_ID_SHOVE)
 	var/active_item = get_active_held_item()
 	if(is_type_in_typecache(active_item, GLOB.shove_disarming_types))
-		visible_message(span_warning("[src.name] regains their grip on \the [active_item]!"), span_warning("I regain your grip on \the [active_item]"), null, COMBAT_MESSAGE_RANGE)
+		visible_message(span_warning("[src.name]?????\the [active_item]?"), span_warning("??????\the [active_item]?"), null, COMBAT_MESSAGE_RANGE)
 
 /mob/living/carbon/human/do_after_coefficent()
 	. = ..()
@@ -1086,20 +1086,20 @@
 /mob/living/carbon/human/update_mobility()
 	. = ..()
 	if(!(mobility_flags & MOBILITY_CANSTAND) && mouth?.spitoutmouth)
-		visible_message(span_warning("[src] spits out [mouth]."))
+		visible_message(span_warning("[src]?[mouth]?????"))
 		dropItemToGround(mouth, silent = FALSE)
 
 /mob/living/carbon/human/proc/cold_warn()
 	if(src.bodytemperature <= BODYTEMP_COLD_LEVEL_ONE_MAX)
-		to_chat(src, span_danger("I feel so cold and numb, I can't stop shivering."))
-	else
-		to_chat(src, span_warning("Everything is cold."))
+		to_chat(src, span_danger("?????????????"))
+	else
+		to_chat(src, span_warning("????????"))
 	return
 /mob/living/carbon/human/proc/heat_warn()
 	if(src.bodytemperature >= BODYTEMP_HEAT_LEVEL_ONE_MAX)
-		to_chat(src, span_danger("My lips feel cracked and dry, and it is unbearably hot."))
-	else
-		to_chat(src, span_warning("Sweat drips down my brow."))
+		to_chat(src, span_danger("????????????????"))
+	else
+		to_chat(src, span_warning("???????????"))
 	return
 
 /mob/living/carbon/human/proc/apply_hypothermia()
@@ -1137,7 +1137,7 @@
 	var/obj/item/bodypart/BP = get_bodypart(def_zone)
 
 	if(BP)
-		to_chat(src, span_warning("I feel painfully cold in my [BP]..."))
+		to_chat(src, span_warning("??????[BP]??????"))
 		BP.add_wound(/datum/wound/hypothermia)
 
 
@@ -1152,7 +1152,7 @@
 		if(istype(W, /datum/wound/heatexhaustion)||istype(W, /datum/wound/heatstroke))
 			return
 	if(BP)
-		to_chat(H, span_userdanger("My head is spinning and I feel terrible!"))
+		to_chat(H, span_userdanger("???????????"))
 		BP.add_wound(/datum/wound/heatexhaustion)
 		BP.update_disabled()
 
@@ -1170,8 +1170,8 @@
 
 	if(found)
 		visible_message(
-			span_notice("[src]'s breathing steadies as the heat leaves their body."),
-			span_notice("The cold helps draw the heat out of your body.")
+			span_notice("[src]?????????????????"),
+			span_notice("??????????????")
 		)
 
 /mob/living/carbon/human/proc/apply_weather_temperature(base_delta, exposure_temp = null)
