diff --git a/code/modules/jobs/job_types/roguetown/byos/tribalvillager.dm b/code/modules/jobs/job_types/roguetown/byos/tribalvillager.dm
--- a/code/modules/jobs/job_types/roguetown/byos/tribalvillager.dm
+++ b/code/modules/jobs/job_types/roguetown/byos/tribalvillager.dm
@@ -1,5 +1,6 @@
 /datum/job/roguetown/tribalvillager
 	title = "Tribal Villager"
+	display_title = "部落村民"
 	flag = TRIBALVILLAGER
 	department_flag = TRIBAL
 	faction = "Station"
@@ -11,9 +12,9 @@
 	allowed_races = list(/datum/species/goblinp, /datum/species/anthromorphsmall, /datum/species/kobold)
 	allowed_ages = list(AGE_ADULT, AGE_MIDDLEAGED, AGE_OLD)
 	// tutorial = "Ooga Chacka WORK-a chacka."
-	tutorial = "You are the lowest of the low in the tribe. You're the camp's laborer, in service to the Chief before all others. \
-	They've chosen you, as the weakest of them all, to prepare and service the camp in whatever fashion they see fit. \
-	Make sure everyone is fed, healthy, and satisfied, all while hoping maybe one day you'll be something more."
+	tutorial = "你是部族里最底层的一员，是营地里的苦工，凡事都得先服侍酋长。 \
+	他们挑中了你这个最弱小的家伙，让你按他们的心意去操持营地的杂务与供给。 \
+	确保人人有饭吃、身体撑得住、没有怨言，同时心里还要盼着，也许总有一天你能变得不止如此。"
 	display_order = JDO_TRIBALVILLAGER
 	whitelist_req = TRUE
