diff --git a/code/modules/roguetown/roguecrafting/leather/leather_armor_recipes.dm b/code/modules/roguetown/roguecrafting/leather/leather_armor_recipes.dm
--- a/code/modules/roguetown/roguecrafting/leather/leather_armor_recipes.dm
+++ b/code/modules/roguetown/roguecrafting/leather/leather_armor_recipes.dm
@@ -3,56 +3,56 @@
 	category = "Armor"
 
 /datum/crafting_recipe/roguetown/leather/armor/lgorget
-	name = "hardened leather gorget"
+	name = "硬化皮护喉"
 	result = /obj/item/clothing/neck/roguetown/leather
 	reqs = list(/obj/item/natural/hide/cured = 1,
 				/obj/item/natural/fibers = 1)
 	craftdiff = 4
 
 /datum/crafting_recipe/roguetown/leather/armor/heavybracers
-	name = "hardened leather bracers"
+	name = "硬化皮护臂"
 	result = /obj/item/clothing/wrists/roguetown/bracers/leather/heavy
 	reqs = list(/obj/item/natural/hide/cured = 2,
 				/obj/item/natural/fibers = 1)
 	craftdiff = 4
 
 /datum/crafting_recipe/roguetown/leather/armor/bracers
-	name = "leather bracers"
+	name = "皮护臂"
 	result = /obj/item/clothing/wrists/roguetown/bracers/leather
 	reqs = list(/obj/item/natural/hide/cured = 1)
 	sellprice = 10
 	craftdiff = 2
 
 /datum/crafting_recipe/roguetown/leather/armor/pants
-	name = "leather pants"
+	name = "皮裤"
 	result = /obj/item/clothing/under/roguetown/trou/leather
 	reqs = list(/obj/item/natural/hide/cured = 2)
 	sellprice = 10
 	craftdiff = 2
 
 /datum/crafting_recipe/roguetown/leather/armor/volfhelm
-	name = "volf helm"
+	name = "Volf 头盔"
 	result = list(/obj/item/clothing/head/roguetown/helmet/leather/volfhelm)
 	reqs = list(/obj/item/natural/hide/cured = 1, /obj/item/natural/fur/wolf = 1, /obj/item/natural/head/volf = 1)
 	sellprice = 20
 	craftdiff = 2
 
 /datum/crafting_recipe/roguetown/leather/armor/volfmantle
-	name = "volf mantle"
+	name = "Volf 披肩"
 	result = /obj/item/clothing/cloak/volfmantle
 	reqs = list(
 		/obj/item/natural/hide/cured = 2,
 		/obj/item/natural/head/volf = 1)
 
 /datum/crafting_recipe/roguetown/leather/armor/saigahelm
-	name = "saiga skull helm"
+	name = "黄羊头骨盔"
 	result = list(/obj/item/clothing/head/roguetown/helmet/leather/saiga)
 	reqs = list(/obj/item/natural/hide/cured = 1, /obj/item/natural/hide = 2, /obj/item/natural/head/saiga = 1)
 	sellprice = 20
 	craftdiff = 2
 
 /datum/crafting_recipe/roguetown/leather/armor/heavy_leather_pants
-	name = "hardened leather pants"
+	name = "硬化皮裤"
 	result = list(/obj/item/clothing/under/roguetown/heavy_leather_pants)
 	reqs = list(
 		/obj/item/natural/hide/cured = 3,
@@ -63,7 +63,7 @@
 	craftdiff = 4
 
 /datum/crafting_recipe/roguetown/leather/armor/heavy_leather_pants/shorts
-	name = "hardened leather shorts"
+	name = "硬化皮短裤"
 	result = list(/obj/item/clothing/under/roguetown/heavy_leather_pants/shorts)
 	reqs = list(
 		/obj/item/natural/hide/cured = 2, //they cover less, you see
@@ -74,7 +74,7 @@
 	craftdiff = 3
 
 /datum/crafting_recipe/roguetown/leather/armor/helmet/advanced
-	name = "hardened leather helmet"
+	name = "硬化皮头盔"
 	result = /obj/item/clothing/head/roguetown/helmet/leather/advanced
 	reqs = list(/obj/item/natural/hide/cured = 1,
 				/obj/item/natural/fibers = 1,
@@ -82,26 +82,26 @@
 	craftdiff = 4
 
 /datum/crafting_recipe/roguetown/leather/armor/armor
-	name = "leather armor"
+	name = "皮甲"
 	result = /obj/item/clothing/suit/roguetown/armor/leather
 	reqs = list(/obj/item/natural/hide/cured = 2)
 	craftdiff = 2
 
 /datum/crafting_recipe/roguetown/leather/armor/cuirass
-	name = "leather cuirass"
+	name = "皮胸甲"
 	result = /obj/item/clothing/suit/roguetown/armor/leather/cuirass
 	reqs = list(/obj/item/natural/hide/cured = 2)
 	craftdiff = 2
 
 /datum/crafting_recipe/roguetown/leather/armor/hidearmor
-	name = "hide armor"
+	name = "兽皮甲"
 	result = /obj/item/clothing/suit/roguetown/armor/leather/hide
 	reqs = list(/obj/item/natural/hide/cured = 1,
 				/obj/item/natural/fur = 1)
 	craftdiff = 2
 
 /datum/crafting_recipe/roguetown/leather/armor/heavy_leather_armor
-	name = "hardened leather armor"
+	name = "硬化皮甲"
 	result = /obj/item/clothing/suit/roguetown/armor/leather/heavy
 	reqs = list(
 		/obj/item/natural/hide/cured = 3,
@@ -110,7 +110,7 @@
 	craftdiff = 4
 
 /datum/crafting_recipe/roguetown/leather/armor/freivest
-	name = "fencing jacket"	//Expensive on purpose.
+	name = "击剑短上衣"	//Expensive on purpose.
 	result = /obj/item/clothing/suit/roguetown/armor/leather/heavy/freifechter
 	reqs = list(
 		/obj/item/natural/hide/cured = 4,
@@ -120,7 +120,7 @@
 	craftdiff = 4
 
 /datum/crafting_recipe/roguetown/leather/armor/heavy_leather_armor/coat
-	name = "hardened leather coat"
+	name = "硬化皮外套"
 	result = /obj/item/clothing/suit/roguetown/armor/leather/heavy/coat
 	reqs = list(
 		/obj/item/natural/hide/cured = 4,
@@ -129,7 +129,7 @@
 	craftdiff = 5
 
 /datum/crafting_recipe/roguetown/leather/armor/heavy_leather_armor/jacket
-	name = "hardened leather jacket"
+	name = "硬化皮夹克"
 	result = /obj/item/clothing/suit/roguetown/armor/leather/heavy/jacket
 	reqs = list(
 		/obj/item/natural/hide/cured = 3,
@@ -138,20 +138,20 @@
 	craftdiff = 4
 
 /datum/crafting_recipe/roguetown/leather/armor/helmet
-	name = "leather helmet"
+	name = "皮头盔"
 	result = /obj/item/clothing/head/roguetown/helmet/leather
 	reqs = list(/obj/item/natural/hide/cured = 1)
 	craftdiff = 2
 
 /datum/crafting_recipe/roguetown/leather/studhood
-	name = "leather padded hood"
+	name = "加垫皮兜帽"
 	result = /obj/item/clothing/head/roguetown/helmet/leather/armorhood
 	reqs = list(/obj/item/natural/hide/cured = 2,
 		/obj/item/natural/fibers = 1)
 	craftdiff = 4
 
 /datum/crafting_recipe/roguetown/leather/armor/duelcoat
-	name = "leather coat"
+	name = "皮外套"
 	result = /obj/item/clothing/armor/leather/jacket/leathercoat/duelcoat
 	reqs = list(
 		/obj/item/natural/hide/cured = 4,
