diff --git a/code/modules/roguetown/roguecrafting/leather/reinforcement.dm b/code/modules/roguetown/roguecrafting/leather/reinforcement.dm
--- a/code/modules/roguetown/roguecrafting/leather/reinforcement.dm
+++ b/code/modules/roguetown/roguecrafting/leather/reinforcement.dm
@@ -3,7 +3,7 @@
 	category = "Reinforcement"
 
 /datum/crafting_recipe/roguetown/leather/reinforcement/crafteast
-	name = "decorated dobo robe"
+	name = "装饰 dobo 长袍"
 	result = list(/obj/item/clothing/suit/roguetown/armor/basiceast/crafteast)
 	reqs = list(
 		/obj/item/natural/hide/cured = 2,
