From 75eff5f3ab02d5912892ceef646765af98be418a Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Sat, 30 May 2026 13:00:00 +0800
Subject: [PATCH] 翻译 code/modules/mob/living/combat/accuracy_checks.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
diff --git a/code/modules/mob/living/combat/accuracy_checks.dm b/code/modules/mob/living/combat/accuracy_checks.dm
--- a/code/modules/mob/living/combat/accuracy_checks.dm
+++ b/code/modules/mob/living/combat/accuracy_checks.dm
@@ -68,7 +68,7 @@
 		if(prob(chance2hit+(user.STAPER - 10)))
 			if(check_zone(zone) == zone)
 				return zone
-			to_chat(user, span_warning("Accuracy fail! [chance2hit]%"))
+			to_chat(user, span_warning("命中偏差！[chance2hit]%"))
 			if(user.STAPER >= 11)
 				if(user.client?.prefs.showrolls)
 					return check_zone(zone)
@@ -76,7 +76,7 @@
 				return BODY_ZONE_CHEST
 		else
 			if(user.client?.prefs.showrolls)
-				to_chat(user, span_warning("Double accuracy fail! [chance2hit]%"))
+				to_chat(user, span_warning("双重命中偏差！[chance2hit]%"))
 			return BODY_ZONE_CHEST
 
 /mob/living/proc/checkmiss(mob/living/user)
@@ -91,7 +91,7 @@
 		return TRUE
 
 /proc/badluckmessage(mob/living/user)
-	var/static/list/usedp = list("Critical miss!", "Damn! Critical miss!", "No! Critical miss!", "It can't be! Critical miss!", "Xylix laughs at me! Critical miss!", "Bad luck! Critical miss!", "Curse creation! Critical miss!", "What?! Critical miss!")
+	var/static/list/usedp = list("致命失误！", "该死！致命失误！", "不！致命失误！", "怎么会这样！致命失误！", "希利克斯在嘲笑我！致命失误！", "真倒霉！致命失误！", "该死的造化！致命失误！", "什么？！致命失误！")
 	to_chat(user, span_boldwarning("[pick(usedp)]"))
 	user.flash_fullscreen("blackflash2")
 	user.aftermiss()
-- 
Translation Patch Toolkit
