From 1838eef3194566910be09b776a8ea065fafd7866 Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Sat, 30 May 2026 13:00:00 +0800
Subject: [PATCH] 翻译 code/modules/mob/living/combat/dodge.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
diff --git a/code/modules/mob/living/combat/dodge.dm b/code/modules/mob/living/combat/dodge.dm
--- a/code/modules/mob/living/combat/dodge.dm
+++ b/code/modules/mob/living/combat/dodge.dm
@@ -56,7 +56,7 @@
 		if(pulledby)
 			return FALSE
 		if(!turfy)
-			to_chat(src, span_boldwarning("There's nowhere to dodge to!"))
+			to_chat(src, span_boldwarning("已经没有可供闪避的地方了！"))
 			return FALSE
 		else
 			if(do_dodge(user, turfy))
@@ -68,7 +68,7 @@
 					if(H.magearmor == 0)
 						H.magearmor = 1
 						H.apply_status_effect(/datum/status_effect/buff/magearmor)
-						to_chat(src, span_boldwarning("My mage armor absorbs the hit and dissipates!"))
+						to_chat(src, span_boldwarning("我的法师护甲吸收了这次打击，随后消散了！"))
 						return TRUE
 					else
 						return FALSE
@@ -192,21 +192,21 @@
 
 		var/attacker_feedback
 		if(user.client?.prefs.showrolls && (attacker_dualw || defender_dualw))
-			attacker_feedback = "Attacking with advantage. ([100 - ((prob2defend / 100) * (prob2defend / 100) * 100)]%)"
+			attacker_feedback = "攻击方具有优势。（[100 - ((prob2defend / 100) * (prob2defend / 100) * 100)]%）"
 
 		if(src.client?.prefs.showrolls)
-			var/text = "Roll to dodge... [prob2defend]%"
+			var/text = "闪避判定……[prob2defend]%"
 			if((defender_dualw || attacker_dualw))
 				if(defender_dualw && attacker_dualw)
-					text += " Our dual wielding cancels out!"
+					text += " 我们的双持优势互相抵消了！"
 				else//If we're defending against or as a dual wielder, we roll disadv. But if we're both dual wielding it cancels out.
-					text += " Twice! Disadvantage! ([(prob2defend / 100) * (prob2defend / 100) * 100]%)"
+					text += " 两次判定！劣势！（[(prob2defend / 100) * (prob2defend / 100) * 100]%）"
 			to_chat(src, span_info("[text]"))
 
 		var/dodge_status = FALSE
 		if((!defender_dualw && !attacker_dualw) || (defender_dualw && attacker_dualw)) //They cancel each other out
 			if(attacker_feedback)
-				attacker_feedback = "Advantage cancelled out!"
+				attacker_feedback = "优势被抵消了！"
 			if(prob(prob2defend))
 				dodge_status = TRUE
 		else if(attacker_dualw)
@@ -224,21 +224,21 @@
 		if(!UH?.mind) // For NPC, reduce the drained to 5 stamina
 			drained = drained_npc
 		if(!H.stamina_add(max(drained,5)))
-			to_chat(src, span_warning("I'm too tired to dodge!"))
+			to_chat(src, span_warning("我太累了，没法闪避！"))
 			return FALSE
 	else //we are a non human
 		prob2defend = clamp(prob2defend, 5, 90)
 		if(client?.prefs.showrolls)
-			to_chat(src, span_info("Roll to dodge... [prob2defend]%"))
+			to_chat(src, span_info("闪避判定……[prob2defend]%"))
 		if(!prob(prob2defend))
 			return FALSE
 	dodgecd = TRUE
 	playsound(src, 'sound/combat/dodge.ogg', 100, FALSE)
 	throw_at(turfy, 1, 2, src, FALSE)
 	if(drained > 0)
-		src.visible_message(span_warning("<b>[src]</b> dodges [user]'s attack!"))
+		src.visible_message(span_warning("<b>[src]</b>闪开了[user]的攻击！"))
 	else
-		src.visible_message(span_warning("<b>[src]</b> easily dodges [user]'s attack!"))
+		src.visible_message(span_warning("<b>[src]</b>轻松闪开了[user]的攻击！"))
 	if(get_dist(src, user) <= user.used_intent?.reach)	//We are still in range of the attacker's weapon post-dodge
 		var/probclip = 50
 		var/obj/item/IS = L.get_active_held_item()
@@ -264,7 +264,7 @@
 			IS.take_damage(intdam, BRUTE, IU.d_type)
 			IS.remove_bintegrity(sharp_loss, src)
 
-			user.visible_message(span_warning("<b>[user]</b> clips [src]'s weapon!"))
+			user.visible_message(span_warning("<b>[user]</b>擦中了[src]的武器！"))
 			playsound(user, 'sound/misc/weapon_clip.ogg', 100)
 
 	if(mind && user.mind && HAS_TRAIT(src, TRAIT_COMBAT_AWARE))
-- 
Translation Patch Toolkit
