diff --git a/code/modules/roguetown/roguejobs/blacksmith/tools.dm b/code/modules/roguetown/roguejobs/blacksmith/tools.dm
--- a/code/modules/roguetown/roguejobs/blacksmith/tools.dm
+++ b/code/modules/roguetown/roguejobs/blacksmith/tools.dm
@@ -2,8 +2,8 @@
 /obj/item/rogueweapon/hammer
 	force = 21
 	possible_item_intents = list(/datum/intent/mace/strike, /datum/intent/mace/smash)
-	name = "template hammer"
-	desc = "If you see this - scream, cry, piss, run, shit yourself, then report it to a dev. Shouldn't be here."
+	name = "模板锤"
+	desc = "如果你看到了这个，那就尖叫、哭泣、失禁、逃跑、吓得屁滚尿流，然后向开发者报告。它本不该出现在这里。"
 	icon_state = "hammer"
 	icon = 'icons/roguetown/weapons/tools.dmi'
 	sharpness = IS_BLUNT
@@ -30,7 +30,7 @@
 
 /obj/item/rogueweapon/hammer/attack_hand(mob/living/user)
 	if(HAS_TRAIT(user, TRAIT_CURSE_MALUM))
-		to_chat(user, span_warning("Your cursed hands burn at the touch of the hammer!"))
+		to_chat(user, span_warning("你受诅咒的双手一碰到锤子就灼痛起来！"))
 		user.freak_out()
 		return
 	. = ..()
@@ -51,7 +51,7 @@
 		if(!attacked_prosthetic.anvilrepair) //No hammering flesh limbs
 			return
 		if(attacked_prosthetic.obj_integrity >= attacked_prosthetic.max_integrity && attacked_prosthetic.brute_dam == 0 && attacked_prosthetic.burn_dam == 0 && attacked_prosthetic.wounds == null && attacked_prosthetic.disabled == BODYPART_NOT_DISABLED) //A mouthful
-			to_chat(user, span_warning("There is nothing to further repair on [attacked_prosthetic]."))
+			to_chat(user, span_warning("[attacked_prosthetic]已经没什么可继续修的了。"))
 			return
 		if(blacksmith.get_skill_level(attacked_prosthetic.anvilrepair) <= 0)
 			if(prob(30))
@@ -70,15 +70,15 @@
 			attacked_prosthetic.wounds = null //Fixing fractures
 			attacked_prosthetic.disabled = BODYPART_NOT_DISABLED
 			if(repair_percent == 0.01) // If an inexperienced repair attempt has been successful
-				to_chat(user, span_warning("You fumble your way into slightly repairing [attacked_prosthetic]."))
-			else
-				user.visible_message(span_info("[user] repairs [attacked_prosthetic]!"))
+				to_chat(user, span_warning("你笨手笨脚地勉强修好了[attacked_prosthetic]一点点。"))
+			else
+				user.visible_message(span_info("[user]修好了[attacked_prosthetic]！"))
 			blacksmith.mind.add_sleep_experience(attacked_prosthetic.anvilrepair, exp_gained/2) //We gain as much exp as we fix divided by 2
 			if(do_after(user, CLICK_CD_MELEE, target = attacked_object))
 				attack_obj(attacked_object, user)
 			return
 		else
-			user.visible_message(span_warning("[user] fumbles trying to repair [attacked_prosthetic]!"))
+			user.visible_message(span_warning("[user]试图修理[attacked_prosthetic]时失了手！"))
 			if(do_after(user, CLICK_CD_MELEE, target = attacked_object))
 				attack_obj(attacked_object, user)
 			return
@@ -100,11 +100,11 @@
 			return
 
 		if(!attacked_item.ontable())
-			to_chat(user, span_warning("I should put this on a table or an anvil first."))
+			to_chat(user, span_warning("我应该先把它放到桌上或铁砧上。"))
 			return
 
 		if (unskilled && !attacked_item.obj_broken && attacked_item.shoddy_repair && integrity_percentage >= 60)
-			to_chat(user, span_warning("I can't do anything else to fix this right now - I should see a skilled craftsman."))
+			to_chat(user, span_warning("我现在没法再继续修它了，我该去找个手艺高超的工匠。"))
 			return
 
 		if(repair_skill <= 0)
@@ -124,36 +124,36 @@
 			attacked_item.obj_integrity = min(attacked_item.obj_integrity + repair_percent, attacked_item.max_integrity)
 			integrity_percentage = (attacked_item.obj_integrity / attacked_item.max_integrity) * 100
 			if(repair_percent == 0.01) // If an inexperienced repair attempt has been successful
-				to_chat(user, span_warning("You fumble your way into slightly repairing [attacked_item]."))
-			else
-				user.visible_message(span_info("[user] repairs [attacked_item]!"))
+				to_chat(user, span_warning("你笨手笨脚地勉强修好了[attacked_item]一点点。"))
+			else
+				user.visible_message(span_info("[user]修好了[attacked_item]！"))
 				if(attacked_item.body_parts_covered != attacked_item.body_parts_covered_dynamic)
-					user.visible_message(span_info("[user] repairs [attacked_item]'s coverage!"))
+					user.visible_message(span_info("[user]修复了[attacked_item]的覆盖部位！"))
 					attacked_item.repair_coverage()
 			if(attacked_item.obj_broken)
 				var/do_fix = FALSE
 				if (unskilled && integrity_percentage >= 60)
 					attacked_item.shoddy_repair = TRUE
-					blacksmith.visible_message(span_info("[blacksmith] finishes field-repairing [attacked_item]."))
-					to_chat(user, span_warning("I should get this properly fixed by a skilled craftsman later."))
+					blacksmith.visible_message(span_info("[blacksmith]完成了对[attacked_item]的临时修补。"))
+					to_chat(user, span_warning("我之后应该找个技术娴熟的工匠把它彻底修好。"))
 					do_fix = TRUE
 				else if (!unskilled && integrity_percentage >= 100)
 					if (attacked_item.shoddy_repair)
 						attacked_item.shoddy_repair = FALSE
-						to_chat(user, span_notice("My skilled hand has fully repaired this item."))
+						to_chat(user, span_notice("我熟练的双手已经把这件物品完全修好了。"))
 					do_fix = TRUE
 				if (do_fix)
 					attacked_item.obj_fix()
 					return
 			else if (!attacked_item.obj_broken && !unskilled && attacked_item.shoddy_repair && integrity_percentage >= 100)
 				attacked_item.shoddy_repair = FALSE
-				to_chat(user, span_notice("My skilled hand has fully repaired this item."))
+				to_chat(user, span_notice("我熟练的双手已经把这件物品完全修好了。"))
 			blacksmith.mind.add_sleep_experience(attacked_item.anvilrepair, exp_gained/2) //We gain as much exp as we fix divided by 2
 			if(do_after(user, CLICK_CD_MELEE, target = attacked_object))
 				attack_obj(attacked_object, user)
 			return
 		else
-			user.visible_message(span_warning("[user] fumbles trying to repair [attacked_item]!"))
+			user.visible_message(span_warning("[user]试图修理[attacked_item]时失了手！"))
 			if(do_after(user, CLICK_CD_MELEE, target = attacked_object))
 				attack_obj(attacked_object, user)
 			return
@@ -163,14 +163,14 @@
 		if(!attacked_structure.hammer_repair || !attacked_structure.max_integrity)
 			return
 		if(blacksmith.get_skill_level(attacked_structure.hammer_repair) <= 0)
-			to_chat(user, span_warning("I don't know how to repair this.."))
+			to_chat(user, span_warning("我不知道该怎么修这个……"))
 			return
 		repair_percent *= blacksmith.get_skill_level(attacked_structure.hammer_repair) * attacked_structure.max_integrity
 		exp_gained = min(attacked_structure.obj_integrity + repair_percent, attacked_structure.max_integrity) - attacked_structure.obj_integrity
 		attacked_structure.obj_integrity = min(attacked_structure.obj_integrity + repair_percent, attacked_structure.max_integrity)
 		blacksmith.mind.add_sleep_experience(attacked_structure.hammer_repair, exp_gained) //We gain as much exp as we fix
 		playsound(src,'sound/items/bsmithfail.ogg', 100, FALSE)
-		user.visible_message(span_info("[user] repairs [attacked_structure]!"))
+		user.visible_message(span_info("[user]修好了[attacked_structure]！"))
 		if(attacked_object.obj_integrity <= attacked_object.max_integrity && do_after(user, CLICK_CD_MELEE, target = attacked_object))
 			attack_obj(attacked_object, user)
 		return
@@ -179,7 +179,7 @@
 
 /obj/item/rogueweapon/hammer/attack_hand(mob/living/user)
 	if(HAS_TRAIT(user, TRAIT_CURSE_MALUM))
-		to_chat(user, span_warning("Your cursed hands burn at the touch of the hammer!"))
+		to_chat(user, span_warning("你受诅咒的双手一碰到锤子就灼痛起来！"))
 		user.freak_out()
 		return
 	. = ..()
@@ -209,7 +209,7 @@
 		while(affecting.get_damage() != 0 || length(affecting.wounds))
 			var/used_time = 7 SECONDS
 			if(M == user)
-				to_chat(user, span_warning("Repairing myself is difficult..."))
+				to_chat(user, span_warning("给自己修理可真难……"))
 				used_time += 3 SECONDS //repairing yourself as a golem is logistically going to be a lot more difficult than someone else doing it for you
 			if(user.mind)
 				used_time -= (user.get_skill_level(/datum/skill/craft/engineering) * 10)
@@ -230,62 +230,62 @@
 			H.update_damage_overlays()
 
 			if(M == user)
-				user.visible_message(span_notice("[user] hammers [user.p_their()] [affecting.name]."), span_notice("I hammer my [affecting.name]."))
-			else
-				user.visible_message(span_notice("[user] hammers [M]'s [affecting.name]."), span_notice("I hammer [M]'s [affecting.name]."))
+				user.visible_message(span_notice("[user]敲打着[user.p_their()]的[affecting.name]。"), span_notice("我敲打着自己的[affecting.name]。"))
+			else
+				user.visible_message(span_notice("[user]敲打着[M]的[affecting.name]。"), span_notice("我敲打着[M]的[affecting.name]。"))
 		if(affecting.get_damage() == 0 && !length(affecting.wounds))//if the bodypart has no damage nor wounds on it...
 			if(M == user)
-				to_chat(user, span_warning("My [affecting.name] is undamaged."))
-			else
-				to_chat(user, span_warning("[M]'s [affecting.name] is undamaged."))
+				to_chat(user, span_warning("我的[affecting.name]没有受损。"))
+			else
+				to_chat(user, span_warning("[M]的[affecting.name]没有受损。"))
 			return
 	else //Non-construct.
-		to_chat(user, span_warning("I can't tinker on living flesh!"))
+		to_chat(user, span_warning("我没法对活生生的血肉搞这些修修补补！"))
 
 /obj/item/rogueweapon/hammer/wood	// wood hammer (mallet)
-	name = "wooden mallet"
-	desc = "A wooden mallet is an artificers second best friend! But it may also come in handy to a smith..."
+	name = "木槌"
+	desc = "木槌是工匠的第二好朋友！不过对铁匠来说，它有时也很好用……"
 	icon_state = "hammer_w"
 	force = 16
 
 /obj/item/rogueweapon/hammer/stone	// stone hammer
-	name = "stone hammer"
-	desc = "A makeshift hammer, made with a crudly chisled-down rock."
+	name = "石锤"
+	desc = "一把临时凑合出来的锤子，由一块粗糙凿削过的石头制成。"
 	icon_state = "hammer_r"
 	force = 18
 	max_integrity = 15
 
 /obj/item/rogueweapon/hammer/decrepit
-	name = "decrepit hammer"
-	desc = "A decrepit old hammer."
+	name = "衰朽锤"
+	desc = "一把衰朽的旧锤子。"
 	icon_state = "ahammer"
 	force = 12
 	max_integrity = 10
 	smeltresult = /obj/item/ingot/aaslag
 
 /obj/item/rogueweapon/hammer/copper
-	name = "copper hammer"
-	desc = "A copper hammer, slightly better than a stone hammer."
+	name = "铜锤"
+	desc = "一把铜锤，性能比石锤稍好一些。"
 	icon_state = "hammer_c"
 	force = 20
 	max_integrity = 100
 
 /obj/item/rogueweapon/hammer/iron	// iron hammer
-	name = "hammer"
-	desc = "Each strikes reverberate loudly chanting war!"
+	name = "锤"
+	desc = "每一次敲击都回荡出高声战歌！"
 	icon_state = "hammer_i"
 	smeltresult = /obj/item/ingot/iron
 
 /obj/item/rogueweapon/hammer/steel	// steel hammer
-	name = "claw hammer"
-	desc = "Steel to drive the iron nail without mercy."
+	name = "羊角锤"
+	desc = "以钢铁之力将铁钉毫不留情地钉入其中。"
 	icon_state = "hammer_s"
 	smeltresult = /obj/item/ingot/steel
 
 /obj/item/rogueweapon/hammer/blacksteel
 	force = 25
-	name = "blacksteel hammer"
-	desc = "A hammer made of blacksteel, to drive even the hardest metals into submission."
+	name = "黑钢锤"
+	desc = "一把由黑钢打造的锤子，足以让最坚硬的金属也低头屈服。"
 	icon = 'icons/roguetown/weapons/tools.dmi'
 	icon_state = "bs_masterhammer"
 	item_state = "bs_masterhammer"
@@ -310,13 +310,13 @@
 			var/repair_percent = 0.05
 			if(user.mind)
 				if(user.get_skill_level(I.hammer_repair) <= 0)
-					to_chat(user, span_warning("I don't know how to repair this.."))
+					to_chat(user, span_warning("我不知道该怎么修这个……"))
 					return
 				repair_percent = max(user.get_skill_level(I.hammer_repair) * 0.05, 0.05)
 			repair_percent = repair_percent * I.max_integrity
 			I.obj_integrity = min(obj_integrity+repair_percent, I.max_integrity)
 			playsound(src,'sound/items/bsmithfail.ogg', 100, FALSE)
-			user.visible_message(span_info("[user] repairs [I]!"))
+			user.visible_message(span_info("[user]修好了[I]！"))
 			return
 	..()
 */
@@ -335,8 +335,8 @@
 /obj/item/rogueweapon/tongs
 	force = 10
 	possible_item_intents = list(/datum/intent/mace/strike)
-	name = "tongs"
-	desc = "A pair of iron jaws used to carry hot ingots."
+	name = "钳子"
+	desc = "一对用于夹持高温金属锭的铁钳。"
 	icon_state = "tongs"
 	icon = 'icons/roguetown/weapons/tools.dmi'
 	sharpness = IS_BLUNT
@@ -354,7 +354,7 @@
 /obj/item/rogueweapon/tongs/examine(mob/user)
 	. = ..()
 	if(hott)
-		. += span_warning("The tip is hot to the touch.")
+		. += span_warning("尖端烫得不能碰。")
 
 /obj/item/rogueweapon/tongs/get_temperature()
 	if(hott)
@@ -409,7 +409,7 @@
 				return list("shrink" = 0.5,"sx" = -2,"sy" = -5,"nx" = 4,"ny" = -5,"wx" = 0,"wy" = -5,"ex" = 2,"ey" = -5,"nturn" = 0,"sturn" = 0,"wturn" = 0,"eturn" = 0,"nflip" = 0,"sflip" = 0,"wflip" = 0,"eflip" = 0,"northabove" = 0,"southabove" = 1,"eastabove" = 1,"westabove" = 0)
 
 /obj/item/rogueweapon/tongs/stone
-	name = "stone tongs"
+	name = "石制钳子"
 	icon_state = "stonetongs"
 	force = 5
 	smeltresult = null
@@ -426,7 +426,7 @@
 			icon_state = "stonetongsi0"
 
 /obj/item/rogueweapon/tongs/decrepit
-	name = "decrepit tongs"
+	name = "衰朽钳子"
 	icon_state = "atongs"
 	force = 5
 	smeltresult = null
@@ -443,8 +443,8 @@
 			icon_state = "atongsi0"
 
 /obj/item/rogueweapon/tongs/blacksteel
-	name = "blacksteel tongs"
-	desc = "A pair of blacksteel jaws almost certainly used as a sign of prestige."
+	name = "黑钢钳子"
+	desc = "一对黑钢钳口，几乎可以肯定更多是身份地位的象征。"
 	icon_state = "bs_tongs"
 	wdefense = 6
 	icon = 'icons/roguetown/weapons/tools.dmi'
