From 38bd7838b6f31921e2ee37f4827d855d9e986e7c Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Wed, 27 May 2026 20:07:25 +0800
Subject: [PATCH] 翻译 modular_deserttown/jobs/desertjobs.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 modular_deserttown/jobs/desertjobs.dm | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/modular_deserttown/jobs/desertjobs.dm b/modular_deserttown/jobs/desertjobs.dm
index 46c8df9813a0..4de84d54cc29 100644
--- a/modular_deserttown/jobs/desertjobs.dm
+++ b/modular_deserttown/jobs/desertjobs.dm
@@ -173,19 +173,19 @@
 		shirt = /obj/item/clothing/suit/roguetown/armor/gambeson/zyb
 
 /obj/effect/proc_holder/spell/self/convertrole/slave
-	name = "Recruit Slave"
+	name = "征召奴隶"
 	new_role = "Slave"
 	overlay_state = "recruit_servant"
 	recruitment_faction = "Servants"
-	recruitment_message = "Serve the crown, %RECRUIT!"
-	accept_message = "I OBEY, MASTER!"
-	refuse_message = "I refuse."
+	recruitment_message = "为王权效力吧，%RECRUIT！"
+	accept_message = "我服从，主人！"
+	refuse_message = "我拒绝。"
 	recharge_time = 100
 
 /obj/effect/proc_holder/spell/self/convertrole/azeb
-	name = "Recruit Azeb"
+	name = "征召阿泽布"
 	new_role = "Azeb"
 	recruitment_faction = "Bog Guard"
-	recruitment_message = "Serve the my will, %RECRUIT!"
-	accept_message = "FOR THE SULTAN!"
-	refuse_message = "I refuse."
+	recruitment_message = "遵奉我的意志，%RECRUIT！"
+	accept_message = "为了苏丹！"
+	refuse_message = "我拒绝。"
-- 
Translation Patch Toolkit
