From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Mon, 25 May 2026 12:43:47 +0800
Subject: [PATCH] 翻译 code/modules/mob/dead/observer/observer.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
diff --git a/code/modules/mob/dead/observer/observer.dm b/code/modules/mob/dead/observer/observer.dm
--- a/code/modules/mob/dead/observer/observer.dm
+++ b/code/modules/mob/dead/observer/observer.dm
@@ -104,7 +104,7 @@
 	if(!istype(src, /mob/dead/observer/rogue/arcaneeye))
 		if(!istype(src, /mob/dead/observer/screye))
 			client?.verbs += GLOB.ghost_verbs
-			to_chat(src, span_danger("Click the <b>SKULL</b> on the left of your HUD to respawn."))
+			to_chat(src, span_danger("点击 HUD 左侧的<b>骷髅</b>即可重生。"))
 
 	if(icon_state in GLOB.ghost_forms_with_directions_list)
 		ghostimage_default = image(src.icon,src,src.icon_state + "")
@@ -213,7 +213,7 @@
 		if(istype(src, /mob/dead/observer/screye))
 			return
 		client?.verbs += GLOB.ghost_verbs
-		to_chat(src, span_danger("Click the <b>SKULL</b> on the left of your HUD to respawn."))
+		to_chat(src, span_danger("点击 HUD 左侧的<b>骷髅</b>即可重生。"))
 
 /mob/dead/observer/narsie_act()
 	var/old_color = color
@@ -382,7 +382,7 @@
 			var/datum/antagonist/zombie/Z = mind.has_antag_datum(/datum/antagonist/zombie)
 			if(!Z.revived)
 				if(!(world.time % 5))
-					to_chat(src, span_warning("I'm preparing to walk again."))
+					to_chat(src, span_warning("我正准备再次行走于世。"))
 				return
 	return ..()
 
@@ -416,7 +416,7 @@
 	if(stat == DEAD)
 		ghostize(1)
 	else
-		var/response = alert(src, "Are you -sure- you want to ghost?\n(You are alive. If you ghost whilst still alive you may not play again this round! You can't change your mind so choose wisely!!)","Are you sure you want to ghost?","????","????")
+		var/response = alert(src, "你-确定-要化作幽灵吗？\n（你还活着。若在仍然活着时化鬼，你这轮可能就无法再游玩！你无法改变主意，所以请谨慎选择！！）","你确定要化作幽灵吗？","化作幽灵","留在体内")
 		if(response != "化作幽灵")
 			return	//didn't want to ghost after-all
 		ghostize(0)						//0 parameter is so we can never re-enter our body, "Charlie, you can never come baaaack~" :3
@@ -428,7 +428,7 @@
 	set hidden = 1
 	if(!usr.client.holder)
 		return
-	var/response = alert(src, "Are you -sure- you want to ghost?\n(You are alive. If you ghost whilst still alive you may not play again this round! You can't change your mind so choose wisely!!)","Are you sure you want to ghost?","????","????")
+	var/response = alert(src, "你-确定-要化作幽灵吗？\n（你还活着。若在仍然活着时化鬼，你这轮可能就无法再游玩！你无法改变主意，所以请谨慎选择！！）","你确定要化作幽灵吗？","化作幽灵","留在体内")
 	if(response != "化作幽灵")
 		return
 	ghostize(0)
@@ -456,18 +456,18 @@
 
 /mob/dead/observer/proc/reenter_corpse()
 	set category = "Ghost"
-	set name = "Re-enter Corpse"
+	set name = "返回尸体"
 	set hidden = 1
 	if(!client)
 		return
 	if(!mind || QDELETED(mind.current))
-		to_chat(src, span_warning("I have no body."))
+		to_chat(src, span_warning("我没有身体。"))
 		return
 	if(!can_reenter_corpse)
-		to_chat(src, span_warning("I cannot re-enter my body."))
+		to_chat(src, span_warning("我无法回到自己的身体里。"))
 		return
 	if(mind.current.key && copytext(mind.current.key,1,2)!="@")	//makes sure we don't accidentally kick any clients
-		to_chat(usr, span_warning("Another consciousness is in my body... It is resisting me."))
+		to_chat(usr, span_warning("另一道意识占据着我的身体……它在抗拒我。"))
 		return
 //	stop_all_loops()
 	SSdroning.kill_rain(src.client)
@@ -487,7 +487,7 @@
 	if (CONFIG_GET(flag/norespawn))
 		return
 	if ((stat != DEAD || !( SSticker )))
-		to_chat(src, span_boldnotice("I must be dead to use this!"))
+		to_chat(src, span_boldnotice("我必须死亡才能使用这个！"))
 		return
 
 //	if(mind?.current && (world.time < mind.current.timeofdeath + RESPAWNTIME))
@@ -502,7 +502,7 @@
 
 	log_game("[key_name(src)] used abandon mob.")
 
-	to_chat(src, span_info("Returned to lobby successfully."))
+	to_chat(src, span_info("已成功返回大厅。"))
 
 	if(!client)
 		log_game("[key_name(src)] AM failed due to disconnect.")
@@ -531,22 +531,22 @@
 
 /mob/dead/observer/verb/stay_dead()
 	set category = "Ghost"
-	set name = "Do Not Resuscitate"
+	set name = "拒绝复活"
 	set hidden = 1
 	if(!check_rights(R_WATCH))
 		return
 	if(!client)
 		return
 	if(!can_reenter_corpse)
-		to_chat(usr, span_warning("You're already stuck out of your body!"))
+		to_chat(usr, span_warning("你已经无法回到自己的身体里了！"))
 		return FALSE
 
-	var/response = alert(src, "Are you sure you want to prevent (almost) all means of resuscitation? This cannot be undone. ","Are you sure you want to stay dead?","????","???")
+	var/response = alert(src, "你确定要阻止（几乎）所有复活手段吗？此操作无法撤销。","你确定要保持死亡吗？","拒绝复活","救救我")
 	if(response != "拒绝复活")
 		return
 
 	can_reenter_corpse = FALSE
-	to_chat(src, span_boldnotice("I can no longer be brought back into your body."))
+	to_chat(src, span_boldnotice("我再也无法回到自己的身体里了。"))
 	return TRUE
 
 /mob/dead/observer/proc/notify_cloning(message, sound, atom/source, flashwindow = TRUE)
@@ -573,20 +573,20 @@
 
 /mob/dead/observer/proc/dead_tele()
 	set category = "Ghost"
-	set name = "Teleport"
-	set desc= "Teleport to a location"
+	set name = "传送"
+	set desc= "传送到一个地点"
 	set hidden = 1
 	if(!check_rights(R_WATCH))
 		return
 	if(!isobserver(usr))
-		to_chat(usr, span_warning("Not when you're not dead!"))
+		to_chat(usr, span_warning("你没死的时候不能这么做！"))
 		return
 	var/list/filtered = list()
 	for(var/V in GLOB.sortedAreas)
 		var/area/A = V
 		if(!A.hidden)
 			filtered += A
-	var/area/thearea  = input("Area to jump to", "BOOYEA") as null|anything in filtered
+	var/area/thearea  = input("选择要跳转的区域", "传送") as null|anything in filtered
 
 	if(!thearea)
 		return
@@ -596,7 +596,7 @@
 		L+=T
 
 	if(!L || !L.len)
-		to_chat(usr, span_warning("No area available."))
+		to_chat(usr, span_warning("没有可用区域。"))
 		return
 
 	usr.forceMove(pick(L))
@@ -604,7 +604,7 @@
 
 /mob/dead/observer/verb/follow()
 	set category = "Ghost"
-	set name = "Orbit" // "Haunt"
+	set name = "环绕" // "缠绕"
 	set desc = ""
 	set hidden = 1
 	var/list/all_mobs = getpois(mobs_only=1,skip_mindless=1)
@@ -623,7 +623,7 @@
 	else
 		allowed_mobs += all_mobs
 
-	var/input = input("Who?!", "Haunt", null, null) as null|anything in allowed_mobs
+	var/input = input("跟着谁？", "环绕", null, null) as null|anything in allowed_mobs
 	var/mob/target = allowed_mobs[input]
 
 	ManualFollow(target)
@@ -673,7 +673,7 @@
 
 /mob/dead/observer/verb/jumptomob() //Moves the ghost instead of just changing the ghosts's eye -Nodrak
 	set category = "Ghost"
-	set name = "Jump to Mob"
+	set name = "跳转到生物"
 	set desc = ""
 	set hidden = 1
 	if(!check_rights(R_WATCH))
@@ -685,7 +685,7 @@
 		var/target = null	   //Chosen target.
 
 		dest += getpois(mobs_only=1) //Fill list, prompt user with list
-		target = input("Please, select a player!", "Jump to Mob", null, null) as null|anything in dest
+		target = input("请选择一名玩家！", "跳转到生物", null, null) as null|anything in dest
 
 		if (!target)//Make sure we actually have a target
 			return
@@ -698,11 +698,11 @@
 				A.forceMove(T)
 				A.update_parallax_contents()
 			else
-				to_chat(A, span_danger("This mob is not located in the game world."))
+				to_chat(A, span_danger("这个生物不在游戏世界中。"))
 
 /mob/dead/observer/verb/change_view_range()
 	set category = "Ghost"
-	set name = "View Range"
+	set name = "视野范围"
 	set desc = ""
 	set hidden = 1
 	if(!check_rights(R_DEBUG))
@@ -712,14 +712,14 @@
 		var/list/views = list()
 		for(var/i in 7 to max_view)
 			views |= i
-		var/new_view = input("Choose your new view", "Modify view range", 7) as null|anything in views
+		var/new_view = input("选择你的新视野范围", "调整视野范围", 7) as null|anything in views
 		if(new_view)
 			client.change_view(CLAMP(new_view, 7, max_view))
 	else
 		client.change_view(CONFIG_GET(string/default_view))
 
 /mob/dead/observer/verb/add_view_range(input as num)
-	set name = "Add View Range"
+	set name = "增加视野范围"
 	set hidden = TRUE
 	var/max_view = client.prefs.unlock_content ? GHOST_MAX_VIEW_RANGE_MEMBER : GHOST_MAX_VIEW_RANGE_DEFAULT
 	if(input)
@@ -727,8 +727,8 @@
 
 /mob/dead/observer/verb/boo()
 	set category = "Ghost"
-	set name = "Boo!"
-	set desc= "Scare your crew members because of boredom!"
+	set name = "吓唬！"
+	set desc= "因为无聊，吓唬一下船员们！"
 
 	if(bootime > world.time)
 		return
@@ -743,23 +743,23 @@
 
 /mob/dead/observer/memory()
 	set hidden = 1
-	to_chat(src, span_danger("I are dead! You have no mind to store memory!"))
+	to_chat(src, span_danger("我已经死了！你没有心智可供记忆存储！"))
 
 /mob/dead/observer/add_memory()
 	set hidden = 1
-	to_chat(src, span_danger("I are dead! You have no mind to store memory!"))
+	to_chat(src, span_danger("我已经死了！你没有心智可供记忆存储！"))
 
 /mob/dead/observer/verb/toggle_ghostsee()
-	set name = "Toggle Ghost Vision"
+	set name = "切换幽灵视觉"
 	set desc = ""
 	set category = "Ghost"
 	set hidden = 1
 	ghostvision = !(ghostvision)
 	update_sight()
-	to_chat(usr, span_boldnotice("I [(ghostvision?"now":"no longer")] have ghost vision."))
+	to_chat(usr, span_boldnotice("我[(ghostvision?"现在":"不再")]拥有幽灵视觉。"))
 
 /mob/dead/observer/verb/toggle_darkness()
-	set name = "Toggle Darkness"
+	set name = "切换黑暗"
 	set category = "Ghost"
 	set hidden = 1
 	switch(lighting_alpha)
@@ -799,9 +799,9 @@
 		return
 	var/bt = world.time
 	SEND_SOUND(src, sound('sound/misc/notice (2).ogg'))
-	if(alert(src, "You have been summoned you to destroy the vale!", "Join the Horde", "Yes", "No") == "Yes")
+	if(alert(src, "你已被召唤去毁灭山谷！", "加入部落", "是", "否") == "是")
 		if(world.time > bt + 5 MINUTES)
-			to_chat(src, span_warning("Too late."))
+			to_chat(src, span_warning("太迟了。"))
 			return FALSE
 		returntolobby(RESPAWNTIME*-1)
 
@@ -828,8 +828,8 @@
 
 /mob/dead/observer/verb/possess()
 	set category = "Ghost"
-	set name = "Possess!"
-	set desc= "Take over the body of a mindless creature!"
+	set name = "附身！"
+	set desc= "接管一具无心智生物的身体！"
 
 	var/list/possessible = list()
 	for(var/mob/living/L in GLOB.alive_mob_list)
@@ -838,16 +838,16 @@
 		if(!(L in GLOB.player_list) && !L.mind)
 			possessible += L
 
-	var/mob/living/target = input("Your new life begins today!", "Possess Mob", null, null) as null|anything in sortNames(possessible)
+	var/mob/living/target = input("你的新生命从今天开始！", "附身生物", null, null) as null|anything in sortNames(possessible)
 
 	if(!target)
 		return FALSE
 
 	if(can_reenter_corpse && mind?.current)
-		if(alert(src, "Your soul is still tied to your former life as [mind.current.name], if you go forward there is no going back to that life. Are you sure you wish to continue?", "Move On", "Yes", "No") == "No")
+		if(alert(src, "你的灵魂仍与前世 [mind.current.name] 相连，如果继续前进，你将再也无法回到那段人生。你确定要继续吗？", "继续前行", "是", "否") == "否")
 			return FALSE
 	if(target.key)
-		to_chat(src, span_warning("Someone has taken this body while you were choosing!"))
+		to_chat(src, span_warning("在你选择的时候，已经有人占据了这具身体！"))
 		return FALSE
 
 	target.key = key
@@ -864,13 +864,13 @@
 	return TRUE
 
 /mob/dead/observer/verb/view_manifest()
-	set name = "View Crew Manifest"
+	set name = "查看船员清单"
 	set category = "Ghost"
 	set hidden = 1
 	if(!check_rights(R_WATCH))
 		return
 	var/dat
-	dat += "<h4>Crew Manifest</h4>"
+	dat += "<h4>船员清单</h4>"
 	dat += GLOB.data_core.get_manifest()
 
 	src << browse(dat, "window=manifest;size=387x420;can_close=1")
@@ -921,7 +921,7 @@
 		H.remove_hud_from(src)
 
 /mob/dead/observer/verb/toggle_data_huds()
-	set name = "Toggle Sec/Med/Diag HUD"
+	set name = "切换安保/医疗/诊断 HUD"
 	set desc = ""
 	set category = "Ghost"
 	set hidden = 1
@@ -929,45 +929,45 @@
 		return
 	if(data_huds_on) //remove old huds
 		remove_data_huds()
-		to_chat(src, span_notice("Data HUDs disabled."))
+		to_chat(src, span_notice("数据 HUD 已禁用。"))
 		data_huds_on = 0
 	else
 		show_data_huds()
-		to_chat(src, span_notice("Data HUDs enabled."))
+		to_chat(src, span_notice("数据 HUD 已启用。"))
 		data_huds_on = 1
 
 /mob/dead/observer/verb/toggle_health_scan()
-	set name = "Toggle Health Scan"
+	set name = "切换健康扫描"
 	set desc = ""
 	set category = "Ghost"
 	set hidden = 1
 	if(!check_rights(R_WATCH))
 		return
 	if(health_scan) //remove old huds
-		to_chat(src, span_notice("Health scan disabled."))
+		to_chat(src, span_notice("健康扫描已禁用。"))
 		health_scan = FALSE
 	else
-		to_chat(src, span_notice("Health scan enabled."))
+		to_chat(src, span_notice("健康扫描已启用。"))
 		health_scan = TRUE
 
 /mob/dead/observer/verb/toggle_gas_scan()
-	set name = "Toggle Gas Scan"
+	set name = "切换气体扫描"
 	set desc = ""
 	set category = "Ghost"
 	set hidden = 1
 	if(!check_rights(R_WATCH))
 		return
 	if(gas_scan)
-		to_chat(src, span_notice("Gas scan disabled."))
+		to_chat(src, span_notice("气体扫描已禁用。"))
 		gas_scan = FALSE
 	else
-		to_chat(src, span_notice("Gas scan enabled."))
+		to_chat(src, span_notice("气体扫描已启用。"))
 		gas_scan = TRUE
 
 /mob/dead/observer/verb/restore_ghost_appearance()
-	set name = "Restore Ghost Character"
-	set desc = "Sets your deadchat name and ghost appearance to your \
-		roundstart character."
+	set name = "恢复幽灵角色"
+	set desc = "将你的死者频道名称与幽灵外观恢复为 \
+		本轮开局角色。"
 	set category = "Ghost"
 	set hidden = 1
 	if(!check_rights(R_WATCH))
@@ -1033,7 +1033,7 @@
 			hud_used.show_hud(hud_used.hud_version)
 
 /mob/dead/observer/verb/observe()
-	set name = "Observe"
+	set name = "观察"
 	set category = "OOC"
 	set hidden = 1
 	if(!check_rights(R_WATCH))
@@ -1044,7 +1044,7 @@
 
 	var/eye_name = null
 
-	eye_name = input("Please, select a player!", "Observe", null, null) as null|anything in creatures
+	eye_name = input("请选择一名玩家！", "观察", null, null) as null|anything in creatures
 
 	if (!eye_name)
 		return
@@ -1067,7 +1067,7 @@
 /mob/dead/observer/examine(mob/user)
 	. = ..()
 	if(!invisibility)
-		. += "It seems extremely obvious."
+		. += "这看起来极其显眼。"
 
 /mob/dead/observer/proc/set_invisibility(value)
 	invisibility = value
@@ -1089,7 +1089,7 @@
 	GLOB.observer_default_invisibility = amount
 
 /mob/dead/observer/proc/open_spawners_menu()
-	set name = "Spawners Menu"
+	set name = "生成点菜单"
 	set desc = ""
 	set category = "Ghost"
 	set hidden = 1
@@ -1102,7 +1102,7 @@
 
 /mob/dead/observer/proc/tray_view()
 	set category = "Ghost"
-	set name = "T-ray view"
+	set name = "T 射线视图"
 	set desc = ""
 	set hidden = 1
 	if(!check_rights(R_WATCH))
-- 
Translation Patch Toolkit
