diff --git a/code/modules/roguetown/roguemachine/heartbeast/fleshcrafting/flesh_organs_limbs.dm b/code/modules/roguetown/roguemachine/heartbeast/fleshcrafting/flesh_organs_limbs.dm
--- a/code/modules/roguetown/roguemachine/heartbeast/fleshcrafting/flesh_organs_limbs.dm
+++ b/code/modules/roguetown/roguemachine/heartbeast/fleshcrafting/flesh_organs_limbs.dm
@@ -1,6 +1,6 @@
 /obj/item/flesh_node
 	icon = 'icons/obj/structures/heart_items.dmi'
 	icon_state = "flesh_node"
-	name = "flesh node"
-	desc = "A little, putrid fleshy nodule that can be further shaped into various fleshy items."
+	name = "????"
+	desc = "??????????????????????????"
 	w_class = WEIGHT_CLASS_SMALL
