diff --git a/code/modules/roguetown/roguemachine/heartbeast/fleshcrafting/flesh_rose.dm b/code/modules/roguetown/roguemachine/heartbeast/fleshcrafting/flesh_rose.dm
--- a/code/modules/roguetown/roguemachine/heartbeast/fleshcrafting/flesh_rose.dm
+++ b/code/modules/roguetown/roguemachine/heartbeast/fleshcrafting/flesh_rose.dm
@@ -1,10 +1,10 @@
 /obj/item/black_rose
 	icon = 'icons/mob/actions/pestraspells.dmi'
 	icon_state = "rot"
-	name = "black rose"
-	desc = "A sickly-sweet scented rose. Like honeyed rotten meat. It's dim petals seem laced with a black ooze that retreats away from your fingers."
+	name = "???"
+	desc = "?????????????????????????????????????????????"
 	w_class = WEIGHT_CLASS_SMALL
-	var/effect_desc = "You know this can be implanted with the cure rot miracle within a follower of Pestra. It protects her followers."
+	var/effect_desc = "????????????????????????????????"
 
 /obj/item/black_rose/examine(mob/user)
 	. = ..()
