diff --git a/code/modules/roguetown/roguejobs/tailor/loom.dm b/code/modules/roguetown/roguejobs/tailor/loom.dm
--- a/code/modules/roguetown/roguejobs/tailor/loom.dm
+++ b/code/modules/roguetown/roguejobs/tailor/loom.dm
@@ -1,7 +1,7 @@
 /obj/machinery/loom
 	icon = 'icons/roguetown/misc/structure.dmi'
-	name = "loom"
-	desc = "A wooden frame with taut threads ready to weave fabric."
+	name = "织布机"
+	desc = "一副绷紧了纱线、随时可以织布的木制框架。"
 	icon_state = "loom"
 	var/storedfiber = 0
 	var/maxfiber = 50
@@ -13,35 +13,35 @@
 		var/obj/item/natural/bundle/fibers/W = I
 		if(src.storedfiber + W.amount > src.maxfiber)
 			W.amount = (W.amount - (src.maxfiber - src.storedfiber))
-			to_chat(user, "You string some fiber onto [src].")
+			to_chat(user, "你把一些纤维挂到了[src]上。")
 			src.storedfiber = src.maxfiber
 			if(W.amount == 1)
 				new /obj/item/natural/fibers(get_turf(user))
 				qdel(W)
 		else
 			src.storedfiber = src.storedfiber + W.amount
-			to_chat(user, "You string some fiber onto [src].")
+			to_chat(user, "你把一些纤维挂到了[src]上。")
 			qdel(W)
 	if(istype(I, /obj/item/natural/fibers))
 		var/obj/item/natural/fibers/W = I
 		if(src.storedfiber < src.maxfiber)
 			src.storedfiber++
-			to_chat(user, "You string a fiber onto [src].")
+			to_chat(user, "你把一缕纤维挂到了[src]上。")
 			qdel(W)
 		else
-			to_chat(user, "You can't add any more fiber.")
+			to_chat(user, "你不能再加更多纤维了。")
 	. = ..()
 
 /obj/machinery/loom/attack_right(mob/user)
 	var/mob/living/L = user
 	if(isliving(user) && L.stat == CONSCIOUS && !user.get_active_held_item())
 		if(src.storedfiber > 0)
-			to_chat(user, "You remove a strand from [src].")
+			to_chat(user, "你从[src]上取下了一缕纤维。")
 			src.storedfiber--
 			var/obj/item/natural/fibers/F = new (src.loc)
 			L.put_in_hands(F)
 		else
-			to_chat(user, "There's nothing to take from [src].")
+			to_chat(user, "[src]上没有东西可取。")
 
 /obj/machinery/loom/attack_hand(mob/user)
 	var/mob/living/weaver = user
@@ -50,9 +50,9 @@
 	var/skill = weaver.get_skill_level(/datum/skill/craft/sewing)
 	if(isliving(user) && weaver.stat == CONSCIOUS)
 		if(src.storedfiber < 2)
-			to_chat(user, "You don't have enough fiber to do this.")
+			to_chat(user, "你的纤维不够，做不了这件事。")
 		else
-			to_chat(user, "You start weaving some cloth...")
+			to_chat(user, "你开始织布……")
 			while(src.storedfiber > 1)
 				if(!do_after(weaver, (weavetime - (skilltimemod*skill)),target = src) || src.storedfiber < 2)
 					break
@@ -61,5 +61,5 @@
 				weaver.mind.add_sleep_experience(/datum/skill/craft/sewing, (weaver.STAINT*0.5))//you get less exp from using the loom
 
 /obj/machinery/loom/examine(mob/user)
-	to_chat(user, span_notice("There are [storedfiber] strands of fiber strung on [src]."))
+	to_chat(user, span_notice("[src]上挂着 [storedfiber] 缕纤维。"))
 	. = ..()
