diff --git a/code/modules/jobs/job_types/roguetown/adventurer/types/antag/gnoll/gnoll_templar.dm b/code/modules/jobs/job_types/roguetown/adventurer/types/antag/gnoll/gnoll_templar.dm
--- a/code/modules/jobs/job_types/roguetown/adventurer/types/antag/gnoll/gnoll_templar.dm
+++ b/code/modules/jobs/job_types/roguetown/adventurer/types/antag/gnoll/gnoll_templar.dm
@@ -1,7 +1,7 @@
 /datum/advclass/gnoll/templar
-	name = "Gnoll Templar"
+	name = "豺狼人圣殿武士"
 	maximum_possible_slots = 1
-	tutorial = "None are as valued to protect graggarite worship as his gnoll champions themselves."
+	tutorial = "若论守护 Graggar 信仰，谁都不如他亲自拣选的豺狼人勇士更受器重。"
 	allowed_sexes = list(MALE, FEMALE)
 	allowed_races = list(/datum/species/gnoll)
 	outfit = /datum/outfit/job/roguetown/gnoll/templar
