diff --git a/code/game/objects/structures/fence.dm b/code/game/objects/structures/fence.dm
--- a/code/game/objects/structures/fence.dm
+++ b/code/game/objects/structures/fence.dm
@@ -11,7 +11,7 @@
 #define MAX_HOLE_SIZE LARGE_HOLE
 
 /obj/structure/fence
-	name = "fence"
+	name = "栅栏"
 	desc = ""
 	density = TRUE
 	anchored = TRUE
@@ -62,7 +62,7 @@
 			to_chat(user, span_warning("This section of the fence can't be cut!"))
 			return
 		if(invulnerable)
-			to_chat(user, span_warning("This fence is too strong to cut through!"))
+			to_chat(user, span_warning("这道栅栏太坚固了，切不开！"))
 			return
 		var/current_stage = hole_size
 		if(current_stage >= MAX_HOLE_SIZE)
