diff --git a/code/game/objects/structures/stairs.dm b/code/game/objects/structures/stairs.dm
--- a/code/game/objects/structures/stairs.dm
+++ b/code/game/objects/structures/stairs.dm
@@ -6,7 +6,7 @@
 // multiple stair objects can be chained together; the Z level transition will happen on the final stair object in the chain
 
 /obj/structure/stairs
-	name = "stairs"
+	name = "楼梯"
 	icon = 'icons/obj/stairs.dmi'
 	icon_state = "stairs"
 	anchored = TRUE
@@ -57,7 +57,7 @@
 	return FALSE
 
 /obj/structure/stairs/stone
-	name = "stone stairs"
+	name = "石阶"
 	icon = 'icons/obj/stairs.dmi'
 	icon_state = "stonestairs"
 	max_integrity = 600
