diff --git a/code/game/objects/structures/tables_racks.dm b/code/game/objects/structures/tables_racks.dm
--- a/code/game/objects/structures/tables_racks.dm
+++ b/code/game/objects/structures/tables_racks.dm
@@ -42,7 +42,7 @@
 
 /obj/structure/table/examine(mob/user)
 	. = ..()
-	. += span_info("Some structures can be used as hiding places. Toggle the 'SNEAK' button on your HUD, then click the structure to hide in it. You can stop hiding by clicking the structure again, or by moving out of it.")
+	. += span_info("有些结构可以用来躲藏。切换 HUD 上的“潜行”按钮，然后点击该结构就能躲进去。再次点击该结构，或直接从里面走出来，即可停止躲藏。")
 
 /obj/structure/table/proc/deconstruction_hints(mob/user)
 	return span_notice("The top is <b>screwed</b> on, but the main <b>bolts</b> are also visible.")
@@ -288,7 +288,7 @@
  */
 
 /obj/structure/table/wood
-	name = "wooden table"
+	name = "木桌"
 	desc = ""
 	icon = 'icons/roguetown/misc/tables.dmi'
 	icon_state = "tablewood"
@@ -565,7 +565,7 @@
  * Racks
  */
 /obj/structure/rack
-	name = "rack"
+	name = "架子"
 	desc = ""
 	icon = 'icons/obj/objects.dmi'
 	icon_state = "rack"
