diff --git a/code/game/objects/structures/walldeco.dm b/code/game/objects/structures/walldeco.dm
--- a/code/game/objects/structures/walldeco.dm
+++ b/code/game/objects/structures/walldeco.dm
@@ -214,8 +214,8 @@
 	..()
 
 /obj/structure/fluff/walldeco/customflag
-	name = "Rotwood Vale flag"
-	desc = "A banner flutters in the breeze in the proud heraldic colors of the Duchy."
+	name = "谷地旗帜"
+	desc = "一面旗帜在风中飘扬，展示着这座公国引以为傲的纹章配色。"
 	icon_state = "wallflag"
 
 /obj/structure/fluff/walldeco/customflag/Initialize(mapload)
