--- a/code/modules/roguetown/roguemachine/crier.dm
+++ b/code/modules/roguetown/roguemachine/crier.dm
@@ -3,8 +3,8 @@
 #define TAB_MANAGESCOMS 3
 
 /obj/structure/roguemachine/crier
-	name = "rous master"
-	desc = "The crier's most trusted friend."
+	name = "鼠群主控"
+	desc = "报信人最可信赖的朋友。"
 	icon = 'icons/roguetown/misc/structure.dmi'
 	icon_state = "crier_machine"
 	density = TRUE
@@ -25,7 +25,7 @@
 			playsound(loc, 'sound/misc/beep.ogg', 100, FALSE, -1)
 			return
 		else
-			to_chat(user, span_warning("Wrong key."))
+			to_chat(user, span_warning("钥匙不对。"))
 			return
 	if(istype(P, /obj/item/storage/keyring))
 		var/obj/item/storage/keyring/K = P
@@ -37,7 +37,7 @@
 				locked = !locked
 				playsound(loc, 'sound/misc/beep.ogg', 100, FALSE, -1)
 				return
-		to_chat(user, span_warning("Wrong key."))
+		to_chat(user, span_warning("钥匙不对。"))
 		return
 
 /obj/structure/roguemachine/crier/Topic(href, href_list)
@@ -53,7 +53,7 @@
 			to_chat(usr, span_notice("You [H.is_locked ? "lock" : "unlock"] [H]."))
 	if(href_list["withdraw"])
 		if(total_payments <= 0)
-			to_chat(usr, span_warning("No mammon to withdraw."))
+			to_chat(usr, span_warning("没有可提取的玛门。"))
 			return
 
 		var/amount = total_payments
@@ -65,7 +65,7 @@
 		while(amount >= 1)
 			new /obj/item/roguecoin/copper(get_turf(src))
 			amount -= 1
-		to_chat(usr, span_notice("You withdraw the broadcaster payments."))
+		to_chat(usr, span_notice("你取出了广播器收入。"))
 
 	return attack_hand(usr)
 
@@ -74,7 +74,7 @@
 	if(.)
 		return
 	if(locked)
-		to_chat(user, span_warning("It's locked. Of course."))
+		to_chat(user, span_warning("锁着呢，当然。"))
 		return
 	user.changeNext_move(CLICK_CD_INTENTCAP)
 	playsound(loc, 'sound/misc/keyboard_enter.ogg', 100, FALSE, -1)
@@ -82,12 +82,12 @@
 	var/contents
 	switch(current_tab)
 		if(TAB_ROUSMAIN)
-			contents += "<center>ROUS MASTER<BR>"
-			contents += "Total stored mammon: [total_payments]<br><br>"
-			contents += "<a href='?src=\ref[src];withdraw=1'>Withdraw All</a><br>"
+			contents += "<center>鼠群主控<BR>"
+			contents += "当前存储玛门：[total_payments]<br><br>"
+			contents += "<a href='?src=\ref[src];withdraw=1'>全部取出</a><br>"
 			contents += "--------------<BR>"
-			contents += "<a href='?src=\ref[src];switchtab=[TAB_SCOMLOG]'>\[Broadcast Log\]</a><BR>"
-			contents += "<a href='?src=\ref[src];switchtab=[TAB_MANAGESCOMS]'>\[Manage Broadcasters\]</a><BR>"
+			contents += "<a href='?src=\ref[src];switchtab=[TAB_SCOMLOG]'>\[广播记录\]</a><BR>"
+			contents += "<a href='?src=\ref[src];switchtab=[TAB_MANAGESCOMS]'>\[管理广播器\]</a><BR>"
 			contents += "</center>"
 
 		if(TAB_SCOMLOG)
