diff --git a/code/modules/roguetown/roguemachine/heartbeast/heart_status_effects.dm b/code/modules/roguetown/roguemachine/heartbeast/heart_status_effects.dm
--- a/code/modules/roguetown/roguemachine/heartbeast/heart_status_effects.dm
+++ b/code/modules/roguetown/roguemachine/heartbeast/heart_status_effects.dm
@@ -1,8 +1,8 @@
 #define TERRITORIAL_FILTER "territorial_rage"
 
 /atom/movable/screen/alert/status_effect/territorial_rage
-	name = "Territorial Rage"
-	desc = "The heartbeast is tearing you apart!"
+	name = "????"
+	desc = "???????????"
 
 /datum/status_effect/territorial_rage
 	id = "territorial_rage"
@@ -20,7 +20,7 @@
 	owner.Immobilize(duration)
 	owner.add_filter(TERRITORIAL_FILTER, 2, list("type" = "outline", "color" = outline_color, "alpha" = 210, "size" = 2))
 	owner.update_icon()
-	to_chat(owner, span_userdanger("The heartbeast's tendrils grab you and start pulling you apart!"))
+	to_chat(owner, span_userdanger("??????????????????????"))
 
 	animate(owner, pixel_y = owner.pixel_y + 8, time = 0.5 SECONDS, easing = SINE_EASING)
 
@@ -46,7 +46,7 @@
 		// Lower back down
 		animate(L, pixel_y = L.pixel_y - 8, time = 0.5 SECONDS, easing = SINE_EASING)
 	
-	owner.visible_message(span_danger("The tendrils release [owner]!"))
+	owner.visible_message(span_danger("?????[owner]?"))
 
 /datum/status_effect/territorial_rage/proc/remove_limb(mob/living/carbon/C)
 	if(limb_removed)
@@ -82,7 +82,7 @@
 		to_remove = right_leg
 	
 	if(to_remove)
-		C.visible_message(span_userdanger("[C]'s [to_remove.name] is torn off by the tendrils!"))
+		C.visible_message(span_userdanger("[C]?[to_remove.name]??????????"))
 		to_remove.dismember()
 
 		var/obj/effect/temp_visual/dir_setting/bloodsplatter/splatter = new(get_turf(C), pick(GLOB.cardinals))
@@ -92,7 +92,7 @@
 /datum/status_effect/territorial_rage/proc/remove_head(mob/living/carbon/C)
 	var/obj/item/bodypart/head = C.get_bodypart(BODY_ZONE_HEAD)
 	if(head)
-		C.visible_message(span_userdanger("[C]'s head is violently torn off by the tendrils!"))
+		C.visible_message(span_userdanger("[C]?????????????"))
 		head.dismember()
 		C.adjustBruteLoss(200)
 		var/obj/effect/temp_visual/dir_setting/bloodsplatter/splatter = new(get_turf(C), pick(GLOB.cardinals))
