diff --git a/code/modules/roguetown/roguejobs/cook/tools/skewer.dm b/code/modules/roguetown/roguejobs/cook/tools/skewer.dm
--- a/code/modules/roguetown/roguejobs/cook/tools/skewer.dm
+++ b/code/modules/roguetown/roguejobs/cook/tools/skewer.dm
@@ -1,8 +1,8 @@
 /obj/item/cooking/skewer
 	force = 15
 	possible_item_intents = list(INTENT_GENERIC)
-	name = "skewer"
-	desc = "A sturdy skewer, smeared with remnants of previous meals."
+	name = "烤签"
+	desc = "一根结实的烤签，上面还沾着先前餐食留下的残渣。"
 	icon_state = "skewer"
 	icon = 'icons/roguetown/items/cooking.dmi'
 	item_state = "rods"
