diff --git a/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/czwarteki/czwartekiheir.dm b/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/czwarteki/czwartekiheir.dm
--- a/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/czwarteki/czwartekiheir.dm
+++ b/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/czwarteki/czwartekiheir.dm
@@ -1,6 +1,6 @@
 /datum/advclass/czwarteki/heir
-	name = "Czwarteki Heir"
-	tutorial = "You are the Czwarteki Lords Heir. Or perhaps one of many. Brought with you by your Parent to march forth on this venture. And to gain experience in the realms beyond your home."
+	name = "Czwarteki 继承人"
+	tutorial = "你是 Czwarteki 领主的继承人。也许只是众多继承人之一。你的父母带着你一同踏上这场远行，让你在故土之外的诸国中历练见识。"
 	outfit = /datum/outfit/job/roguetown/czwarteki/heir
 	traits_applied = list(TRAIT_NOBLE, TRAIT_MEDIUMARMOR, TRAIT_STEELHEARTED)
 	category_tags = list(CTAG_CZWAR_HEIR)
