diff --git a/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/czwarteki/czwartekiservant.dm b/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/czwarteki/czwartekiservant.dm
--- a/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/czwarteki/czwartekiservant.dm
+++ b/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/czwarteki/czwartekiservant.dm
@@ -1,6 +1,6 @@
 /datum/advclass/czwarteki/servant
-	name = "Czwarteki Servant"
-	tutorial = "You are Servants of your Lord. Taken along upon the Journey through the Vale with the Retinue. Your only goals are but to ensure your Lord and his Heir's well being upon the trip."
+	name = "Czwarteki 仆役"
+	tutorial = "你是领主身边的仆役，随扈从一同踏上穿越谷地的旅程。你唯一的职责，就是确保领主及其继承人在这趟途中安然无恙。"
 	outfit = /datum/outfit/job/roguetown/czwarteki/servant
 	traits_applied = list(TRAIT_SLEUTH, TRAIT_KEENEARS, TRAIT_CICERONE, TRAIT_HOMESTEAD_EXPERT)
 	category_tags = list(CTAG_CZWAR_SERVANT)
