From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Sat, 30 May 2026 13:30:00 +0800
Subject: [PATCH] 翻译 code/modules/mob/camera/camera.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
diff --git a/code/modules/mob/camera/camera.dm b/code/modules/mob/camera/camera.dm
--- a/code/modules/mob/camera/camera.dm
+++ b/code/modules/mob/camera/camera.dm
@@ -1,7 +1,7 @@
 // Camera mob, used by AI camera and blob.
 
 /mob/camera
-	name = "camera mob"
+	name = "摄像机实体"
 	density = FALSE
 	move_force = INFINITY
 	move_resist = INFINITY
-- 
Translation Patch Toolkit
