From d2ec20a27fd6f1d5ff08933413acb4077069e66c Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Sat, 30 May 2026 13:00:00 +0800
Subject: [PATCH] 翻译 code/modules/mob/dead/new_player/hair_gradients.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

diff --git a/code/modules/mob/dead/new_player/hair_gradients.dm b/code/modules/mob/dead/new_player/hair_gradients.dm
index b6141b1c53e7..b3b50a5d811a 100644
--- a/code/modules/mob/dead/new_player/hair_gradients.dm
+++ b/code/modules/mob/dead/new_player/hair_gradients.dm
@@ -5,58 +5,58 @@
 	var/icon_state
 
 /datum/hair_gradient/none
-	name = "None"
+	name = "无"
 	icon = null
 	icon_state = null
 
 /datum/hair_gradient/fadeup
-	name = "Fade Up"
+	name = "向上渐隐"
 	icon_state = "fadeup"
 
 /datum/hair_gradient/fadedown
-	name = "Fade Down"
+	name = "向下渐隐"
 	icon_state = "fadedown"
 
 /datum/hair_gradient/vertical_split
-	name = "Vertical Split"
+	name = "垂直分割"
 	icon_state = "vsplit"
 
 /datum/hair_gradient/_split
-	name = "Horizontal Split"
+	name = "水平分割"
 	icon_state = "bottomflat"
 
 /datum/hair_gradient/reflected
-	name = "Reflected"
+	name = "镜像"
 	icon_state = "reflected_high"
 
 /datum/hair_gradient/reflected_inverse
-	name = "Reflected Inverse"
+	name = "反向镜像"
 	icon_state = "reflected_inverse_high"
 
 /datum/hair_gradient/wavy
-	name = "Wavy"
+	name = "波浪"
 	icon_state = "wavy"
 
 /datum/hair_gradient/long_fade_up
-	name = "Long Fade Up"
+	name = "长向上渐隐"
 	icon_state = "long_fade_up"
 
 /datum/hair_gradient/long_fade_down
-	name = "Long Fade Down"
+	name = "长向下渐隐"
 	icon_state = "long_fade_down"
 
 /datum/hair_gradient/short_fade_up
-	name = "Short Fade Up"
+	name = "短向上渐隐"
 	icon_state = "short_fade_up"
 
 /datum/hair_gradient/short_fade_down
-	name = "Short Fade Down"
+	name = "短向下渐隐"
 	icon_state = "short_fade_down"
 
 /datum/hair_gradient/wavy_spike
-	name = "Spiked Wavy"
+	name = "尖刺波浪"
 	icon_state = "wavy_spiked"
 
 /datum/hair_gradient/streaks
-	name = "Streaked"
+	name = "条纹"
 	icon_state = "streaks"
-- 
Translation Patch Toolkit
