diff --git a/code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/tools.dm b/code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/tools.dm
--- a/code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/tools.dm
+++ b/code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/tools.dm
@@ -45,29 +45,29 @@
 
 // --------- Copper -----------
 /datum/anvil_recipe/tools/copper/sickle
-	name = "Sickle, Copper (+1 Stick)"
+	name = "镰刀, 铜 (+1 木棍)"
 	additional_items = list(/obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/sickle/copper
 	i_type = "Tools"
 
 /datum/anvil_recipe/tools/copper/pick
-	name = "Pick, Copper (+1 Stick)"
+	name = "镐, 铜 (+1 木棍)"
 	additional_items = list(/obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/pick/copper
 	i_type = "Tools"
 
 /datum/anvil_recipe/tools/copper/pitchfork
-	name = "Pitchfork, Copper (+2 Sticks)"
+	name = "干草叉, 铜 (+2 木棍s)"
 	additional_items = list(/obj/item/grown/log/tree/stick, /obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/pitchfork/copper
 	i_type = "Tools"
 
 /datum/anvil_recipe/tools/copper/lamptern
-	name = "Lamptern, Copper"
+	name = "提灯, 铜"
 	created_item = /obj/item/flashlight/flare/torch/lantern/copper
 
 /datum/anvil_recipe/tools/copper/hammer
-	name = "Hammer, Copper (+Stick)"
+	name = "锤, 铜 (+木棍)"
 	additional_items = list(/obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/hammer/copper
 	i_type = "Tools"
@@ -76,94 +76,94 @@
 // --------- ANCIENT ALLOY -----------
 
 /datum/anvil_recipe/tools/decrepit/thresher
-	name = "Thresher, Decrepit (+1 Stick)"
+	name = "脱粒连枷, 衰朽 (+1 木棍)"
 	additional_items = list(/obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/thresher/decrepit
 	i_type = "Tools"
 
 /datum/anvil_recipe/tools/decrepit/hoe
-	name = "Hoe, Decrepit (+2 Sticks)"
+	name = "锄头, 衰朽 (+2 木棍s)"
 	additional_items = list(/obj/item/grown/log/tree/stick, /obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/hoe/decrepit
 	i_type = "Tools"
 
 /datum/anvil_recipe/tools/decrepit/pitchfork
-	name = "Pitchfork, Decrepit (+2 Sticks)"
+	name = "干草叉, 衰朽 (+2 木棍s)"
 	additional_items = list(/obj/item/grown/log/tree/stick, /obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/pitchfork/decrepit
 	i_type = "Tools"
 
 /datum/anvil_recipe/tools/decrepit/hammer
-	name = "Hammer, Decrepit (+1 Stick)"
+	name = "锤, 衰朽 (+1 木棍)"
 	additional_items = list(/obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/hammer/decrepit
 	i_type = "Tools"
 
 /datum/anvil_recipe/tools/decrepit/sickle
-	name = "Sickle, Decrepit (+1 Stick)"
+	name = "镰刀, 衰朽 (+1 木棍)"
 	additional_items = list(/obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/sickle/decrepit
 	i_type = "Tools"
 
 
 /datum/anvil_recipe/tools/decrepit/tongs
-	name = "Tongs, Decrepit"
+	name = "钳子, 衰朽"
 	created_item = /obj/item/rogueweapon/tongs/decrepit
 	i_type = "Tools"
 
 /datum/anvil_recipe/tools/decrepit/pick
-	name = "Pickaxe, Decrepit (+1 Stick)"
+	name = "镐, 衰朽 (+1 木棍)"
 	additional_items = list(/obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/pick/decrepit
 	i_type = "Tools"
 
 /datum/anvil_recipe/tools/decrepit/shovel
-	name = "Shovel, Decrepit (+2 Sticks)"
+	name = "铲, 衰朽 (+2 木棍s)"
 	additional_items = list(/obj/item/grown/log/tree/stick, /obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/shovel/decrepit
 	i_type = "Tools"
 
 /datum/anvil_recipe/tools/decrepit/sewingneedle
-	name = "Needles, Decrepit (x3)"
+	name = "针, 衰朽 (x3)"
 	created_item = /obj/item/needle/decrepit
 	createditem_num = 3
 
 /datum/anvil_recipe/tools/decrepit/pan
-	name = "Frypan, Decrepit"
+	name = "煎锅, 衰朽"
 	created_item = /obj/item/cooking/pan/decrepit
 
 /datum/anvil_recipe/tools/decrepit/agobs
-	name = "Goblets x2"
+	name = "高脚杯 x2"
 	created_item = /obj/item/reagent_containers/glass/cup/decrepitgob
 	createditem_num = 2
 
 /datum/anvil_recipe/tools/decrepit/amugs
-	name = "Mug, Decrepit (x3)"
+	name = "马克杯, 衰朽 (x3)"
 	created_item = /obj/item/reagent_containers/glass/cup/decrepitmug
 	createditem_num = 3
 
 
 /datum/anvil_recipe/tools/decrepit/pot
-	name = "Cooking Pot, Decrepit"
+	name = "炊锅, 衰朽"
 	created_item = /obj/item/reagent_containers/glass/bucket/pot/decrepit
 
 
 /datum/anvil_recipe/tools/decrepit/platter
-	name = "Platter, Decrepit (x3)"
+	name = "大盘, 衰朽 (x3)"
 	created_item = /obj/item/cooking/platter/decrepit
 	createditem_num = 3
 
 /datum/anvil_recipe/tools/decrepit/bowl
-	name = "Bowl, Decrepit (x2)"
+	name = "碗, 衰朽 (x2)"
 	created_item = /obj/item/reagent_containers/glass/bowl/decrepit
 
 /datum/anvil_recipe/tools/decrepit/fork
-	name = "Fork, Decrepit (x3)"
+	name = "叉, 衰朽 (x3)"
 	created_item = /obj/item/kitchen/fork/decrepit
 	createditem_num = 3
 
 /datum/anvil_recipe/tools/decrepit/spoon
-	name = "Spoon, Decrepit (x3)"
+	name = "勺, 衰朽 (x3)"
 	created_item = /obj/item/kitchen/spoon/decrepit
 	createditem_num = 3
 
@@ -171,22 +171,22 @@
 // --------- IRON -----------
 
 /datum/anvil_recipe/tools/iron/blowrod
-	name = "Glass Blowing Rod"
+	name = "吹制杆"
 	created_item = /obj/item/rogueweapon/blowrod
 
 /datum/anvil_recipe/tools/iron/surgerytools
-	name = "Surgeon's Bag (+1 Iron, +1 Cured Leather)"
+	name = "外科包 (+1 铁, +1 熟皮)"
 	additional_items = list(/obj/item/ingot/iron, /obj/item/natural/hide/cured)
 	created_item = /obj/item/storage/belt/rogue/surgery_bag/full
 
 /datum/anvil_recipe/tools/iron/torch
-	name = "Fieftorches (x5) (+1 Coal)"
+	name = "Fieftorches (x5) (+1 煤炭)"
 	additional_items = list(/obj/item/rogueore/coal)
 	created_item = /obj/item/flashlight/flare/torch/metal
 	createditem_num = 5
 	
 /datum/anvil_recipe/tools/iron/pan
-	name = "Frypan, Iron"
+	name = "煎锅, 铁"
 	created_item = /obj/item/cooking/pan
 
 /datum/anvil_recipe/tools/iron/keyring
@@ -195,18 +195,18 @@
 	createditem_num = 3
 
 /datum/anvil_recipe/tools/iron/sewingneedle
-	name = "Needles, Iron (x3)"
+	name = "针, 铁 (x3)"
 	created_item = /obj/item/needle
 	createditem_num = 3 // They can be refilled with fiber now
 
 /* Movning under Engineering
 /datum/anvil_recipe/tools/iron/lockpicks
-	name = "Lockpicks x3"
+	name = "开锁器 x3"
 	created_item = /obj/item/lockpick
 	createditem_num = 3
 
 /datum/anvil_recipe/tools/iron/lockpickring
-	name = "Lockpickrings x3"
+	name = "开锁环 x3"
 	created_item = /obj/item/lockpickring
 	createditem_num = 3
 */
@@ -217,102 +217,102 @@
 	created_item = /obj/item/rogueweapon/surgery/cautery/branding
 
 /datum/anvil_recipe/tools/iron/shovel
-	name = "Shovel, Iron (+2 Sticks)"
+	name = "铲, 铁 (+2 木棍s)"
 	additional_items = list(/obj/item/grown/log/tree/stick, /obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/shovel
 	i_type = "Tools"
 
 /datum/anvil_recipe/tools/iron/hammer
-	name = "Hammer, Iron (+1 Stick)"
+	name = "锤, 铁 (+1 木棍)"
 	additional_items = list(/obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/hammer/iron
 	i_type = "Tools"
 
 /datum/anvil_recipe/tools/iron/handsaw
-	name = "Handsaw, Iron (+1 Stick)"
+	name = "手锯, 铁 (+1 木棍)"
 	additional_items = list(/obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/handsaw
 
 /datum/anvil_recipe/tools/iron/chisel
-	name = "Chisel, Iron"
+	name = "凿子, 铁"
 	created_item = /obj/item/rogueweapon/chisel
 
 /datum/anvil_recipe/tools/iron/tongs
-	name = "Tongs, Iron"
+	name = "钳子, 铁"
 	created_item = /obj/item/rogueweapon/tongs
 	i_type = "Tools"
 
 /datum/anvil_recipe/tools/iron/sickle
-	name = "Sickle, Iron (+1 Stick)"
+	name = "镰刀, 铁 (+1 木棍)"
 	additional_items = list(/obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/sickle
 	i_type = "Tools"
 
 /datum/anvil_recipe/tools/iron/pick
-	name = "Pickaxe, Iron (+1 Stick)"
+	name = "镐, 铁 (+1 木棍)"
 	additional_items = list(/obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/pick
 	i_type = "Tools"
 
 /datum/anvil_recipe/tools/iron/hoe
-	name = "Hoe, Iron (+2 Sticks)"
+	name = "锄头, 铁 (+2 木棍s)"
 	additional_items = list(/obj/item/grown/log/tree/stick, /obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/hoe
 	i_type = "Tools"
 
 /datum/anvil_recipe/tools/iron/pitchfork
-	name = "Pitchfork, Iron (+2 Sticks)"
+	name = "干草叉, 铁 (+2 木棍s)"
 	additional_items = list(/obj/item/grown/log/tree/stick, /obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/pitchfork
 	i_type = "Tools"
 
 /datum/anvil_recipe/tools/iron/lamptern
-	name = "Lampterns, Iron (x3)"
+	name = "提灯, 铁 (x3)"
 	created_item = /obj/item/flashlight/flare/torch/lantern
 	createditem_num = 3
 
 /datum/anvil_recipe/tools/iron/cups
-	name = "Cups, Iron (x3)"
+	name = "杯, 铁 (x3)"
 	created_item = /obj/item/reagent_containers/glass/cup
 	createditem_num = 3
 
 /datum/anvil_recipe/tools/iron/thresher
-	name = "Thresher, Iron (+1 Stick)"
+	name = "脱粒连枷, 铁 (+1 木棍)"
 	additional_items = list(/obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/thresher
 	i_type = "Tools"
 
 /datum/anvil_recipe/tools/iron/headhook
-	name = "Headhook, Iron (+2 Fibers)"
+	name = "头钩, 铁 (+2 纤维s)"
 	additional_items = list(/obj/item/natural/fibers, /obj/item/natural/fibers)
 	created_item = /obj/item/storage/hip/headhook
 	i_type = "Tools"
 
 /datum/anvil_recipe/tools/iron/scissors
-	name = "Scissors"
+	name = "剪刀"
 	created_item = /obj/item/rogueweapon/huntingknife/scissors
 	i_type = "Tools"
 
 // --------- Steel -----------
 
 /datum/anvil_recipe/tools/steel/hammer
-	name = "Claw Hammer (+1 Stick)"
+	name = "羊角锤 (+1 木棍)"
 	additional_items = list(/obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/hammer/steel
 
 /datum/anvil_recipe/tools/steel/pick
-	name = "Pickaxe, Steel (+1 Stick)"
+	name = "镐, 钢 (+1 木棍)"
 	additional_items = list(/obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/pick/steel
 	i_type = "Tools"
 
 /datum/anvil_recipe/tools/steel/cups
-	name = "Goblets x2"
+	name = "高脚杯 x2"
 	created_item = /obj/item/reagent_containers/glass/cup/steel
 	createditem_num = 2
 
 /datum/anvil_recipe/tools/steel/chefknife
-	name = "Chef's Knife"
+	name = "厨刀"
 	created_item = /obj/item/rogueweapon/huntingknife/chefknife
 
 /datum/anvil_recipe/tools/steel/cleaver
@@ -320,117 +320,117 @@
 	created_item = /obj/item/rogueweapon/huntingknife/cleaver
 
 /datum/anvil_recipe/tools/steel/scissors
-	name = "Scissors"
+	name = "剪刀"
 	created_item = /obj/item/rogueweapon/huntingknife/scissors/steel
 	i_type = "Tools"
 
 // --------- SILVER -----------
 
 /datum/anvil_recipe/tools/silver/cups
-	name = "Goblets x2"
+	name = "高脚杯 x2"
 	created_item = /obj/item/reagent_containers/glass/cup/silver
 	createditem_num = 2
 
 /datum/anvil_recipe/tools/silver/cups/small
-	name = "Cups x3"
+	name = "杯 x3"
 	created_item = /obj/item/reagent_containers/glass/cup/silver/small
 	createditem_num = 3
 
 /datum/anvil_recipe/tools/silver/shovel
-	name = "Shovel, Silver (+1 Silver, +1 Small Log)"
+	name = "铲, 白银 (+1 白银, +1 小原木)"
 	additional_items = list(/obj/item/ingot/silver, /obj/item/grown/log/tree/small)
 	created_item = /obj/item/rogueweapon/shovel/silver
 
 // --------- GOLD RECIPES-----------
 
 /datum/anvil_recipe/tools/gold/cups
-	name = "Goblets x2"
+	name = "高脚杯 x2"
 	created_item = /obj/item/reagent_containers/glass/cup/golden
 	createditem_num = 2
 
 /datum/anvil_recipe/tools/gold/cups/small
-	name = "Cups x3"
+	name = "杯 x3"
 	created_item = /obj/item/reagent_containers/glass/cup/golden/small
 	createditem_num = 3
 
 // --------- TIN -----------
 
 /datum/anvil_recipe/tools/tin/cups
-	name = "Goblets x2"
+	name = "高脚杯 x2"
 	created_item = /obj/item/reagent_containers/glass/cup/tin
 	createditem_num = 2
 
 /datum/anvil_recipe/tools/tin/cups/small
-	name = "Cups x3"
+	name = "杯 x3"
 	created_item = /obj/item/reagent_containers/glass/cup/tin/small
 	createditem_num = 3
 
 
 // --------- COOKING RECIPES -----------
 /datum/anvil_recipe/tools/iron/pot
-	name = "Cooking Pot, Iron"
+	name = "炊锅, 铁"
 	created_item = /obj/item/reagent_containers/glass/bucket/pot
 
 /datum/anvil_recipe/tools/iron/kettle
-	name = "Cooking Kettle, Iron"
+	name = "炊壶, 铁"
 	created_item = /obj/item/reagent_containers/glass/bucket/pot/kettle
 
 /datum/anvil_recipe/tools/copper/pot
-	name = "Cooking Pot, Copper"
+	name = "炊锅, 铜"
 	created_item = /obj/item/reagent_containers/glass/bucket/pot/copper
 
 /datum/anvil_recipe/tools/copper/platter
-	name = "Platter, Copper (x2)"
+	name = "大盘, 铜 (x2)"
 	created_item = /obj/item/cooking/platter/copper
 	createditem_num = 2
 
 /datum/anvil_recipe/tools/tin/platter
-	name = "Platter, Tin (x2)"
+	name = "大盘, Tin (x2)"
 	created_item = /obj/item/cooking/platter/pewter
 	createditem_num = 2
 
 /datum/anvil_recipe/tools/gold/platter
-	name = "Platter, Gold (x2)"
+	name = "大盘, 黄金 (x2)"
 	created_item = /obj/item/cooking/platter/gold
 	createditem_num = 2
 
 /datum/anvil_recipe/tools/silver/platter
-	name = "Platter, Silver (x2)"
+	name = "大盘, 白银 (x2)"
 	created_item = /obj/item/cooking/platter/silver
 	createditem_num = 2
 
 /datum/anvil_recipe/tools/iron/spoon
-	name = "Spoon, Iron (x3)"
+	name = "勺, 铁 (x3)"
 	created_item = /obj/item/kitchen/spoon/iron
 	createditem_num = 3
 
 /datum/anvil_recipe/tools/tin/spoon
-	name = "Spoon, Tin (x3)"
+	name = "勺, Tin (x3)"
 	created_item = /obj/item/kitchen/spoon/tin
 	createditem_num = 3
 
 /datum/anvil_recipe/tools/iron/fork
-	name = "Fork, Iron (x3)"
+	name = "叉, 铁 (x3)"
 	created_item = /obj/item/kitchen/fork/iron
 	createditem_num = 3
 
 /datum/anvil_recipe/tools/tin/fork
-	name = "Fork, Tin (x3)"
+	name = "叉, Tin (x3)"
 	created_item = /obj/item/kitchen/fork/tin
 	createditem_num = 3
 
 /datum/anvil_recipe/tools/silver/fork
-	name = "Fork, Silver (x3)"
+	name = "叉, 白银 (x3)"
 	created_item = /obj/item/kitchen/fork/silver
 	createditem_num = 3
 
 /datum/anvil_recipe/tools/gold/fork
-	name = "Fork, gold (x3)"
+	name = "叉, gold (x3)"
 	created_item = /obj/item/kitchen/fork/gold
 	createditem_num = 3
 
 /datum/anvil_recipe/tools/decrepit/fork
-	name = "Fork, decrepit (x3)"
+	name = "叉, decrepit (x3)"
 	created_item = /obj/item/kitchen/fork/decrepit
 	createditem_num = 3
 
@@ -443,7 +443,7 @@
 // --------- CASTING TOOLS -----------
 
 /datum/anvil_recipe/tools/crucible
-	name = "Crucible"
+	name = "坩埚"
 	req_bar = /obj/item/ingot/steel
 	additional_items = list(/obj/item/ingot/steel)
 	created_item = /obj/item/reagent_containers/glass/crucible
@@ -451,49 +451,49 @@
 	i_type = "Casting"
 
 /datum/anvil_recipe/tools/sprue_funnel
-	name = "Sprue and Funnel"
+	name = "浇口与漏斗"
 	req_bar = /obj/item/ingot/steel
 	created_item = /obj/item/sprue_funnel
 	craftdiff = SKILL_LEVEL_MASTER
 	i_type = "Casting"
 
 /datum/anvil_recipe/tools/mold_axe
-	name = "Axe Blade Mold"
+	name = "斧 Blade Mold"
 	req_bar = /obj/item/ingot/steel
 	created_item = /obj/item/mold/axe
 	craftdiff = SKILL_LEVEL_MASTER
 	i_type = "Casting"
 
 /datum/anvil_recipe/tools/mold_sword
-	name = "Sword Blade Mold"
+	name = "剑 Blade Mold"
 	req_bar = /obj/item/ingot/steel
 	created_item = /obj/item/mold/sword
 	craftdiff = SKILL_LEVEL_MASTER
 	i_type = "Casting"
 
 /datum/anvil_recipe/tools/mold_knife
-	name = "Knife Blade Mold"
+	name = "小刀 Blade Mold"
 	req_bar = /obj/item/ingot/steel
 	created_item = /obj/item/mold/knife
 	craftdiff = SKILL_LEVEL_MASTER
 	i_type = "Casting"
 
 /datum/anvil_recipe/tools/mold_mace
-	name = "Mace Head Mold"
+	name = "钉锤 Head Mold"
 	req_bar = /obj/item/ingot/steel
 	created_item = /obj/item/mold/mace
 	craftdiff = SKILL_LEVEL_MASTER
 	i_type = "Casting"
 
 /datum/anvil_recipe/tools/mold_polearm
-	name = "Polearm Blade Mold"
+	name = "长柄刃模具"
 	req_bar = /obj/item/ingot/steel
 	created_item = /obj/item/mold/polearm
 	craftdiff = SKILL_LEVEL_MASTER
 	i_type = "Casting"
 
 /datum/anvil_recipe/tools/mold_plate
-	name = "Plate Mold"
+	name = "甲片模具"
 	req_bar = /obj/item/ingot/steel
 	created_item = /obj/item/mold/plate
 	craftdiff = SKILL_LEVEL_MASTER
@@ -502,17 +502,17 @@
 //black steel tools
 
 /datum/anvil_recipe/tools/blacksteel/hammer
-	name = "Blacksteel hammer (+1 Stick)"
+	name = "黑钢 hammer (+1 木棍)"
 	additional_items = list(/obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/hammer/blacksteel
 
 /datum/anvil_recipe/tools/blacksteel/pick
-	name = "Blacksteel Pickaxe (+1 Stick)"
+	name = "黑钢 镐 (+1 木棍)"
 	additional_items = list(/obj/item/grown/log/tree/stick)
 	created_item = /obj/item/rogueweapon/pick/blacksteel
 
 /datum/anvil_recipe/tools/blacksteel/tongs
-	name = "Blacksteel Tongs"
+	name = "黑钢 钳子"
 	created_item = /obj/item/rogueweapon/tongs/blacksteel
 	
 // --------- HEARTBEAST TOOLS -----------
