diff --git a/code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/valuables.dm b/code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/valuables.dm
--- a/code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/valuables.dm
+++ b/code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/valuables.dm
@@ -5,19 +5,19 @@
 	i_type = "Valuables"
 
 /datum/anvil_recipe/valuables/gold
-	name = "Statue, Gold"
+	name = "雕像, 黄金"
 	req_bar = /obj/item/ingot/gold
 	created_item = /obj/item/roguestatue/gold
 	craftdiff = SKILL_LEVEL_EXPERT
 
 /datum/anvil_recipe/valuables/silver
-	name = "Statue, Silver"
+	name = "雕像, 白银"
 	req_bar = /obj/item/ingot/silver
 	created_item = /obj/item/roguestatue/silver
 	craftdiff = SKILL_LEVEL_EXPERT
 
 /datum/anvil_recipe/valuables/iron
-	name = "Statue, Iron"
+	name = "雕像, 铁"
 	req_bar = /obj/item/ingot/iron
 	created_item = /obj/item/roguestatue/iron
 
@@ -27,17 +27,17 @@
 	created_item = /obj/item/roguestatue/decrepit
 
 /datum/anvil_recipe/valuables/steel
-	name = "Statue, Steel"
+	name = "雕像, 钢"
 	req_bar = /obj/item/ingot/steel
 	created_item = /obj/item/roguestatue/steel
 
 /datum/anvil_recipe/valuables/blacksteel
-	name = "Statue, Blacksteel"
+	name = "雕像, 黑钢"
 	req_bar = /obj/item/ingot/blacksteel
 	created_item = /obj/item/roguestatue/blacksteel
 /*
 /datum/anvil_recipe/valuables/eargol
-	name = "gold earrings"
+	name = "gold 耳环"
 	req_bar = /obj/item/ingot/gold
 	created_item = list(/obj/item/rogueacc/eargold,
 						/obj/item/rogueacc/eargold,
@@ -45,7 +45,7 @@
 	type = "Valuables"
 
 /datum/anvil_recipe/valuables/earsil
-	name = "silver earrings"
+	name = "silver 耳环"
 	req_bar = /obj/item/ingot/silver
 	created_item = list(/obj/item/rogueacc/earsilver,
 						/obj/item/rogueacc/earsilver,
@@ -53,67 +53,67 @@
 //	i_type = "Valuables"
 
 /datum/anvil_recipe/valuables/ringg
-	name = "Rings, Gold (x3)"
+	name = "戒指, 黄金 (x3)"
 	req_bar = /obj/item/ingot/gold
 	created_item = /obj/item/clothing/ring/gold
 	craftdiff = SKILL_LEVEL_EXPERT
 	createditem_num = 3
 
 /datum/anvil_recipe/valuables/ringa
-	name = "Rings, Decrepit (x3)"
+	name = "戒指, 衰朽 (x3)"
 	req_bar = /obj/item/ingot/decrepit
 	created_item = /obj/item/clothing/ring/decrepit
 	createditem_num = 3
 
 /datum/anvil_recipe/valuables/rings
-	name = "Rings, Silver (x3)"
+	name = "戒指, 白银 (x3)"
 	req_bar = /obj/item/ingot/silver
 	created_item = /obj/item/clothing/ring/silver
 	craftdiff = SKILL_LEVEL_EXPERT
 	createditem_num = 3
 
 /datum/anvil_recipe/valuables/ringbs
-	name = "Rings, Blacksteel (x3)"
+	name = "戒指, 黑钢 (x3)"
 	req_bar = /obj/item/ingot/blacksteel
 	created_item = /obj/item/clothing/ring/blacksteel
 	createditem_num = 3
 
 /datum/anvil_recipe/valuables/ornateamulet
-	name = "Ornate Amulet"
+	name = "华饰护符"
 	req_bar = /obj/item/ingot/gold
 	created_item = /obj/item/clothing/neck/roguetown/ornateamulet
 	craftdiff = SKILL_LEVEL_EXPERT
 
 /datum/anvil_recipe/valuables/skullamulet
-	name = "Skull Amulet"
+	name = "颅骨护符"
 	req_bar = /obj/item/ingot/gold
 	created_item = /obj/item/clothing/neck/roguetown/skullamulet
 	craftdiff = SKILL_LEVEL_EXPERT
 
 //Gold Rings
 /datum/anvil_recipe/valuables/emeringg
-	name = "Gemerald Ring, Gold (+1 Gemerald)"
+	name = "Gemerald 戒指, 黄金 (+1 Gemerald)"
 	req_bar = /obj/item/ingot/gold
 	additional_items = list(/obj/item/roguegem/green)
 	craftdiff = SKILL_LEVEL_EXPERT
 	created_item = /obj/item/clothing/ring/emerald
 
 /datum/anvil_recipe/valuables/rubyg
-	name = "Rontz Ring, Gold (+1 Rontz)"
+	name = "Rontz 戒指, 黄金 (+1 Rontz)"
 	req_bar = /obj/item/ingot/gold
 	craftdiff = SKILL_LEVEL_EXPERT
 	additional_items = list(/obj/item/roguegem/ruby)
 	created_item = /obj/item/clothing/ring/ruby
 
 /datum/anvil_recipe/valuables/topazg
-	name = "Toper Ring, Gold (+1 Toper)"
+	name = "Toper 戒指, 黄金 (+1 Toper)"
 	req_bar = /obj/item/ingot/gold
 	craftdiff = SKILL_LEVEL_EXPERT
 	additional_items = list(/obj/item/roguegem/yellow)
 	created_item = /obj/item/clothing/ring/topaz
 
 /datum/anvil_recipe/valuables/quartzg
-	name = "Blortz Ring, Gold (+1 Blortz)"
+	name = "Blortz 戒指, 黄金 (+1 Blortz)"
 	req_bar = /obj/item/ingot/gold
 	craftdiff = SKILL_LEVEL_EXPERT
 	additional_items = list(/obj/item/roguegem/blue)
@@ -121,33 +121,33 @@
 	i_type = "Valuables"
 
 /datum/anvil_recipe/valuables/sapphireg
-	name = "Saffira Ring, Gold (+1 Saffira)"
+	name = "Saffira 戒指, 黄金 (+1 Saffira)"
 	req_bar = /obj/item/ingot/gold
 	craftdiff = SKILL_LEVEL_EXPERT
 	additional_items = list(/obj/item/roguegem/violet)
 	created_item = /obj/item/clothing/ring/sapphire
 
 /datum/anvil_recipe/valuables/diamondg
-	name = "Dorpel Ring, Gold (+1 Dorpel)"
+	name = "Dorpel 戒指, 黄金 (+1 Dorpel)"
 	req_bar = /obj/item/ingot/gold
 	craftdiff = SKILL_LEVEL_EXPERT
 	additional_items = list(/obj/item/roguegem/diamond)
 	created_item = /obj/item/clothing/ring/diamond
 
 /datum/anvil_recipe/valuables/signet
-	name = "Signet Ring"
+	name = "印戒"
 	req_bar = /obj/item/ingot/gold
 	craftdiff = SKILL_LEVEL_EXPERT
 	created_item = /obj/item/clothing/ring/signet
 
 /datum/anvil_recipe/valuables/signet/silver
-	name = "Blessed Silver Signet Ring"
+	name = "祝福白银印戒"
 	craftdiff = SKILL_LEVEL_MASTER
 	req_bar = /obj/item/ingot/silverblessed
 	created_item = /obj/item/clothing/ring/signet/silver
 
 /datum/anvil_recipe/valuables/signet/silver/inq
-	name = "Blessed Silver Signet Ring"
+	name = "祝福白银印戒"
 	craftdiff = SKILL_LEVEL_MASTER
 	req_bar = /obj/item/ingot/silverblessed/bullion
 	created_item = /obj/item/clothing/ring/signet/silver
@@ -155,49 +155,49 @@
 // Silver ingots are now in play, and as such, the steel rings have been converted to silver with their value adjusted accordingly. -Kyogon
 
 /datum/anvil_recipe/valuables/emerings
-	name = "Gemerald Ring, Silver (+1 Gemerald)"
+	name = "Gemerald 戒指, 白银 (+1 Gemerald)"
 	req_bar = /obj/item/ingot/silver
 	craftdiff = SKILL_LEVEL_MASTER
 	additional_items = list(/obj/item/roguegem/green)
 	created_item = /obj/item/clothing/ring/emeralds
 
 /datum/anvil_recipe/valuables/rubys
-	name = "Rontz Ring, Silver (+1 Rontz)"
+	name = "Rontz 戒指, 白银 (+1 Rontz)"
 	req_bar = /obj/item/ingot/silver
 	craftdiff = SKILL_LEVEL_MASTER
 	additional_items = list(/obj/item/roguegem/ruby)
 	created_item = /obj/item/clothing/ring/rubys
 
 /datum/anvil_recipe/valuables/topazs
-	name = "Toper Ring, Silver (+1 Toper)"
+	name = "Toper 戒指, 白银 (+1 Toper)"
 	req_bar = /obj/item/ingot/silver
 	craftdiff = SKILL_LEVEL_MASTER
 	additional_items = list(/obj/item/roguegem/yellow)
 	created_item = /obj/item/clothing/ring/topazs
 
 /datum/anvil_recipe/valuables/quartzs
-	name = "Blortz Ring, Silver (+1 Blortz)"
+	name = "Blortz 戒指, 白银 (+1 Blortz)"
 	req_bar = /obj/item/ingot/silver
 	craftdiff = SKILL_LEVEL_MASTER
 	additional_items = list(/obj/item/roguegem/blue)
 	created_item = /obj/item/clothing/ring/quartzs
 
 /datum/anvil_recipe/valuables/sapphires
-	name = "Saffira Ring, Silver (+1 Saffira)"
+	name = "Saffira 戒指, 白银 (+1 Saffira)"
 	req_bar = /obj/item/ingot/silver
 	craftdiff = SKILL_LEVEL_MASTER
 	additional_items = list(/obj/item/roguegem/violet)
 	created_item = /obj/item/clothing/ring/sapphires
 
 /datum/anvil_recipe/valuables/diamonds
-	name = "Dorpel Ring, Silver (+1 Dorpel)"
+	name = "Dorpel 戒指, 白银 (+1 Dorpel)"
 	req_bar = /obj/item/ingot/silver
 	craftdiff = SKILL_LEVEL_MASTER
 	additional_items = list(/obj/item/roguegem/diamond)
 	created_item = /obj/item/clothing/ring/diamonds
 
 /datum/anvil_recipe/valuables/terminus
-	name = "Terminus Est (+1 Gold Bar, +1 Steel, +1 Rontz)"
+	name = "特米努斯之终 (+1 黄金 Bar, +1 钢, +1 Rontz)"
 	req_bar = /obj/item/ingot/gold
 	additional_items = list(/obj/item/ingot/gold, /obj/item/ingot/steel, /obj/item/roguegem/ruby)
 	created_item = /obj/item/rogueweapon/sword/long/exe/cloth
@@ -206,7 +206,7 @@
 	i_type = "Weapons"
 
 /datum/anvil_recipe/valuables/dragon
-	name = "Dragonstone Ring (Secret!)"
+	name = "龙石戒指 (秘密！)"
 	req_bar = /obj/item/ingot/blacksteel
 	hides_from_books = TRUE
 	additional_items = list(/obj/item/ingot/gold, /obj/item/roguegem/blue, /obj/item/roguegem/violet, /obj/item/clothing/neck/roguetown/psicross/silver)
@@ -214,51 +214,51 @@
 	craftdiff = SKILL_LEVEL_LEGENDARY
 
 /datum/anvil_recipe/valuables/zcross_iron
-	name = "Inverted Psycross"
+	name = "逆十字灵印"
 	req_bar = /obj/item/ingot/iron
 	created_item = /obj/item/clothing/neck/roguetown/psicross/inhumen
 	craftdiff = 1
 
 //blacksteel Rings
 /datum/anvil_recipe/valuables/emeringbs
-	name = "Gemerald Ring, Blacksteel (+1 Gemerald)"
+	name = "Gemerald 戒指, 黑钢 (+1 Gemerald)"
 	req_bar = /obj/item/ingot/blacksteel
 	additional_items = list(/obj/item/roguegem/green)
 	created_item = /obj/item/clothing/ring/emeraldbs
 
 /datum/anvil_recipe/valuables/rubybs
-	name = "Rontz Ring, Blacksteel (+1 Rontz)"
+	name = "Rontz 戒指, 黑钢 (+1 Rontz)"
 	req_bar = /obj/item/ingot/blacksteel
 	additional_items = list(/obj/item/roguegem/ruby)
 	created_item = /obj/item/clothing/ring/rubybs
 
 /datum/anvil_recipe/valuables/topazbs
-	name = "Toper Ring, Blacksteel (+1 Toper)"
+	name = "Toper 戒指, 黑钢 (+1 Toper)"
 	req_bar = /obj/item/ingot/blacksteel
 	additional_items = list(/obj/item/roguegem/yellow)
 	created_item = /obj/item/clothing/ring/topazbs
 
 /datum/anvil_recipe/valuables/quartzbs
-	name = "Blortz Ring, Blacksteel (+1 Blortz)"
+	name = "Blortz 戒指, 黑钢 (+1 Blortz)"
 	req_bar = /obj/item/ingot/blacksteel
 	additional_items = list(/obj/item/roguegem/blue)
 	created_item = /obj/item/clothing/ring/quartzbs
 	i_type = "Valuables"
 
 /datum/anvil_recipe/valuables/sapphirebs
-	name = "Saffira Ring, Blacksteel (+1 Saffira)"
+	name = "Saffira 戒指, 黑钢 (+1 Saffira)"
 	req_bar = /obj/item/ingot/blacksteel
 	additional_items = list(/obj/item/roguegem/violet)
 	created_item = /obj/item/clothing/ring/sapphirebs
 
 /datum/anvil_recipe/valuables/diamondbs
-	name = "Dorpel Ring, Blacksteel (+1 Dorpel)"
+	name = "Dorpel 戒指, 黑钢 (+1 Dorpel)"
 	req_bar = /obj/item/ingot/blacksteel
 	additional_items = list(/obj/item/roguegem/diamond)
 	created_item = /obj/item/clothing/ring/diamondbs
 
 /datum/anvil_recipe/valuables/hope
-	name = "Ring Of Omnipotence (Secret!)"
+	name = "全能之戒 (秘密！)"
 	req_bar = /obj/item/ingot/silver
 	hides_from_books = TRUE
 	additional_items = list(/obj/item/clothing/ring/statgemerald, /obj/item/clothing/ring/statonyx, /obj/item/clothing/ring/statamythortz, /obj/item/clothing/ring/statrontz)
@@ -266,7 +266,7 @@
 	craftdiff = SKILL_LEVEL_LEGENDARY
 
 /datum/anvil_recipe/valuables/daemonslayer
-	name = "Daemonslayer (Secret!)"
+	name = "斩魔者 (秘密！)"
 	req_bar = /obj/item/ingot/silver
 	hides_from_books = TRUE
 	additional_items = list(/obj/item/ingot/silver, /obj/item/ingot/silver, /obj/item/ingot/silver, /obj/item/ingot/silver/, /obj/item/ingot/draconic, /obj/item/ingot/weeping, /obj/item/riddleofsteel, /obj/item/grown/log/tree)
