diff --git a/code/modules/roguetown/roguejobs/mages/rituals/_ritual.dm b/code/modules/roguetown/roguejobs/mages/rituals/_ritual.dm
--- a/code/modules/roguetown/roguejobs/mages/rituals/_ritual.dm
+++ b/code/modules/roguetown/roguejobs/mages/rituals/_ritual.dm
@@ -132,7 +132,7 @@
 	abstract_type = /datum/runeritual
 	var/name
 	var/desc
-	var/category = "Other"
+	var/category = "其他"
 	var/list/required_atoms = list()
 	var/list/result_atoms = list()
 	var/list/banned_atom_types = list()
@@ -145,7 +145,7 @@
 
 /datum/runeritual/proc/generate_html(mob/user)
 	var/client/client = user
-	var/tool = tier >= 2 ? "Arcyne Silver Dagger" : "Arcyne Chalk"
+	var/tool = tier >= 2 ? "奥术银匕首" : "奥术粉笔"
 	if(!istype(client))
 		client = user.client
 	user << browse_rsc('html/book.png')
@@ -159,23 +159,23 @@
 		  <div>
 		    <h1>[name]</h1>
 		    <div>
-			  <h2>Complexity Tier: [tier] </h2>
+			  <h2>复杂度等级：[tier]</h2>
 			  <br>
-			  <h2>Requirements</h2>
+			  <h2>需求</h2>
 			  <br>
 		"}
 	if(length(required_atoms))
-		html += "<strong>Items Required</strong><br>"
+		html += "<strong>所需物品</strong><br>"
 		for(var/atom/path as anything in required_atoms)
 			var/count = required_atoms[path]
 			if(ispath(path, /datum/reagent))
 				var/datum/reagent/R = path
-				html += "- [FLOOR(count, 1)] [UNIT_FORM_STRING(FLOOR(count, 1))] of [initial(R.name)]<br>"
+				html += "- [FLOOR(count, 1)] [UNIT_FORM_STRING(FLOOR(count, 1))] [initial(R.name)]<br>"
 			else
-				html += "- [count] counts of [initial(path.name)]<br>"
-
-	html += "<h1>Steps</h1>"
-	html += "To start any ritual draw the required rune with [tool], then supply with the above items."
+				html += "- [count] 个[initial(path.name)]<br>"
+
+	html += "<h1>步骤</h1>"
+	html += "开始任意仪式时，请先用[tool]绘制所需符文，再备齐上述物品。"
 	html += {"
 		</div>
 		</div>
@@ -195,10 +195,10 @@
 /datum/runeritual/proc/parse_required_item(atom/item_path, number_of_things)
 	// If we need a human, there is a high likelihood we actually need a (dead) body
 	if(ispath(item_path, /mob/living/carbon/human))
-		return "bod[number_of_things > 1 ? "ies" : "y"]"
+		return "尸体"
 	if(ispath(item_path, /mob/living))
-		return "carcass[number_of_things > 1 ? "es" : ""] of any kind"
-	return "[initial(item_path.name)]\s"
+		return "任意种类的尸骸"
+	return "[initial(item_path.name)]"
 
 /**
  * Called after on_finished_recipe returns TRUE
