diff --git a/code/modules/roguetown/roguecrafting/carpentry.dm b/code/modules/roguetown/roguecrafting/carpentry.dm
--- a/code/modules/roguetown/roguecrafting/carpentry.dm
+++ b/code/modules/roguetown/roguecrafting/carpentry.dm
@@ -1,5 +1,5 @@
 /datum/crafting_recipe/roguetown/survival/wood_stake
-	name = "stake"
+	name = "木桩"
 	result = /obj/item/grown/log/tree/stake
 	reqs = list(
 		/obj/item/grown/log/tree/stick = 1,
