diff --git a/code/modules/roguetown/roguecrafting/sewing.dm b/code/modules/roguetown/roguecrafting/sewing.dm
--- a/code/modules/roguetown/roguecrafting/sewing.dm
+++ b/code/modules/roguetown/roguecrafting/sewing.dm
@@ -69,7 +69,7 @@
 	craftdiff = 0
 
 /datum/crafting_recipe/roguetown/sewing/necramask
-	name = "death mask, reassembled"
+	name = "死者面具，重新组装"
 	result = list(/obj/item/clothing/head/roguetown/necramask)
 	reqs = list(/obj/item/clothing/head/roguetown/necrahood = 1,
 				/obj/item/natural/bone = 1)
@@ -292,7 +292,7 @@
 	craftdiff = 2
 
 /datum/crafting_recipe/roguetown/sewing/necramask
-	name = "death mask"
+	name = "死者面具"
 	result = list(/obj/item/clothing/head/roguetown/necramask)
 	reqs = list(/obj/item/natural/cloth = 2,
 				/obj/item/natural/fibers = 1,
