diff --git a/code/modules/roguetown/roguecrafting/tallow.dm b/code/modules/roguetown/roguecrafting/tallow.dm
--- a/code/modules/roguetown/roguecrafting/tallow.dm
+++ b/code/modules/roguetown/roguecrafting/tallow.dm
@@ -7,7 +7,7 @@
 	skillcraft = /datum/skill/craft/cooking
 
 /datum/crafting_recipe/roguetown/tallow/hearth_tallow
-	name = "render tallow over the hearth (x4) (1 fat)"
+	name = "在壁炉上熬炼兽脂（x4）（1 份脂肪）"
 	result = list(
 				/obj/item/reagent_containers/food/snacks/tallow,
 				/obj/item/reagent_containers/food/snacks/tallow,
@@ -23,7 +23,7 @@
 	verbage = "renders"
 
 /datum/crafting_recipe/roguetown/tallow/campfire_tallow
-	name = "render tallow over the campfire (x3) (1 fat)"
+	name = "在营火上熬炼兽脂（x3）（1 份脂肪）"
 	result = list(
 				/obj/item/reagent_containers/food/snacks/tallow,
 				/obj/item/reagent_containers/food/snacks/tallow,
@@ -38,7 +38,7 @@
 	verbage = "renders"
 
 /datum/crafting_recipe/roguetown/cooking/mobilestove_tallow
-	name = "render tallow over the mobile stove (x3) (1 fat)"
+	name = "在便携炉上熬炼兽脂（x3）（1 份脂肪）"
 	result = list(
 				/obj/item/reagent_containers/food/snacks/tallow,
 				/obj/item/reagent_containers/food/snacks/tallow,
