diff --git a/code/modules/jobs/job_types/roguetown/mercenaries/classes/condottiero.dm b/code/modules/jobs/job_types/roguetown/mercenaries/classes/condottiero.dm
--- a/code/modules/jobs/job_types/roguetown/mercenaries/classes/condottiero.dm
+++ b/code/modules/jobs/job_types/roguetown/mercenaries/classes/condottiero.dm
@@ -1,6 +1,6 @@
 /datum/advclass/mercenary/condottiero
-	name = "Condottiero Guildsman"
-	tutorial = "Hailing from the Kingdom of Etrusca, Condottieri are soldiers of fortune oft found in the employ of powerful merchants. Without a war to fight back home, the Grand Duchy has become a much-needed source of contracts. Afterall, this land is rich in coin yet poor in manpower..."
+	name = "孔多蒂耶罗佣兵"
+	tutorial = "你来自 Etrusca 王国。孔多蒂耶罗是典型的职业佣兵，常受雇于势力雄厚的商人。故乡眼下无大战可打，这片大公国便成了亟需的新契约来源。毕竟，这里钱财丰厚，却偏偏缺人卖命……"
 	allowed_sexes = list(MALE, FEMALE)
 	allowed_races = RACES_ALL_KINDS
 	outfit = /datum/outfit/job/roguetown/mercenary/condottiero
