diff --git a/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/heartfelt/retinue/heartfeltphyscian.dm b/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/heartfelt/retinue/heartfeltphyscian.dm
--- a/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/heartfelt/retinue/heartfeltphyscian.dm
+++ b/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/heartfelt/retinue/heartfeltphyscian.dm
@@ -1,8 +1,8 @@
 
 /datum/advclass/heartfelt/retinue/physician
-	name = "Heartfeltian Physician"
-	tutorial = "You are the Physician of Heartfelt, once celebrated for your steady hands and healing wisdom, yet powerless to save your barony from its grim fate. \
-	Haunted by those you could not protect, you ascend toward the Vale-seeking redemption, renewed purpose, and perhaps a cure for the wounds the world has inflicted."
+	name = "Heartfelt 医师"
+	tutorial = "你是 Heartfelt 的医师，曾因沉稳的双手与治愈之智而备受赞誉，却依旧无力拯救男爵领的悲惨命运。\
+	那些未能救下的人始终萦绕心头，你正前往谷地，寻求救赎、崭新的使命，也许还有治愈这世界伤口的方法。"
 	allowed_sexes = list(MALE, FEMALE)
 	allowed_races = RACES_NO_CONSTRUCT
 	outfit = /datum/outfit/job/roguetown/heartfelt/retinue/physician
