diff --git a/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/heartfelt/retinue/heartfeltprior.dm b/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/heartfelt/retinue/heartfeltprior.dm
--- a/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/heartfelt/retinue/heartfeltprior.dm
+++ b/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/heartfelt/retinue/heartfeltprior.dm
@@ -1,8 +1,8 @@
 
 /datum/advclass/heartfelt/retinue/prior
-	name = "Heartfeltian Prior"
-	tutorial = "The Prior of Heartfelt, you were destined for ascension within the Church, but fate intervened with the barony's downfall, \
-	delaying it indefinitely. Still guided by the blessings of Astrata, you journey to the Vale, determined to offer what aid and solace you can."
+	name = "Heartfelt 修院长"
+	tutorial = "你是 Heartfelt 的修院长，本该在教会中步步高升，却因男爵领的覆灭被命运横插一手，\
+	将那一天无限推迟。即便如此，你仍蒙 Astrata 的祝福引导，前往谷地，决意尽己所能带去援助与慰藉。"
 	allowed_sexes = list(MALE, FEMALE)
 	allowed_races = RACES_NO_CONSTRUCT
 	outfit = /datum/outfit/job/roguetown/heartfelt/prior
