diff --git a/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/heartfelt/retinue/heartfeltcourtier.dm b/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/heartfelt/retinue/heartfeltcourtier.dm
--- a/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/heartfelt/retinue/heartfeltcourtier.dm
+++ b/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/heartfelt/retinue/heartfeltcourtier.dm
@@ -1,8 +1,8 @@
 
 /datum/advclass/heartfelt/retinue/courtier
-	name = "Heartfeltian Courtier"
-	tutorial = "You are a Courtier of Heartfelt, once a respected noblewoman now struggling to survive in a desolate landscape. \
-	With your home in ruins, you look to the Vale, hoping to find new purpose or refuge amidst the chaos."
+	name = "Heartfelt 廷臣"
+	tutorial = "你是 Heartfelt 的廷臣，曾是一位备受敬重的贵族女子，如今却只能在荒凉的土地上挣扎求生。\
+	故乡已成废墟，你望向谷地，希望能在这片混乱中找到新的使命，或一处容身之所。"
 	allowed_sexes = list(FEMALE)
 	allowed_races = RACES_NO_CONSTRUCT
 	outfit = /datum/outfit/job/roguetown/heartfelt/retinue/courtier
