diff --git a/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/heartfelt/retinue/retinue.dm b/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/heartfelt/retinue/retinue.dm
--- a/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/heartfelt/retinue/retinue.dm
+++ b/code/modules/jobs/job_types/roguetown/adventurer/types/migrant/heartfelt/retinue/retinue.dm
@@ -1,6 +1,7 @@
 
 /datum/job/roguetown/heartfelt/retinue
 	title = "Heartfeltian Retinue"
+	display_title = "Heartfelt 扈从"
 	allowed_sexes = list(MALE, FEMALE)
 	allowed_races = ACCEPTED_RACES
 	job_traits = list(TRAIT_HEARTFELT)
