diff --git a/code/modules/jobs/job_types/roguetown/adventurer/types/combat/foreigner/etrusca.dm b/code/modules/jobs/job_types/roguetown/adventurer/types/combat/foreigner/etrusca.dm
--- a/code/modules/jobs/job_types/roguetown/adventurer/types/combat/foreigner/etrusca.dm
+++ b/code/modules/jobs/job_types/roguetown/adventurer/types/combat/foreigner/etrusca.dm
@@ -3,12 +3,12 @@
 //Navigator, focused on survival skills.
 //Castaway, focused on fancy footwork.
 /datum/advclass/foreigner/nostromo
-	name = "Etruscan Nostromo"
-	tutorial = "Nostromo, a word torn from the common Etruscan tongue, butchered by Imperial. \
-	The meaning it had is long since lost to the common man. You're neither swashbuckler nor romantic. For you are no hero. \
-	To be branded 'Nostromo', is to lose your lot. To be known as one who prattles, rather than using their sword. \
-	Perhaps you were a great captain? A joyous sailor, in better yils? \
-	It hardly matters, now. Forge a new name."
+	name = "Etrusca 失途水手"
+	tutorial = "Nostromo，这个词从通俗 Etrusca 语中被硬生生扯了出来，又被帝国语言嚼得面目全非。\
+	它原本的含义，平民们早已无人知晓。你既不是什么海上浪客，也不是什么浪漫人物，因为你根本不是英雄。\
+	被贴上“Nostromo”这个名号，便意味着你已经失了自己的位置，成了个只会夸夸其谈、却不肯拔剑的人。\
+	也许你曾是伟大的船长？也许你在更好的年岁里，是个快活水手？\
+	如今这些都不重要了。给自己重新挣一个名字吧。"
 	allowed_races = RACES_ALL_KINDS
 	subclass_languages = list(/datum/language/etruscan)
 	outfit = /datum/outfit/job/roguetown/adventurer/nostromo
@@ -28,9 +28,9 @@
 		/datum/skill/misc/reading = SKILL_LEVEL_JOURNEYMAN,
 	)
 
-	extra_context = "This subclass provides the player with two loadouts. \
-	Navigator: +2PER, JMAN swords. \
-	Castaway: +1PER/WIL, JMAN lockpicking, T1 Bardic Inspiration."
+	extra_context = "该子职业为玩家提供两种配置。\
+	领航员：+2 感知，熟练级剑术。\
+	落难者：+1 感知/意志，熟练级开锁，1 级吟游激励。"
 
 /datum/outfit/job/roguetown/adventurer/nostromo/pre_equip(mob/living/carbon/human/H)
 	..()
@@ -45,10 +45,10 @@
 	backl = /obj/item/storage/backpack/rogue/satchel
 	beltr = /obj/item/rogueweapon/scabbard/sheath
 	if(H.mind)
-		var/nostromo_purpose = list("Navigator","Castaway")
-		var/purpose_choice = input(H, "Choose your FAILING", "WHY THE PLANK") as anything in nostromo_purpose
+		var/nostromo_purpose = list("领航员","落难者")
+		var/purpose_choice = input(H, "选择你的失途。", "你为何被逼上跳板") as anything in nostromo_purpose
 		switch(purpose_choice)
-			if("Navigator")
+			if("领航员")
 				H.change_stat(STATKEY_PER, 2)
 				H.adjust_skillrank_up_to(/datum/skill/combat/swords, 3, TRUE)//No shield skill, since you're buckler reliant.
 				r_hand = /obj/item/rogueweapon/sword/long/etruscan//You'd stolen this, probably. It's just a longsword reskin.
@@ -59,7 +59,7 @@
 								/obj/item/rogueweapon/huntingknife/idagger/navaja = 1,
 								/obj/item/flashlight/flare/torch = 1,
 								)
-			if("Castaway")
+			if("落难者")
 				var/datum/inspiration/I = new /datum/inspiration(H)
 				I.grant_inspiration(H, bard_tier = BARD_T1)
 				H.change_stat(STATKEY_PER, 1)
