diff --git a/code/modules/jobs/job_types/roguetown/adventurer/types/combat/foreigner/otava.dm b/code/modules/jobs/job_types/roguetown/adventurer/types/combat/foreigner/otava.dm
--- a/code/modules/jobs/job_types/roguetown/adventurer/types/combat/foreigner/otava.dm
+++ b/code/modules/jobs/job_types/roguetown/adventurer/types/combat/foreigner/otava.dm
@@ -1,8 +1,8 @@
 /datum/advclass/foreigner/repentant
-	name = "Otavan Repentant"
-	tutorial = "An exile from the Holy See of Otava, accused of heresy and cast out of your homeland as penance. \
-	Some consider yours a fate worse than death; the metal alloy mask seared onto your face serving as a permanent reminder of your sins. \
-	You are a living example of what becomes of those who stand in defiance of the Otavan inquisition."
+	name = "Otava 悔罪者"
+	tutorial = "你是被逐出 Otava 圣座疆域的流放者，背负异端之名，被故土放逐以作赎罪。\
+	有些人觉得你的命运比死亡更惨；那张烙在脸上的合金面具，便是你罪孽永不褪去的见证。\
+	你活生生地昭示着，违逆 Otava 宗教裁判所之人最终会落得什么下场。"
 	allowed_races = RACES_ALL_KINDS
 	outfit = /datum/outfit/job/roguetown/adventurer/repentant
 	subclass_languages = list(/datum/language/otavan)
@@ -27,9 +27,9 @@
 
 /datum/outfit/job/roguetown/adventurer/repentant/pre_equip(mob/living/carbon/human/H)
 	..()
-	to_chat(H, span_warning("An exile from the Holy See of Otava, accused of heresy and cast out of your homeland as penance. \
-	Some consider yours a fate worse than death; the metal alloy mask seared onto your face serving as a permanent reminder of your sins. \
-	You are a living example of what becomes of those who stand in defiance of the Otavan inquisition."))
+	to_chat(H, span_warning("你是被逐出 Otava 圣座疆域的流放者，背负异端之名，被故土放逐以作赎罪。\
+	有些人觉得你的命运比死亡更惨；那张烙在脸上的合金面具，便是你罪孽永不褪去的见证。\
+	你活生生地昭示着，违逆 Otava 宗教裁判所之人最终会落得什么下场。"))
 	mask = /obj/item/clothing/mask/rogue/facemask/steel/paalloy/mad_touched
 	wrists = /obj/item/clothing/neck/roguetown/psicross
 	shirt = /obj/item/clothing/cloak/psydontabard
