From bf0d01ff48f55043fd29dee8d71af723de01a947 Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Sat, 30 May 2026 13:00:00 +0800
Subject: [PATCH] 翻译 code/modules/mob/living/carbon/carbon.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
diff --git a/code/modules/mob/living/carbon/carbon.dm b/code/modules/mob/living/carbon/carbon.dm
--- a/code/modules/mob/living/carbon/carbon.dm
+++ b/code/modules/mob/living/carbon/carbon.dm
@@ -27,26 +27,26 @@
 	var/obj/item/bodypart/affecting
 	if(prob(66))
 		affecting = get_bodypart("[pick("r","l")]_leg")
-		to_chat(src, "<span class='warning'>I land on my leg!</span>")
+		to_chat(src, "<span class='warning'>我摔在了腿上！</span>")
 		if(affecting && apply_damage((levels * 10), BRUTE, affecting))		// 100 brute damage
 			update_damage_overlays()
 	else
 		switch(rand(1,3))
 			if(1)
 				affecting = get_bodypart("[pick("r","l")]_arm")
-				to_chat(src, "<span class='warning'>I land on my arm!</span>")
+				to_chat(src, "<span class='warning'>我摔在了手臂上！</span>")
 				if(affecting && apply_damage((levels * 10), BRUTE, affecting))		// 100 brute damage
 					update_damage_overlays()
 			if(2)
 				affecting = get_bodypart("chest")
-				to_chat(src, "<span class='warning'>I land on my chest!</span>")
+				to_chat(src, "<span class='warning'>我摔在了胸口上！</span>")
 				adjustOxyLoss(50)
 				emote("breathgasp")
 				if(affecting && apply_damage((levels * 10), BRUTE, affecting))		// 100 brute damage
 					update_damage_overlays()
 			if(3)
 				affecting = get_bodypart("head")
-				to_chat(src, "<span class='warning'>I land on my head!</span>")
+				to_chat(src, "<span class='warning'>我摔在了头上！</span>")
 				if(levels > 2)
 					AdjustUnconscious(levels * 100)
 					if(affecting && apply_damage((levels * 10), BRUTE, affecting))		// 100 brute damage
@@ -67,7 +67,7 @@
 	if(item_in_hand) //this segment checks if the item in your hand is twohanded.
 		if(istype(item_in_hand))
 			if(item_in_hand.wielded == 1)
-				to_chat(usr, "<span class='warning'>My other hand is too busy holding [item_in_hand].</span>")
+				to_chat(usr, "<span class='warning'>我的另一只手正忙着握着[item_in_hand]。</span>")
 				return FALSE
 	if(atkswinging || atkreleasing)
 		stop_attack(FALSE)
@@ -90,7 +90,7 @@
 	if(ishuman(src))
 		var/mob/living/carbon/human/H = src
 		if(H.has_status_effect(/datum/status_effect/buff/clash))
-			H.bad_guard(span_warning("I swapped away from the weapon!"))
+			H.bad_guard(span_warning("我把手切离武器了！"))
 	return TRUE
 
 
@@ -127,7 +127,7 @@
 		if(possible_len)
 			var/datum/surgery_step/done_step
 			if(possible_len > 1)
-				var/input = input(user, "Which surgery step do you want to perform?", "PESTRA", ) as null|anything in possible_steps
+				var/input = input(user, "你想进行哪一步手术？", "PESTRA", ) as null|anything in possible_steps
 				if(input)
 					done_step = possible_steps[input]
 			else
@@ -160,8 +160,8 @@
 		if(hurt)
 			victim.take_bodypart_damage(10,check_armor = TRUE)
 			take_bodypart_damage(10,check_armor = TRUE)
-			visible_message("<span class='danger'>[src] crashes into [victim]!",\
-				"<span class='danger'>I violently crash into [victim]!</span>")
+			visible_message("<span class='danger'>[src]猛地撞上了[victim]！",\
+				"<span class='danger'>我狠狠撞上了[victim]！</span>")
 		playsound(src,"genblunt",100,TRUE)
 
 
@@ -247,7 +247,7 @@
 			dropItemToGround(I, silent = TRUE)
 
 			if(HAS_TRAIT(src, TRAIT_PACIFISM) && I.throwforce)
-				to_chat(src, "<span class='notice'>I set [I] down gently on the ground.</span>")
+				to_chat(src, "<span class='notice'>我把[I]轻轻放到了地上。</span>")
 				return
 
 
@@ -264,8 +264,8 @@
 		if(!thrown_range)
 			thrown_range = thrown_thing.throw_range
 		do_attack_animation(target, no_effect = TRUE)
-		visible_message("<span class='danger'>[src] throws [thrown_thing].</span>", \
-						"<span class='danger'>I toss [thrown_thing].</span>")
+		visible_message("<span class='danger'>[src]掷出了[thrown_thing]。</span>", \
+						"<span class='danger'>我把[thrown_thing]扔了出去。</span>")
 		log_message("has thrown [thrown_thing]", LOG_ATTACK)
 		thrown_thing.safe_throw_at(target, thrown_range, thrown_speed, src, null, null, null, move_force)
 		changeNext_move(CLICK_CD_MELEE)
@@ -291,30 +291,30 @@
 	<HR>
 	<B><FONT size=3>[name]</FONT></B>
 	<HR>
-	<BR><B>Head:</B> <A href='?src=[REF(src)];item=[SLOT_HEAD]'>[(head && !(head.item_flags & ABSTRACT)) ? head : "Nothing"]</A>"}
+	<BR><B>头部：</B> <A href='?src=[REF(src)];item=[SLOT_HEAD]'>[(head && !(head.item_flags & ABSTRACT)) ? head : "无"]</A>"}
 
 	var/list/obscured = check_obscured_slots()
 
 	if(SLOT_NECK in obscured)
-		dat += "<BR><B>Neck:</B> Obscured"
-	else
-		dat += "<BR><B>Neck:</B> <A href='?src=[REF(src)];item=[SLOT_NECK]'>[(wear_neck && !(wear_neck.item_flags & ABSTRACT)) ? (wear_neck) : "Nothing"]</A>"
+		dat += "<BR><B>颈部：</B>被遮挡"
+	else
+		dat += "<BR><B>颈部：</B> <A href='?src=[REF(src)];item=[SLOT_NECK]'>[(wear_neck && !(wear_neck.item_flags & ABSTRACT)) ? (wear_neck) : "无"]</A>"
 
 	if(SLOT_WEAR_MASK in obscured)
-		dat += "<BR><B>Mask:</B> Obscured"
-	else
-		dat += "<BR><B>Mask:</B> <A href='?src=[REF(src)];item=[SLOT_WEAR_MASK]'>[(wear_mask && !(wear_mask.item_flags & ABSTRACT))	? wear_mask	: "Nothing"]</a>"
+		dat += "<BR><B>面部：</B>被遮挡"
+	else
+		dat += "<BR><B>面部：</B> <A href='?src=[REF(src)];item=[SLOT_WEAR_MASK]'>[(wear_mask && !(wear_mask.item_flags & ABSTRACT))	? wear_mask	: "无"]</a>"
 
 	for(var/i in 1 to held_items.len)
 		var/obj/item/I = get_item_for_held_index(i)
-		dat += "<BR><B>[get_held_index_name(i)]:</B> </td><td><A href='?src=[REF(src)];item=[SLOT_HANDS];hand_index=[i]'>[(I && !(I.item_flags & ABSTRACT)) ? I : "Nothing"]</a>"
-
-	dat += "<BR><B>Back:</B> <A href='?src=[REF(src)];item=[SLOT_BACK]'>[back ? back : "Nothing"]</A>"
+		dat += "<BR><B>[get_held_index_name(i)]:</B> </td><td><A href='?src=[REF(src)];item=[SLOT_HANDS];hand_index=[i]'>[(I && !(I.item_flags & ABSTRACT)) ? I : "无"]</a>"
+
+	dat += "<BR><B>背部：</B> <A href='?src=[REF(src)];item=[SLOT_BACK]'>[back ? back : "无"]</A>"
 
 	if(handcuffed)
-		dat += "<BR><A href='?src=[REF(src)];item=[SLOT_HANDCUFFED]'>Handcuffed</A>"
+		dat += "<BR><A href='?src=[REF(src)];item=[SLOT_HANDCUFFED]'>已铐住</A>"
 	if(legcuffed)
-		dat += "<BR><A href='?src=[REF(src)];item=[SLOT_LEGCUFFED]'>Legcuffed</A>"
+		dat += "<BR><A href='?src=[REF(src)];item=[SLOT_LEGCUFFED]'>已束腿</A>"
 
 	var/datum/status_effect/bugged/effect = has_status_effect(/datum/status_effect/bugged)
 	if(effect && HAS_TRAIT(user, TRAIT_INQUISITION))
@@ -322,7 +322,7 @@
 
 	dat += {"
 	<BR>
-	<BR><A href='?src=[REF(user)];mach_close=mob[REF(src)]'>Close</A>
+	<BR><A href='?src=[REF(user)];mach_close=mob[REF(src)]'>关闭</A>
 	"}
 	user << browse(dat, "window=mob[REF(src)];size=325x500")
 	onclose(user, "mob[REF(src)]")
@@ -369,7 +369,7 @@
 			buckled.user_unbuckle_mob(src,src)
 		else
 			if(src && buckled)
-				to_chat(src, "<span class='warning'>I fail to struggle free!</span>")
+				to_chat(src, "<span class='warning'>我没能挣脱！</span>")
 	else
 		buckled.user_unbuckle_mob(src,src)
 
@@ -390,9 +390,9 @@
 		adjust_fire_stacks(-5, /datum/status_effect/fire_handler/fire_stacks/sunder)
 		adjust_fire_stacks(-5, /datum/status_effect/fire_handler/fire_stacks/divine)
 		adjust_fire_stacks(-5, /datum/status_effect/fire_handler/fire_stacks/sunder/blessed)
-		visible_message(span_warning("[src] rolls on the ground, trying to put [p_them()]self out!"))
-	else
-		visible_message(span_notice("[src] pats the flames to extinguish them."))
+		visible_message(span_warning("[src]在地上翻滚，试图扑灭自己身上的火！"))
+	else
+		visible_message(span_notice("[src]拍打着火焰，试图将其熄灭。"))
 	addtimer(CALLBACK(src, PROC_REF(check_try_extinguish)), 3 SECONDS)
 
 /mob/living/carbon/proc/check_try_extinguish()
@@ -400,14 +400,14 @@
 		extinguish_mob(TRUE)
 
 /mob/living/carbon/resist_leash()
-	to_chat(src, span_notice("I reach for the hook on my collar..."))
+	to_chat(src, span_notice("我伸手去摸项圈上的钩扣……"))
 	//Determine how long it takes to remove the leash
 	var/deleash = 15
 	if(src.handcuffed)
 		deleash = 60
 	if(move_after(src, deleash, 0, target = src))
 		if(!QDELETED(src))
-			to_chat(src, "<span class='warning'>[src] has removed their leash!</span>")
+			to_chat(src, "<span class='warning'>[src]解开了自己身上的牵绳！</span>")
 			src.remove_status_effect(/datum/status_effect/leash_pet)
 
 /mob/living/carbon/resist_restraints()
@@ -431,7 +431,7 @@
 
 /mob/living/carbon/proc/cuff_resist(obj/item/I, breakouttime = 600, cuff_break = 0)
 	if(I.item_flags & BEING_REMOVED)
-		to_chat(src, "<span class='warning'>You're already attempting to remove [I]!</span>")
+		to_chat(src, "<span class='warning'>我已经在尝试移除[I]了！</span>")
 		return
 	I.item_flags |= BEING_REMOVED
 	breakouttime = I.slipouttime
@@ -444,18 +444,18 @@
 		cuff_break = INSTANT_CUFFBREAK
 		breakouttime = I.breakouttime
 	if(!cuff_break)
-		to_chat(src, "<span class='notice'>I attempt to remove [I]...</span>")
+		to_chat(src, "<span class='notice'>我试图移除[I]……</span>")
 		if(do_after(src, breakouttime, 0, target = src))
 			clear_cuffs(I, cuff_break)
 		else
-			to_chat(src, "<span class='danger'>I fail to remove [I]!</span>")
+			to_chat(src, "<span class='danger'>我没能移除[I]！</span>")
 
 	else if(cuff_break == FAST_CUFFBREAK)
-		to_chat(src, "<span class='notice'>I attempt to break [I]...</span>")
+		to_chat(src, "<span class='notice'>我试图挣断[I]……</span>")
 		if(do_after(src, breakouttime, 0, target = src))
 			clear_cuffs(I, cuff_break)
 		else
-			to_chat(src, "<span class='danger'>I fail to break [I]!</span>")
+			to_chat(src, "<span class='danger'>我没能挣断[I]！</span>")
 
 	else if(cuff_break == INSTANT_CUFFBREAK)
 		clear_cuffs(I, cuff_break)
@@ -496,10 +496,10 @@
 		return FALSE
 	if(I != handcuffed && I != legcuffed)
 		return FALSE
-	visible_message("[cuff_break ? "<span class='danger'>" : "<span class='warning'>"][src] manages to [cuff_break ? "break" : "slip"] out of [I]!</span>")
+	visible_message("[cuff_break ? "<span class='danger'>" : "<span class='warning'>"][src]设法从[I]中[cuff_break ? "挣断" : "滑脱"]了出来！</span>")
 	if(cuff_break)
 		playsound(src, 'sound/misc/chain_snap.ogg', 100, FALSE, 10)
-	to_chat(src, "<span class='notice'>I [cuff_break ? "break" : "slip"] out of [I]!</span>")
+	to_chat(src, "<span class='notice'>我从[I]中[cuff_break ? "挣断" : "滑脱"]了出来！</span>")
 
 	if(istype(I, /obj/item/net))
 		if(has_status_effect(/datum/status_effect/debuff/netted))
@@ -608,15 +608,15 @@
 			if(world.time > mob_timers["puke"] + 16 SECONDS)
 				mob_timers["puke"] = world.time
 				if(getorgan(/obj/item/organ/stomach))
-					to_chat(src, "<span class='warning'>I'm going to puke...</span>")
+					to_chat(src, "<span class='warning'>我快要吐了……</span>")
 					addtimer(CALLBACK(src, PROC_REF(vomit), 50), rand(8 SECONDS, 15 SECONDS))
 			else
 				if(prob(3))
-					to_chat(src, "<span class='warning'>I feel sick...</span>")
+					to_chat(src, "<span class='warning'>我感觉很恶心……</span>")
 		else
 			if(getorgan(/obj/item/organ/stomach))
 				mob_timers["puke"] = world.time
-				to_chat(src, "<span class='warning'>I'm going to puke...</span>")
+				to_chat(src, "<span class='warning'>我快要吐了……</span>")
 				addtimer(CALLBACK(src, PROC_REF(vomit), 50), rand(8 SECONDS, 15 SECONDS))
 	add_nausea(-1)
 
@@ -1180,7 +1180,7 @@
 			O.Remove(src)
 			O.forceMove(drop_location())
 	if(organs_amt)
-		to_chat(user, "<span class='notice'>I retrieve some of [src]\'s internal organs!</span>")
+		to_chat(user, "<span class='notice'>我取出了[src]的一些内脏！</span>")
 
 /mob/living/carbon/extinguish_mob(itemz = TRUE)
 	if(itemz)
@@ -1267,21 +1267,21 @@
 					if(BP)
 						BP.drop_limb()
 					else
-						to_chat(usr, "<span class='boldwarning'>[src] doesn't have such bodypart.</span>")
+						to_chat(usr, "<span class='boldwarning'>[src]没有这样的肢体部位。</span>")
 				if("add")
 					if(BP)
-						to_chat(usr, "<span class='boldwarning'>[src] already has such bodypart.</span>")
+						to_chat(usr, "<span class='boldwarning'>[src]已经有这样的肢体部位了。</span>")
 					else
 						if(!regenerate_limb(result))
-							to_chat(usr, "<span class='boldwarning'>[src] cannot have such bodypart.</span>")
+							to_chat(usr, "<span class='boldwarning'>[src]无法拥有这样的肢体部位。</span>")
 				if("augment")
 					if(ishuman(src))
 						if(BP)
 							BP.change_bodypart_status(BODYPART_ROBOTIC, TRUE, TRUE)
 						else
-							to_chat(usr, "<span class='boldwarning'>[src] doesn't have such bodypart.</span>")
+							to_chat(usr, "<span class='boldwarning'>[src]没有这样的肢体部位。</span>")
 					else
-						to_chat(usr, "<span class='boldwarning'>Only humans can be augmented.</span>")
+						to_chat(usr, "<span class='boldwarning'>只有人类可以接受强化。</span>")
 		admin_ticket_log("[key_name_admin(usr)] has modified the bodyparts of [src]")
 	if(href_list[VV_HK_MODIFY_ORGANS])
 		if(!check_rights(NONE))
@@ -1299,7 +1299,7 @@
 		if(!usr)
 			return
 		if(QDELETED(src))
-			to_chat(usr, "<span class='boldwarning'>Mob doesn't exist anymore.</span>")
+			to_chat(usr, "<span class='boldwarning'>这个生物已经不存在了。</span>")
 			return
 		if(result)
 			var/chosenart = artnames[result]
@@ -1315,7 +1315,7 @@
 		if(!usr)
 			return
 		if(QDELETED(src))
-			to_chat(usr, "Mob doesn't exist anymore")
+			to_chat(usr, "这个生物已经不存在了")
 			return
 		if(!result)
 			return
@@ -1337,7 +1337,7 @@
 		if(!usr)
 			return
 		if(QDELETED(src))
-			to_chat(usr, "Mob doesn't exist anymore")
+			to_chat(usr, "这个生物已经不存在了")
 			return
 		if(result)
 			new result(src, TRUE)
@@ -1368,14 +1368,14 @@
 		return FALSE
 
 /mob/living/carbon/resist_leash()
-	to_chat(src, span_notice("I reach for the hook on my collar..."))
+	to_chat(src, span_notice("我伸手去摸项圈上的钩扣……"))
 	//Determine how long it takes to remove the leash
 	var/deleash = 5 SECONDS
 	if(src.handcuffed)
 		deleash = 20 SECONDS
 	if(move_after(src, deleash, 0, target = src))
 		if(!QDELETED(src))
-			to_chat(src, "<span class='warning'>[src] has removed their leash!</span>")
+			to_chat(src, "<span class='warning'>[src]解开了自己身上的牵绳！</span>")
 			src.remove_status_effect(/datum/status_effect/leash_pet)
 
 /mob/living/carbon/can_buckle()
-- 
Translation Patch Toolkit
