From 9f46ea82656554d4dad6510b0e0ea22428af840d Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Sat, 30 May 2026 13:00:00 +0800
Subject: [PATCH] 翻译 code/modules/mob/living/carbon/death.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
diff --git a/code/modules/mob/living/carbon/death.dm b/code/modules/mob/living/carbon/death.dm
--- a/code/modules/mob/living/carbon/death.dm
+++ b/code/modules/mob/living/carbon/death.dm
@@ -39,7 +39,7 @@
 	var/atom/Tsec = drop_location()
 	for(var/mob/M in src)
 		M.forceMove(Tsec)
-		visible_message(span_danger("[M] bursts out of [src]!"))
+		visible_message(span_danger("[M]从[src]体内破体而出！"))
 	. = ..()
 
 /mob/living/carbon/spill_embedded_objects()
-- 
Translation Patch Toolkit
