From d206458ba877523a696c3c1eaaf8ce77e33da288 Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Sat, 30 May 2026 13:00:00 +0800
Subject: [PATCH] 翻译 code/modules/mob/living/carbon/status_procs.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
diff --git a/code/modules/mob/living/carbon/status_procs.dm b/code/modules/mob/living/carbon/status_procs.dm
--- a/code/modules/mob/living/carbon/status_procs.dm
+++ b/code/modules/mob/living/carbon/status_procs.dm
@@ -12,7 +12,7 @@
 	if(absorb_stun(0)) //continuous effect, so we don't want it to increment the stuns absorbed.
 		return
 	if(!IsParalyzed())
-		to_chat(src, span_notice("You're too exhausted to keep going..."))
+		to_chat(src, span_notice("你已经累得撑不下去了……"))
 	var/prev = stam_paralyzed
 	stam_paralyzed = TRUE
 	if(!prev && getStaminaLoss() < 120) // Puts you a little further into the initial stamcrit, makes stamcrit harder to outright counter with chems.
-- 
Translation Patch Toolkit
