From ccd2f97a6d349a1fa60be0018f92bdca329206c6 Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Sat, 30 May 2026 12:00:00 +0800
Subject: [PATCH] 翻译 code/modules/mob/dead/new_player/body_markings/body_markings.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

diff --git a/code/modules/mob/dead/new_player/body_markings/body_markings.dm b/code/modules/mob/dead/new_player/body_markings/body_markings.dm
index b7e6b7ef61cb..f69381bc5ac5 100644
--- a/code/modules/mob/dead/new_player/body_markings/body_markings.dm
+++ b/code/modules/mob/dead/new_player/body_markings/body_markings.dm
@@ -44,28 +44,28 @@
 
 /datum/body_marking/nose
 	icon = 'icons/mob/body_markings/other_markings.dmi'
-	name = "Nose"
+	name = "鼻部"
 	icon_state = "nose"
 	default_color = "FF0000"
 	affected_bodyparts = HEAD
 
 /datum/body_marking/flushed_cheeks
 	icon = 'icons/mob/body_markings/other_markings.dmi'
-	name = "Flushed Cheeks"
+	name = "泛红双颊"
 	icon_state = "flushed_cheeks"
 	default_color = "FF0000"
 	affected_bodyparts = HEAD
 
 /datum/body_marking/eyeliner
 	icon = 'icons/mob/body_markings/other_markings.dmi'
-	name = "Eyeliner"
+	name = "眼线"
 	icon_state = "eyeliner"
 	default_color = "FF0000"
 	affected_bodyparts = HEAD
 
 /datum/body_marking/plain
 	icon = 'icons/mob/body_markings/plain_markings.dmi'
-	name = "Plain"
+	name = "普通"
 	icon_state = "plain"
 	affected_bodyparts = HEAD | CHEST | HAND_LEFT | HAND_RIGHT | ARM_LEFT | ARM_RIGHT | LEG_LEFT | LEG_RIGHT
 	default_color = DEFAULT_SECONDARY
@@ -73,57 +73,57 @@
 
 /datum/body_marking/spotted
 	icon = 'icons/mob/body_markings/spotted_markings.dmi'
-	name = "Spots"
+	name = "斑点"
 	icon_state = "spotted"
 	affected_bodyparts = HEAD | CHEST | HAND_LEFT | HAND_RIGHT | ARM_LEFT | ARM_RIGHT | LEG_LEFT | LEG_RIGHT
 	default_color = DEFAULT_TERTIARY
 
 /datum/body_marking/tiger
 	icon = 'icons/mob/body_markings/tiger_markings.dmi'
-	name = "Tiger"
+	name = "虎纹"
 	icon_state = "ltiger"
 	affected_bodyparts = CHEST | ARM_LEFT | ARM_RIGHT | LEG_LEFT | LEG_RIGHT
 	default_color = DEFAULT_SECONDARY
 
 /datum/body_marking/tiger/dark
-	name = "Tiger (Dark)"
+	name = "虎纹（深色）"
 	default_color = "444444"
 
 /datum/body_marking/gradient
 	icon = 'icons/mob/body_markings/gradient_markings.dmi'
-	name = "Gradient"
+	name = "渐变"
 	icon_state = "gradient"
 	affected_bodyparts = ARM_LEFT | ARM_RIGHT | HAND_LEFT | HAND_RIGHT | LEG_LEFT | LEG_RIGHT
 	default_color = DEFAULT_SECONDARY
 
 /datum/body_marking/sock
 	icon = 'icons/mob/body_markings/sock_markings.dmi'
-	name = "Sock"
+	name = "短袜纹"
 	icon_state = "sock"
 	affected_bodyparts = HAND_LEFT | HAND_RIGHT | LEG_LEFT | LEG_RIGHT
 	default_color = DEFAULT_SECONDARY
 
 /datum/body_marking/sock/tertiary
-	name = "Sock (Tertiary)"
+	name = "短袜纹（第三色）"
 	default_color = DEFAULT_TERTIARY
 
 /datum/body_marking/socklonger
 	icon = 'icons/mob/body_markings/sock_markings.dmi'
-	name = "Sock (Longer)"
+	name = "长短袜纹"
 	icon_state = "socklonger"
 	affected_bodyparts = ARM_LEFT | ARM_RIGHT | LEG_LEFT | LEG_RIGHT
 	default_color = DEFAULT_SECONDARY
 
 /datum/body_marking/tips
 	icon = 'icons/mob/body_markings/tips_markings.dmi'
-	name = "Tips"
+	name = "尖端"
 	icon_state = "tips"
 	affected_bodyparts = HAND_LEFT | HAND_RIGHT | LEG_LEFT | LEG_RIGHT
 	default_color = DEFAULT_SECONDARY
 
 /datum/body_marking/wolf
 	icon = 'icons/mob/body_markings/chest_markings.dmi'
-	name = "Wolf"
+	name = "狼纹"
 	icon_state = "wolf"
 	affected_bodyparts = CHEST
 	default_color = DEFAULT_SECONDARY
@@ -131,7 +131,7 @@
 
 /datum/body_marking/bellyscale
 	icon = 'icons/mob/body_markings/chest_markings.dmi'
-	name = "Belly Scales"
+	name = "腹部鳞片"
 	icon_state = "bellyscale"
 	affected_bodyparts = CHEST
 	default_color = DEFAULT_SECONDARY
@@ -139,7 +139,7 @@
 
 /datum/body_marking/kobold_scale
 	icon = 'icons/mob/body_markings/chest_markings.dmi'
-	name = "Kobold Scales"
+	name = "狗头人鳞片"
 	icon_state = "kobold_scale"
 	affected_bodyparts = CHEST
 	default_color = DEFAULT_SECONDARY
@@ -147,7 +147,7 @@
 
 /datum/body_marking/bellyscaleslim
 	icon = 'icons/mob/body_markings/chest_markings.dmi'
-	name = "Belly Scales (Slim)"
+	name = "腹部鳞片（纤细）"
 	icon_state = "bellyscaleslim"
 	affected_bodyparts = CHEST
 	default_color = DEFAULT_SECONDARY
@@ -155,7 +155,7 @@
 
 /datum/body_marking/bellyscalesmooth
 	icon = 'icons/mob/body_markings/chest_markings.dmi'
-	name = "Belly Scales (Smooth)"
+	name = "腹部鳞片（平滑）"
 	icon_state = "bellyscalesmooth"
 	affected_bodyparts = CHEST
 	default_color = DEFAULT_SECONDARY
@@ -163,7 +163,7 @@
 
 /datum/body_marking/bellyscaleslimsmooth
 	icon = 'icons/mob/body_markings/chest_markings.dmi'
-	name = "Belly Scales (Slim, Smooth)"
+	name = "腹部鳞片（纤细，平滑）"
 	icon_state = "bellyscaleslimsmooth"
 	affected_bodyparts = CHEST
 	default_color = DEFAULT_SECONDARY
@@ -171,14 +171,14 @@
 
 /datum/body_marking/buttscale
 	icon = 'icons/mob/body_markings/chest_markings.dmi'
-	name = "Butt Scales"
+	name = "臀部鳞片"
 	icon_state = "buttscale"
 	affected_bodyparts = CHEST
 	default_color = DEFAULT_SECONDARY
 
 /datum/body_marking/belly
 	icon = 'icons/mob/body_markings/chest_markings.dmi'
-	name = "Belly"
+	name = "腹部"
 	icon_state = "belly"
 	affected_bodyparts = CHEST
 	default_color = DEFAULT_SECONDARY
@@ -186,7 +186,7 @@
 
 /datum/body_marking/bellyslim
 	icon = 'icons/mob/body_markings/chest_markings.dmi'
-	name = "Belly (Slim)"
+	name = "腹部（纤细）"
 	icon_state = "bellyslim"
 	affected_bodyparts = CHEST
 	default_color = DEFAULT_SECONDARY
@@ -194,14 +194,14 @@
 
 /datum/body_marking/butt
 	icon = 'icons/mob/body_markings/chest_markings.dmi'
-	name = "Butt"
+	name = "臀部"
 	icon_state = "butt"
 	affected_bodyparts = CHEST
 	default_color = DEFAULT_SECONDARY
 
 /datum/body_marking/tie
 	icon = 'icons/mob/body_markings/chest_markings.dmi'
-	name = "Tie"
+	name = "领带纹"
 	icon_state = "tie"
 	affected_bodyparts = CHEST
 	default_color = DEFAULT_SECONDARY
@@ -209,7 +209,7 @@
 
 /datum/body_marking/tiesmall
 	icon = 'icons/mob/body_markings/chest_markings.dmi'
-	name = "Tie (Small)"
+	name = "领带纹（小）"
 	icon_state = "tiesmall"
 	affected_bodyparts = CHEST
 	default_color = DEFAULT_SECONDARY
@@ -217,14 +217,14 @@
 
 /datum/body_marking/backspots
 	icon = 'icons/mob/body_markings/chest_markings.dmi'
-	name = "Back Spots"
+	name = "背部斑点"
 	icon_state = "backspots"
 	affected_bodyparts = CHEST
 	default_color = DEFAULT_SECONDARY
 
 /datum/body_marking/front
 	icon = 'icons/mob/body_markings/chest_markings.dmi'
-	name = "Front"
+	name = "前部"
 	icon_state = "front"
 	affected_bodyparts = CHEST
 	default_color = DEFAULT_SECONDARY
@@ -232,28 +232,28 @@
 
 /datum/body_marking/tonage
 	icon = 'icons/mob/body_markings/other_markings.dmi'
-	name = "Tonage"
+	name = "色调"
 	icon_state = "tonage"
 	affected_bodyparts = CHEST
 	default_color = "555555"
 
 /datum/body_marking/drake_eyes
 	icon = 'icons/mob/body_markings/other_markings.dmi'
-	name = "Drake Eyes"
+	name = "龙目纹"
 	icon_state = "drake_eyes"
 	affected_bodyparts = HEAD
 	default_color = "FF0000"
 
 /datum/body_marking/small/nose
 	icon = 'icons/mob/body_markings/other_markings.dmi'
-	name = "Nose (Volk)"
+	name = "鼻部（Volk）"
 	icon_state = "nose_s"
 	default_color = "FF0000"
 	affected_bodyparts = HEAD
 
 /datum/body_marking/small/plain
 	icon = 'icons/mob/body_markings/small_plain_markings.dmi'
-	name = "Plain (Volk)"
+	name = "普通（Volk）"
 	icon_state = "plain_s"
 	affected_bodyparts = HEAD | CHEST | HAND_LEFT | HAND_RIGHT | ARM_LEFT | ARM_RIGHT | LEG_LEFT | LEG_RIGHT
 	default_color = DEFAULT_SECONDARY
@@ -261,39 +261,39 @@
 
 /datum/body_marking/small/spotted
 	icon = 'icons/mob/body_markings/small_spotted_markings.dmi'
-	name = "Spots (Volk)"
+	name = "斑点（Volk）"
 	icon_state = "spotted"
 	affected_bodyparts = HEAD | CHEST | HAND_LEFT | HAND_RIGHT | ARM_LEFT | ARM_RIGHT | LEG_LEFT | LEG_RIGHT
 	default_color = DEFAULT_TERTIARY
 
 /datum/body_marking/small/sock
 	icon = 'icons/mob/body_markings/small_sock_markings.dmi'
-	name = "Sock (Volk)"
+	name = "短袜纹（Volk）"
 	icon_state = "sock_s"
 	affected_bodyparts = HAND_LEFT | HAND_RIGHT | LEG_LEFT | LEG_RIGHT
 	default_color = DEFAULT_SECONDARY
 
 /datum/body_marking/small/sock/tertiary
-	name = "Sock (Tertiary) (Volk)"
+	name = "短袜纹（第三色）（Volk）"
 	default_color = DEFAULT_TERTIARY
 
 /datum/body_marking/small/socklonger
 	icon = 'icons/mob/body_markings/small_sock_markings.dmi'
-	name = "Sock (Longer) (Volk)"
+	name = "长短袜纹（Volk）"
 	icon_state = "socklonger_s"
 	affected_bodyparts = ARM_LEFT | ARM_RIGHT | LEG_LEFT | LEG_RIGHT
 	default_color = DEFAULT_SECONDARY
 
 /datum/body_marking/small/tips
 	icon = 'icons/mob/body_markings/small_tips_markings.dmi'
-	name = "Tips (Volk)"
+	name = "尖端（Volk）"
 	icon_state = "tips_s"
 	affected_bodyparts = HAND_LEFT | HAND_RIGHT | LEG_LEFT | LEG_RIGHT
 	default_color = DEFAULT_SECONDARY
 
 /datum/body_marking/small/belly
 	icon = 'icons/mob/body_markings/small_chest_markings.dmi'
-	name = "Belly (Volk)"
+	name = "腹部（Volk）"
 	icon_state = "belly_s"
 	affected_bodyparts = CHEST
 	default_color = DEFAULT_SECONDARY
@@ -301,7 +301,7 @@
 
 /datum/body_marking/small/bellyslim
 	icon = 'icons/mob/body_markings/small_chest_markings.dmi'
-	name = "Belly (Slim) (Volk)"
+	name = "腹部（纤细）（Volk）"
 	icon_state = "bellyslim_s"
 	affected_bodyparts = CHEST
 	default_color = DEFAULT_SECONDARY
@@ -309,14 +309,14 @@
 
 /datum/body_marking/small/butt
 	icon = 'icons/mob/body_markings/small_chest_markings.dmi'
-	name = "Butt (Volk)"
+	name = "臀部（Volk）"
 	icon_state = "butt_s"
 	affected_bodyparts = CHEST
 	default_color = DEFAULT_SECONDARY
 
 /datum/body_marking/small/tie
 	icon = 'icons/mob/body_markings/small_chest_markings.dmi'
-	name = "Tie (Volk)"
+	name = "领带纹（Volk）"
 	icon_state = "tie_s"
 	affected_bodyparts = CHEST
 	default_color = DEFAULT_SECONDARY
@@ -324,7 +324,7 @@
 
 /datum/body_marking/small/tiesmall
 	icon = 'icons/mob/body_markings/small_chest_markings.dmi'
-	name = "Tie (Small) (Volk)"
+	name = "领带纹（小）（Volk）"
 	icon_state = "tiesmall_s"
 	affected_bodyparts = CHEST
 	default_color = DEFAULT_SECONDARY
@@ -332,14 +332,14 @@
 
 /datum/body_marking/small/backspots
 	icon = 'icons/mob/body_markings/small_chest_markings.dmi'
-	name = "Back Spots (Volk)"
+	name = "背部斑点（Volk）"
 	icon_state = "backspots_s"
 	affected_bodyparts = CHEST
 	default_color = DEFAULT_SECONDARY
 
 /datum/body_marking/small/front
 	icon = 'icons/mob/body_markings/small_chest_markings.dmi'
-	name = "Front (Volk)"
+	name = "前部（Volk）"
 	icon_state = "front_s"
 	affected_bodyparts = CHEST
 	default_color = DEFAULT_SECONDARY
@@ -347,7 +347,7 @@
 
 /datum/body_marking/harlequin
 	icon = 'icons/mob/body_markings/plain_markings.dmi'
-	name = "Harlequin"
+	name = "丑角"
 	icon_state = "harlequin"
 	affected_bodyparts = HEAD | CHEST
 	default_color = DEFAULT_SECONDARY
@@ -355,7 +355,7 @@
 
 /datum/body_marking/harlequinreversed
 	icon = 'icons/mob/body_markings/plain_markings.dmi'
-	name = "Harlequin (Reversed)"
+	name = "丑角（反转）"
 	icon_state = "harlequin_reversed"
 	affected_bodyparts = HEAD | CHEST
 	default_color = DEFAULT_SECONDARY
@@ -365,42 +365,42 @@
 
 /datum/body_marking/bangs
 	icon = 'icons/mob/body_markings/other_markings.dmi'
-	name = "Bangs"
+	name = "刘海"
 	icon_state = "bangs"
 	default_color = "FFFFFF"
 	affected_bodyparts = HEAD
 	
 /datum/body_marking/small/bangs
 	icon = 'icons/mob/body_markings/other_markings.dmi'
-	name = "Bangs (Volk)"
+	name = "刘海（Volk）"
 	icon_state = "bangs_s"
 	default_color = "FFFFFF"
 	affected_bodyparts = HEAD
 	
 /datum/body_marking/bun
 	icon = 'icons/mob/body_markings/other_markings.dmi'
-	name = "Bun"
+	name = "发髻"
 	icon_state = "bun"
 	default_color = "FFFFFF"
 	affected_bodyparts = HEAD
 
 /datum/body_marking/small/bun
 	icon = 'icons/mob/body_markings/other_markings.dmi'
-	name = "Bun (Volk)"
+	name = "发髻（Volk）"
 	icon_state = "bun_s"
 	default_color = "FFFFFF"
 	affected_bodyparts = HEAD
 
 /datum/body_marking/harpy_feet_claws
 	icon = 'icons/mob/body_markings/harpy_markings.dmi'
-	name = "Feet Claws"
+	name = "足爪"
 	icon_state = "hclaw"
 	affected_bodyparts = LEG_LEFT | LEG_RIGHT
 	default_color = DEFAULT_SECONDARY
 
 /datum/body_marking/harpy_leg
 	icon = 'icons/mob/body_markings/harpy_markings.dmi'
-	name = "Avian Leg Color Override"
+	name = "鸟类腿部颜色覆盖"
 	icon_state = "hleg"
 	affected_bodyparts = LEG_LEFT | LEG_RIGHT
 	default_color = DEFAULT_SECONDARY
-- 
Translation Patch Toolkit
