diff --git a/code/modules/roguetown/roguejobs/engineer/gadgets.dm b/code/modules/roguetown/roguejobs/engineer/gadgets.dm
--- a/code/modules/roguetown/roguejobs/engineer/gadgets.dm
+++ b/code/modules/roguetown/roguejobs/engineer/gadgets.dm
@@ -1,6 +1,6 @@
 /obj/item/storage/gadget
-	name = "gadget"
-	desc = "A gadget of some sort."
+	name = "装置"
+	desc = "某种装置。"
 	icon = 'icons/roguetown/misc/gadgets.dmi'
 	icon_state = "gadget"
 	w_class = WEIGHT_CLASS_TINY
@@ -9,8 +9,8 @@
 	resistance_flags = FIRE_PROOF
 
 /obj/item/storage/gadget/messkit
-	name = "mess kit"
-	desc = "A small, portable mess kit. It can be used to cook food."
+	name = "行军炊具"
+	desc = "一套小巧便携的炊具，可以用来烹饪食物。"
 	slot_flags = ITEM_SLOT_HIP
 	grid_width = 64
 	grid_height = 32
@@ -24,8 +24,8 @@
 	new /obj/item/reagent_containers/glass/bucket/pot/kettle(src)
 
 /obj/item/folding_table_stored
-	name = "folding table"
-	desc = "A folding table, useful for setting up a temporary workspace."
+	name = "折叠桌"
+	desc = "一张折叠桌，适合搭建临时工作台。"
 	icon = 'icons/roguetown/misc/gadgets.dmi'
 	icon_state = "foldingTableStored"
 	w_class = WEIGHT_CLASS_SMALL
@@ -39,7 +39,7 @@
 	//deploy the table if the user clicks on it with an open turf in front of them
 	var/turf/target_turf = get_step(user,user.dir)
 	if(target_turf.is_blocked_turf(TRUE) || (locate(/mob/living) in target_turf))
-		to_chat(user, span_danger("I can't deploy the [name] here!"))
+		to_chat(user, span_danger("我不能在这里展开[name]！"))
 		return NONE
 	if(isopenturf(target_turf))
 		deploy_folding_table(user, target_turf)
@@ -48,28 +48,28 @@
 
 /obj/item/folding_table_stored/proc/deploy_folding_table(mob/user, atom/location)
 	if(do_after(user, 3 SECONDS))
-		to_chat(user, "<span class='notice'>You deploy the [name].</span>")
+		to_chat(user, "<span class='notice'>你展开了[name]。</span>")
 		new unfolded_structure(location)
 		qdel(src)
 
 /obj/item/folding_table_stored/alchstation
-	name = "alchemical station kit"
-	desc = "A compact portable laboratory that stores everything necessary for the activities of a wandering alchemist."
+	name = "炼金站工具包"
+	desc = "一套紧凑的便携式实验工具，收纳了流浪炼金术士活动所需的一切。"
 	icon_state = "foldingAlchstationStored"
 	grid_height = 64
 	grid_width = 64
 	unfolded_structure = /obj/structure/fluff/alch/folding
 
 /obj/item/folding_table_stored/alchcauldron
-	name = "folding cauldron"
-	desc = "A folding table, useful for setting up a temporary workspace."
+	name = "折叠坩埚架"
+	desc = "一张折叠桌，适合搭建临时工作台。"
 	icon_state = "FoldingCauldronStored"
 	grid_height = 64
 	grid_width = 64
 	unfolded_structure = /obj/machinery/light/rogue/cauldron/folding
 
 /datum/anvil_recipe/engineering/construct_skill_core
-	name = "Golem Skill Exhibitor"
+	name = "魔像技能展示器"
 	created_item = /obj/item/construct_skill_core
 	req_bar = /obj/item/ingot/copper
 	additional_items = list(/obj/item/roguegear, /obj/item/roguegear)
