diff --git a/code/modules/roguetown/roguejobs/fisher/leeches.dm b/code/modules/roguetown/roguejobs/fisher/leeches.dm
--- a/code/modules/roguetown/roguejobs/fisher/leeches.dm
+++ b/code/modules/roguetown/roguejobs/fisher/leeches.dm
@@ -1,8 +1,8 @@
 #define MAX_LEECH_EVILNESS 10
 
 /obj/item/natural/worms/leech
-	name = "leech"
-	desc = "A disgusting, blood-sucking parasite."
+	name = "水蛭"
+	desc = "一种恶心的吸血寄生虫。"
 	icon = 'icons/roguetown/items/surgery.dmi'
 	icon_state = "leech"
 	dropshrink = 0.8
@@ -126,34 +126,34 @@
 	. = ..()
 	switch(blood_storage/blood_maximum)
 		if(0.8 to INFINITY)
-			. += span_bloody("<B>[p_theyre(TRUE)] fat and engorged with blood.</B>")
+			. += span_bloody("<B>[p_theyre(TRUE)]肥硕鼓胀，吸满了鲜血。</B>")
 		if(0.5 to 0.8)
-			. += span_bloody("[p_theyre(TRUE)] well fed.")
+			. += span_bloody("[p_theyre(TRUE)]吃得很饱。")
 		if(0.1 to 0.5)
-			. += span_warning("[p_they(TRUE)] want[p_s()] a meal.")
+			. += span_warning("[p_they(TRUE)]想进食了。")
 		if(-INFINITY to 0.1)
-			. += span_dead("[p_theyre(TRUE)] starved.")
+			. += span_dead("[p_theyre(TRUE)]饿死了。")
 	if(!giving)
-		. += span_warning("[p_theyre(TRUE)] [pick("slurping", "sucking", "inhaling")].")
+		. += span_warning("[p_theyre(TRUE)] [pick("吮吸", "吸食", "猛吸")].")
 	else
-		. += span_notice("[p_theyre(TRUE)] [pick("vomiting", "gorfing", "exhaling")].")
+		. += span_notice("[p_theyre(TRUE)] [pick("呕吐", "反刍", "喷吐")].")
 	if(drainage)
 		START_PROCESSING(SSobj, src)
 
 /obj/item/natural/worms/leech/attack(mob/living/M, mob/user)
 	if(ishuman(M))
 		if(!giving && M.stat == DEAD)
-			to_chat(user, span_warning("They are deceased. Only running blood may be extracted."))
+			to_chat(user, span_warning("它们已经死了。只有流动的血液才能被抽取。"))
 			return
 		if(!giving && !M.mind && !mindless_attach)
-			to_chat(user, span_warning("They are mindless. The [src] won't attach."))
+			to_chat(user, span_warning("对方毫无意识，[src]不会附着上去。"))
 			return
 		var/mob/living/carbon/human/H = M
 		var/obj/item/bodypart/affecting = H.get_bodypart(check_zone(user.zone_selected))
 		if(!affecting)
 			return
 		if(!get_location_accessible(H, check_zone(user.zone_selected)))
-			to_chat(user, span_warning("Something in the way."))
+			to_chat(user, span_warning("有东西挡着。"))
 			return
 		var/used_time = (70 - (user.get_skill_level(/datum/skill/misc/medicine) * 10))/2
 		if(!do_mob(user, H, used_time))
@@ -164,9 +164,9 @@
 		src.forceMove(H)
 		affecting.add_embedded_object(src, silent = TRUE, crit_message = FALSE)
 		if(M == user)
-			user.visible_message(span_notice("[user] places [src] on [user.p_their()] [affecting]."), span_notice("I place a leech on my [affecting]."))
+			user.visible_message(span_notice("[user] places [src] on [user.p_their()] [affecting]."), span_notice("我把一只水蛭放到了自己的[affecting]上。"))
 		else
-			user.visible_message(span_notice("[user] places [src] on [M]'s [affecting]."), span_notice("I place a leech on [M]'s [affecting]."))
+			user.visible_message(span_notice("[user] places [src] on [M]'s [affecting]."), span_notice("我把一只水蛭放到了[M]的[affecting]上。"))
 		return
 	return ..()
 
@@ -214,14 +214,14 @@
 		if(MAX_LEECH_EVILNESS to INFINITY) //maximized evilness holy shit
 			color = "#ff0000"
 			adjectives += pick("evil", "malevolent", "misanthropic")
-			descs += span_danger("This one is bursting with hatred!")
+			descs += span_danger("这只充满了仇恨！")
 		if(5) //this leech is painfully average, it gets no adjectives
 			if(prob(3))
 				adjectives += pick("average", "ordinary", "boring")
 				descs += "This one is extremely boring to look at."
 		if(-INFINITY to 1) //this leech is pretty terrible at being a leech
 			adjectives += pick("pitiful", "pathetic", "depressing")
-			descs += span_dead("This one yearns for nothing but death.")
+			descs += span_dead("这只一心求死。")
 		else
 			var/adjective_amount = 1
 			if(prob(5))
@@ -246,8 +246,8 @@
 	return TRUE
 
 /obj/item/natural/worms/leech/cheele
-	name = "cheele"
-	desc = "A beautiful, blood-infusing altruistic organism made by Pestra herself."
+	name = "奇勒"
+	desc = "一只由佩丝特拉亲手造出的美丽生物，会无私地向宿主注入鲜血。"
 	icon_state = "cheele"
 	color = null
 	consistent = TRUE
@@ -273,10 +273,10 @@
 	giving = !giving
 	if(giving)
 		user.visible_message(span_notice("[user] squeezes [src]."),\
-							span_notice("I squeeze [src]. It will now infuse blood."))
+							span_notice("我挤压了[src]。它现在会注入血液。"))
 	else
 		user.visible_message(span_notice("[user] squeezes [src]."),\
-							span_notice("I squeeze [src]. It will now extract blood."))
+							span_notice("我挤压了[src]。它现在会抽取血液。"))
 
 #undef MAX_LEECH_EVILNESS
 
@@ -284,8 +284,8 @@
 	return
 
 /obj/item/natural/worms/leech/abyssoid
-	name = "abyssoid leech"
-	desc = "A holy leech sent by Abyssor himself."
+	name = "深渊水蛭"
+	desc = "由阿比索尔亲自降下的神圣水蛭。"
 	icon_state = "leech"
 	drainage = 0
 	blood_sucking = 0
