--- a/code/modules/roguetown/roguemachine/merchant/navigator.dm
+++ b/code/modules/roguetown/roguemachine/merchant/navigator.dm
@@ -2,8 +2,8 @@
 #define EXPORT_TIME_TESTING 5 SECONDS
 
 /obj/item/roguemachine/navigator
-	name = "navigator"
-	desc = "A machine that attracts the attention of trading balloons."
+	name = "引航机"
+	desc = "一台会吸引贸易气球注意的机器。"
 	icon = 'icons/roguetown/misc/machines.dmi'
 	icon_state = "ballooner"
 	density = TRUE
@@ -15,7 +15,7 @@
 	/// A fixed tax on all items sold through the balloon that overrides queens tax. Used for blackmarket
 	var/fixed_tax = 0
 	/// Motto displayed at the top of the vendor interface
-	var/motto = "NAVIGATOR - Your goods, airborne."
+	var/motto = "引航机 - 让你的货物飞上天空。"
 
 /obj/item/roguemachine/navigator/examine()
 	. = ..()
@@ -23,11 +23,11 @@
 	#ifdef LOCALTEST
 	export_time = EXPORT_TIME_TESTING
 	#endif
-	. += span_notice("This machine attracts trading balloons every [DisplayTimeText(export_time)]. Goods are sucked into the air and mammons are dropped after tax has been collected.")
+	. += span_notice("这台机器每隔 [DisplayTimeText(export_time)] 就会吸引来贸易气球。货物会被吸上天空，在扣税后投下玛门。")
 
 /obj/item/roguemachine/navigator/blackmarket
-	name = "suspicious navigator"
-	desc = "Freedom has a price."
+	name = "可疑引航机"
+	desc = "自由是有代价的。"
 	motto = "NA?!G@#OR - ████ ██████ █████████ - FREEDOM OF TRANSACTION."
 	fixed_tax = 0.5 // 50% taxation and rip off to encourage people to risk it with merchant / others
 
@@ -108,7 +108,7 @@
 				if(prize >= 1)
 					play_sound=TRUE
 					budgie += prize
-					I.visible_message(span_warning("[I] is sucked into the air!"))
+					I.visible_message(span_warning("[I]被吸上了天空！"))
 					qdel(I)
 			budgie = round(budgie)
 			record_round_statistic(STATS_TRADE_VALUE_EXPORTED, budgie)
