diff --git a/code/modules/roguetown/roguejobs/artificer/artificer_recipes/artificer_recipes.dm b/code/modules/roguetown/roguejobs/artificer/artificer_recipes/artificer_recipes.dm
--- a/code/modules/roguetown/roguejobs/artificer/artificer_recipes/artificer_recipes.dm
+++ b/code/modules/roguetown/roguejobs/artificer/artificer_recipes/artificer_recipes.dm
@@ -25,12 +25,12 @@
 		return
 	if(hammers_per_item == 0)
 		hammered = TRUE
-		user.visible_message(span_warning("[user] hammers the contraption."))
+		user.visible_message(span_warning("[user]敲打着这件装置。"))
 		if(additional_items.len)
 			needed_item = pick(additional_items)
 			additional_items -= needed_item
 		if(needed_item)
-			to_chat(user, span_info("Now it's time to add \a [initial(needed_item.name)]."))
+			to_chat(user, span_info("现在该加入[initial(needed_item.name)]了。"))
 			return
 	if(!needed_item && hammered)
 		progress = 100
@@ -45,11 +45,11 @@
 				hammers_per_item = max(0, hammers_per_item -= 2)
 			if(SKILL_LEVEL_LEGENDARY to INFINITY)
 				hammers_per_item = max(0, hammers_per_item -= 3)
-		user.visible_message(span_warning("[user] hammers the contraption."))
+		user.visible_message(span_warning("[user]敲打着这件装置。"))
 		return
 
 /datum/artificer_recipe/proc/item_added(mob/user)
-	user.visible_message(span_info("[user] adds [initial(needed_item.name)]."))
+	user.visible_message(span_info("[user]加入了[initial(needed_item.name)]。"))
 	if(istype(needed_item, /obj/item/natural/wood/plank))
 		playsound(user, 'sound/misc/wood_saw.ogg', 100, TRUE)
 	needed_item = null
@@ -65,7 +65,7 @@
 	i_type = "General"
 
 /datum/artificer_recipe/wood //TNevermind this being silly, I was silly and this needs to be redone proper
-	name = "Wooden Cog"
+	name = "木制齿轮"
 	required_item = /obj/item/natural/wood/plank
 	created_item = /obj/item/roguegear/wood/basic
 	hammers_per_item = 5
@@ -73,21 +73,21 @@
 	i_type = "General"
 
 /datum/artificer_recipe/wood/reliable
-	name = "Reliable Wooden Cog (+1 Essence of Lumber)"
+	name = "可靠木制齿轮 (+1 Essence of Lumber)"
 	created_item = /obj/item/roguegear/wood/reliable
 	additional_items = list(/obj/item/grown/log/tree/small/essence = 1)
 	hammers_per_item = 10
 	skill_level = 2
 
 /datum/artificer_recipe/wood/unstable
-	name = "Unstable Wooden Cog (+1 Essence of Wilderness)"
+	name = "不稳定木制齿轮 (+1 Essence of Wilderness)"
 	created_item = /obj/item/roguegear/wood/unstable
 	additional_items = list(/obj/item/natural/cured/essence = 1)
 	hammers_per_item = 10
 	skill_level = 3
 
 /datum/artificer_recipe/bronze
-	name = "Bronze Cog 2x"
+	name = "青铜 Cog 2x"
 	required_item = /obj/item/ingot/bronze
 	created_item = list(/obj/item/roguegear/bronze,/obj/item/roguegear/bronze)
 	hammers_per_item = 10
@@ -98,14 +98,14 @@
 	i_type = "General"
 
 /datum/artificer_recipe/iron/locks
-	name = "Locks 3x"
+	name = "锁 3x"
 	required_item = /obj/item/ingot/iron
 	created_item = list(/obj/item/customlock, /obj/item/customlock, /obj/item/customlock)
 	hammers_per_item = 5
 	skill_level = 1
 
 /datum/artificer_recipe/iron/keys
-	name = "Keys 3x"
+	name = "钥匙 3x"
 	required_item = /obj/item/ingot/iron
 	created_item = list(/obj/item/customblank, /obj/item/customblank, /obj/item/customblank)
 	hammers_per_item = 5
@@ -113,14 +113,14 @@
 
 
 /datum/artificer_recipe/iron/lockpicks
-	name = "Lockpick (x3)"
+	name = "开锁器 (x3)"
 	required_item = /obj/item/ingot/iron
 	created_item = list(/obj/item/lockpick,/obj/item/lockpick,/obj/item/lockpick)
 	hammers_per_item = 5
 	skill_level = 2
 
 /datum/artificer_recipe/iron/lockpickring
-	name = "Lockpickring (x3)"
+	name = "开锁环 (x3)"
 	required_item = /obj/item/ingot/iron
 	created_item = list(/obj/item/lockpickring,/obj/item/lockpickring,/obj/item/lockpickring)
 	hammers_per_item = 5
@@ -135,33 +135,33 @@
 	i_type = "Tools"
 
 /datum/artificer_recipe/bronze/tools
-	name = "Bronze Lamptern 3x"
+	name = "青铜 提灯 3x"
 	created_item = list(/obj/item/flashlight/flare/torch/lantern/bronzelamptern,/obj/item/flashlight/flare/torch/lantern/bronzelamptern,/obj/item/flashlight/flare/torch/lantern/bronzelamptern)
 	hammers_per_item = 9
 	skill_level = 3
 	i_type = "Tools"
 
 /datum/artificer_recipe/bronze/tools/wrench
-	name = "Engineering Wrench (+1 cog)"
+	name = "工程扳手 (+1 cog)"
 	created_item = /obj/item/contraption/linker
 	additional_items = list(/obj/item/roguegear/bronze)
 	skill_level = 0
 	hammers_per_item = 5
 
 /datum/artificer_recipe/bronze/tools/headhook
-	name = "Headhook (+2 Fibers)"
+	name = "头钩 (+2 纤维s)"
 	created_item = /obj/item/storage/hip/headhook/bronze
 	additional_items = list(/obj/item/natural/fibers, /obj/item/natural/fibers)
 	skill_level = 3
 
 /datum/artificer_recipe/bronze/tools/waterpurifier
-	name = "Self-Purifying Waterskin (+1 waterskin)"
+	name = "自净水袋 (+1 waterskin)"
 	required_item = /obj/item/ingot/bronze
 	created_item = /obj/item/reagent_containers/glass/bottle/waterskin/purifier
 	additional_items = list(/obj/item/reagent_containers/glass/bottle/waterskin)
 
 /datum/artificer_recipe/bronze/tools/grappler
-	name = "Grappler (+1 ironpick)(+2 bronze gears)(+1 rope)"
+	name = "抓钩发射器 (+1 ironpick)(+2 bronze gears)(+1 rope)"
 	required_item = /obj/item/ingot/bronze
 	created_item = /obj/item/grapplinghook
 	additional_items = list(/obj/item/rogueweapon/pick, /obj/item/roguegear/bronze, /obj/item/roguegear/bronze, /obj/item/rope)
@@ -173,35 +173,35 @@
 	i_type = "Contraptions"
 
 /datum/artificer_recipe/contraptions/serfstone
-	name = "Serf Stone (+1 cog, +1 Topar)"
+	name = "农奴石 (+1 cog, +1 Topar)"
 	required_item = /obj/item/ingot/steel
 	additional_items = list(/obj/item/roguegear/bronze, /obj/item/roguegem/yellow) //using topar since the description calls it a "dull gem"
 	created_item = /obj/item/scomstone/bad
 	skill_level = 5
 
 /datum/artificer_recipe/contraptions/houndstone
-	name = "Houndstone (+1 cog, +1 Topar)"
+	name = "猎犬石 (+1 cog, +1 Topar)"
 	required_item = /obj/item/ingot/steel
 	additional_items = list(/obj/item/roguegear/bronze, /obj/item/roguegem/yellow)
 	created_item = /obj/item/scomstone/bad/garrison
 	skill_level = 5
 
 /datum/artificer_recipe/contraptions/scomstone
-	name = "SCOM Stone (+1 cog, +1 Gemerald, Arcyne)"
+	name = "SCOM 石 (+1 cog, +1 Gemerald, Arcyne)"
 	required_item = /obj/item/ingot/gold
 	additional_items = list(/obj/item/roguegear/bronze, /obj/item/roguegem/green)
 	created_item = /obj/item/scomstone
 	skill_level = 5
 
 /datum/artificer_recipe/contraptions/emeraldchoker
-	name = "emerald choker (+1 cog, +Gold, +1 Gemerald, Arcyne)"
+	name = "绿宝石项圈 (+1 cog, +黄金, +1 Gemerald, Arcyne)"
 	required_item = /obj/item/ingot/gold
 	additional_items = list(/obj/item/roguegear/bronze, /obj/item/ingot/gold, /obj/item/roguegem/green)
 	created_item = /obj/item/listenstone
 	skill_level = 5
 
 /datum/artificer_recipe/contraptions/folding_table
-	name = "Folding Table (+1 Small Log)"
+	name = "折叠桌 (+1 小原木)"
 	required_item = /obj/item/ingot/bronze
 	additional_items = list(/obj/item/grown/log/tree/small)
 	created_item = /obj/item/contraption/folding_table_stored
@@ -209,14 +209,14 @@
 	skill_level = 1
 
 /datum/artificer_recipe/contraptions/folding_alchcauldron
-	name = "Folding Cauldron (+1 Small Log, +Stone Pot, +Tin)"
+	name = "折叠坩埚架 (+1 小原木, +Stone Pot, +Tin)"
 	required_item = /obj/item/ingot/bronze
 	additional_items = list(/obj/item/grown/log/tree/small, /obj/item/reagent_containers/glass/bucket/pot/stone, /obj/item/ingot/tin)
 	created_item = /obj/item/folding_table_stored/alchcauldron
 	skill_level = 3
 
 /datum/artificer_recipe/contraptions/folding_alchstation_stored
-	name = "Alchemical Station Kit (+2 Small Log, +Bottle, +Cog)"
+	name = "炼金站工具包 (+2 小原木, +Bottle, +Cog)"
 	required_item = /obj/item/ingot/bronze
 	additional_items = list(/obj/item/grown/log/tree/small, /obj/item/grown/log/tree/small, /obj/item/reagent_containers/glass/bottle, /obj/item/roguegear)
 	created_item = /obj/item/folding_table_stored/alchstation
@@ -239,7 +239,7 @@
 	skill_level = 3
 
 /datum/artificer_recipe/contraptions/shears
-	name = "Amputation Shears (+2 Bronze)"
+	name = "截肢剪 (+2 青铜)"
 	required_item = /obj/item/ingot/bronze
 	additional_items = list(/obj/item/ingot/bronze = 2)
 	created_item = /obj/item/contraption/shears
@@ -247,7 +247,7 @@
 	skill_level = 4
 
 /datum/artificer_recipe/contraptions/metalizer
-	name = "Wood Metalizer (+1 Wooden Cog)"
+	name = "木转金属器 (+1 木制齿轮)"
 	required_item = /obj/item/ingot/bronze
 	additional_items = list(/obj/item/roguegear/wood/basic = 1)
 	created_item = /obj/item/contraption/wood_metalizer
@@ -255,7 +255,7 @@
 	skill_level = 4
 
 /datum/artificer_recipe/contraptions/smelter
-	name = "Portable Smelter (+1 Coal)"
+	name = "便携熔炉 (+1 煤炭)"
 	required_item = /obj/item/ingot/bronze
 	additional_items = list(/obj/item/rogueore/coal = 1)
 	created_item = /obj/item/contraption/smelter
@@ -263,7 +263,7 @@
 	skill_level = 3
 
 /datum/artificer_recipe/contraptions/imprinter
-	name = "Lock Imprinter (+1 Reliable Wooden Cog)"
+	name = "锁印器 (+1 可靠木制齿轮)"
 	required_item = /obj/item/ingot/bronze
 	additional_items = list(/obj/item/roguegear/wood/reliable = 1)
 	created_item = /obj/item/contraption/lock_imprinter
@@ -272,7 +272,7 @@
 
 
 /datum/artificer_recipe/contraptions/smokebombs
-	name = "smokebomb(s) (3x) (+1 Coal)"
+	name = "烟雾弹 (3x) (+1 煤炭)"
 	required_item = /obj/item/ingot/iron
 	additional_items = list(/obj/item/rogueore/coal = 1)
 	created_item = list(/obj/item/bomb/smoke,
@@ -282,7 +282,7 @@
 	skill_level = 3
 
 /datum/artificer_recipe/contraptions/orestore
-	name = "Mechanized Ore Bag, Bronze (+1 sac, +1 cog)"
+	name = "机械矿袋, 青铜 (+1 sac, +1 cog)"
 	required_item = /obj/item/ingot/bronze
 	created_item = /obj/item/storage/hip/orestore/bronze
 	hammers_per_item = 12
@@ -290,7 +290,7 @@
 	skill_level = 3
 
 /datum/artificer_recipe/contraptions/artificerarmor
-	name = "Artificer armor (+3 Gilbranze, +2 Bronze Gear)"
+	name = "工匠护甲 (+3 Gilbranze, +2 青铜 Gear)"
 	required_item = /obj/item/ingot/gilbranze
 	additional_items = list(/obj/item/ingot/gilbranze,/obj/item/ingot/gilbranze, /obj/item/roguegear/bronze,/obj/item/roguegear/bronze)
 	created_item = /obj/item/clothing/suit/roguetown/armor/plate/ancient/artificer
@@ -298,7 +298,7 @@
 	skill_level = 4
 
 /datum/artificer_recipe/contraptions/volticgauntlet
-	name = "Voltic Gauntlet (+1 Tin ingot)(+2 Bronze gear)(+1 cinnabar ore)"
+	name = "伏特护手 (+1 Tin ingot)(+2 青铜 gear)(+1 cinnabar ore)"
 	required_item = /obj/item/ingot/bronze
 	additional_items = list(/obj/item/roguegear/bronze, /obj/item/roguegear/bronze,/obj/item/ingot/tin, /obj/item/rogueore/cinnabar)
 	created_item = /obj/item/clothing/gloves/roguetown/contraption/voltic
@@ -313,7 +313,7 @@
 	skill_level = 5
 
 /datum/artificer_recipe/contraptions/cursed_collar // lets people make the actual cursed collar, but gates it behind semi rare mats and engineering 
-	name = "True Cursed Collar (+1 Bronze Cog, +1 Essence of Wilderness)"
+	name = "真诅咒项圈 (+1 青铜 Cog, +1 Essence of Wilderness)"
 	required_item = /obj/item/ingot/steel
 	additional_items = list(/obj/item/roguegear/bronze, /obj/item/natural/cured/essence)
 	created_item = /obj/item/clothing/neck/roguetown/cursed_collar
@@ -322,27 +322,27 @@
 // --------- WEAPON -----------
 
 /datum/artificer_recipe/wood/weapons //Again, a bit silly, but is important
-	name = "Wooden Staff (+1 Plank)"
+	name = "木杖 (+1 木板)"
 	created_item = /obj/item/rogueweapon/woodstaff
 	additional_items = list(/obj/item/natural/wood/plank = 1)
 	hammers_per_item = 3
 	i_type = "Weapons"
 
 /datum/artificer_recipe/wood/weapons/bow // easier recipe for bows
-	name = "Wooden Bow (+1 Fiber) (+1 Plank)"
+	name = "木弓 (+1 纤维) (+1 木板)"
 	created_item = /obj/item/gun/ballistic/revolver/grenadelauncher/bow
 	hammers_per_item = 3
 	additional_items = list(/obj/item/natural/wood/plank = 1, /obj/item/natural/fibers = 1)
 
 /datum/artificer_recipe/wood/weapons/wsword
-	name = "Wooden Sword (+1 Plank)"
+	name = "木剑 (+1 木板)"
 	created_item = /obj/item/rogueweapon/mace/wsword
 	additional_items = list(/obj/item/natural/wood/plank = 1)
 	hammers_per_item = 3
 
 /datum/artificer_recipe/wood/weapons/wdagger
 
-	name = "Wooden Dagger (3x) (+1 Plank)"
+	name = "木匕首 (3x) (+1 木板)"
 	created_item = list(/obj/item/rogueweapon/huntingknife/idagger/wood,
 						/obj/item/rogueweapon/huntingknife/idagger/wood,
 						/obj/item/rogueweapon/huntingknife/idagger/wood
@@ -351,7 +351,7 @@
 	hammers_per_item = 2
 
 /datum/artificer_recipe/wood/weapons/wshield
-	name = "Wooden Shield (+1 Plank)"
+	name = "木盾 (+1 木板)"
 	created_item = /obj/item/rogueweapon/shield/wood/crafted
 	additional_items = list(/obj/item/natural/wood/plank = 1)
 	hammers_per_item = 6
@@ -361,7 +361,7 @@
 	sellprice = 6
 
 /datum/artificer_recipe/wood/weapons/hshield
-	name = "Heater Shield (+1 Cured Leather)"
+	name = "鸢形盾 (+1 熟皮)"
 	created_item = /obj/item/rogueweapon/shield/heater/crafted
 	additional_items = list(/obj/item/natural/wood/plank = 1, /obj/item/natural/hide/cured = 1)
 	hammers_per_item = 6
@@ -371,7 +371,7 @@
 	sellprice = 6
 
 /datum/artificer_recipe/wood/weapons/steamshield
-	name = "Steam Shield (+1 wood plank)(+2 Bronze gear)(+2 bronze ingot)"
+	name = "蒸汽盾 (+1 wood plank)(+2 青铜 gear)(+2 bronze ingot)"
 	additional_items = list(/obj/item/roguegear/bronze, /obj/item/roguegear/bronze,/obj/item/natural/wood/plank, /obj/item/ingot/bronze,/obj/item/ingot/bronze)
 	created_item = /obj/item/rogueweapon/shield/steam
 	hammers_per_item = 12
@@ -380,7 +380,7 @@
 /// CROSSBOW
 
 /datum/artificer_recipe/wood/weapons/crossbow
-	name = "Crossbow (+1 Steel) (+1 Fiber)"
+	name = "弩 (+1 钢) (+1 纤维)"
 	created_item = /obj/item/gun/ballistic/revolver/grenadelauncher/crossbow
 	additional_items = list(/obj/item/ingot/steel, /obj/item/natural/fibers)
 	hammers_per_item = 10
@@ -392,7 +392,7 @@
 	i_type = "Ammunition"
 
 /datum/artificer_recipe/ammunition/bolts
-	name = "Crossbow Bolts 20x (+2 planks, +1 Iron)"
+	name = "弩箭 20x (+2 planks, +1 铁)"
 	required_item = /obj/item/natural/wood/plank
 	additional_items = list(/obj/item/natural/wood/plank, /obj/item/natural/wood/plank, /obj/item/ingot/iron)
 	created_item = list(/obj/item/ammo_casing/caseless/rogue/bolt,
@@ -420,7 +420,7 @@
 	skill_level = 2
 
 /datum/artificer_recipe/ammunition/arrows
-	name = "Arrows 20x (+2 Planks, +1 Iron)"
+	name = "Arrows 20x (+2 木板s, +1 铁)"
 	required_item = /obj/item/natural/wood/plank
 	additional_items = list(/obj/item/natural/wood/plank, /obj/item/natural/wood/plank,  /obj/item/ingot/iron)
 	created_item = list(/obj/item/ammo_casing/caseless/rogue/arrow/iron,
@@ -448,7 +448,7 @@
 	skill_level = 2
 
 /datum/artificer_recipe/ammunition/pyrobolt_five
-	name = "pyroclastic bolt x5 (+1 iron) (+1 fyritius)"
+	name = "火碎弩箭 x5 (+1 iron) (+1 fyritius)"
 	required_item = /obj/item/natural/wood/plank
 	additional_items = list(/obj/item/ingot/iron, /obj/item/reagent_containers/food/snacks/grown/rogue/fyritius)
 	created_item = list(
@@ -463,7 +463,7 @@
 
 
 /datum/artificer_recipe/ammunition/pyroarrow_five
-	name = "pyroclastic arrow x5 (+1 iron) (+1 fyritius)"
+	name = "火碎箭 x5 (+1 iron) (+1 fyritius)"
 	required_item = /obj/item/natural/wood/plank
 	additional_items = list(/obj/item/ingot/iron, /obj/item/reagent_containers/food/snacks/grown/rogue/fyritius)
 	created_item = list(
@@ -477,7 +477,7 @@
 	skill_level = 2
 
 /datum/artificer_recipe/ammunition/lead_ball
-	name = "lead ball x8 (+2 iron)"
+	name = "铅弹 x8 (+2 iron)"
 	required_item = /obj/item/ingot/iron
 	additional_items = list(/obj/item/ingot/iron, /obj/item/ingot/iron)
 	created_item = list(//Enough for a pouch.
@@ -494,7 +494,7 @@
 	skill_level = 4
 
 /datum/artificer_recipe/ammunition/grapeshot
-	name = "grapeshot x8 (+3 iron)"
+	name = "葡萄弹 x8 (+3 iron)"
 	required_item = /obj/item/ingot/iron
 	additional_items = list(/obj/item/ingot/iron, /obj/item/ingot/iron, /obj/item/ingot/iron)
 	created_item = list(
@@ -516,7 +516,7 @@
 	i_type = "Prosthetics"
 
 /datum/artificer_recipe/prosthetics/wood/arm_left
-	name = "Left Wooden Arm (+2 Plank) (+1 Wooden Cog)"
+	name = "Left Wooden Arm (+2 木板) (+1 木制齿轮)"
 	required_item = /obj/item/natural/wood/plank
 	additional_items = list(/obj/item/natural/wood/plank = 2, /obj/item/roguegear/wood/basic = 1)
 	created_item = /obj/item/bodypart/l_arm/prosthetic/woodleft
@@ -524,7 +524,7 @@
 	skill_level = 2
 
 /datum/artificer_recipe/prosthetics/wood/arm_right
-	name = "Right Wooden Arm (+2 Plank) (+1 Wooden Cog)"
+	name = "Right Wooden Arm (+2 木板) (+1 木制齿轮)"
 	required_item = /obj/item/natural/wood/plank
 	additional_items = list(/obj/item/natural/wood/plank = 2, /obj/item/roguegear/wood/basic = 1)
 	created_item = /obj/item/bodypart/r_arm/prosthetic/woodright
@@ -532,7 +532,7 @@
 	skill_level = 2
 
 /datum/artificer_recipe/prosthetics/wood/leg_left
-	name = "Left Wooden Leg (+2 Plank) (+1 Wooden Cog)"
+	name = "Left Wooden Leg (+2 木板) (+1 木制齿轮)"
 	required_item = /obj/item/natural/wood/plank
 	additional_items = list(/obj/item/natural/wood/plank = 2, /obj/item/roguegear/wood/basic = 1)
 	created_item = /obj/item/bodypart/l_leg/prosthetic
@@ -540,7 +540,7 @@
 	skill_level = 2
 
 /datum/artificer_recipe/prosthetics/wood/leg_right
-	name = "Right Wooden Leg (+2 Plank) (+1 Wooden Cog)"
+	name = "Right Wooden Leg (+2 木板) (+1 木制齿轮)"
 	required_item = /obj/item/natural/wood/plank
 	additional_items = list(/obj/item/natural/wood/plank = 2, /obj/item/roguegear/wood/basic = 1)
 	created_item = /obj/item/bodypart/r_leg/prosthetic
@@ -550,7 +550,7 @@
 // --------- BRONZE -----------
 
 /datum/artificer_recipe/bronze/prosthetic
-	name = "Bronze Prosthetic (+1 Cog)"
+	name = "青铜义肢 (+1 Cog)"
 	created_item = /obj/item/contraption/bronzeprosthetic
 	hammers_per_item = 15
 	skill_level = 4
@@ -560,7 +560,7 @@
 // --------- GOLD -----------
 
 /datum/artificer_recipe/gold/prosthetic
-	name = "Gold Prosthetic (+2 Cog)"
+	name = "金制义肢 (+2 Cog)"
 	required_item = /obj/item/ingot/gold
 	created_item = /obj/item/contraption/goldprosthetic
 	additional_items = list(/obj/item/roguegear/bronze,/obj/item/roguegear/bronze)
@@ -571,7 +571,7 @@
 // --------- STEEL -----------
 
 /datum/artificer_recipe/steel/prosthetic
-	name = "Steel Prosthetic (+1 Steel, +1 Cog)"
+	name = "钢制义肢 (+1 钢, +1 Cog)"
 	created_item = /obj/item/contraption/steelprosthetic
 	required_item = /obj/item/ingot/steel
 	additional_items = list(/obj/item/ingot/steel = 1, /obj/item/roguegear/bronze = 1)
@@ -582,7 +582,7 @@
 // --------- IRON -----------
 
 /datum/artificer_recipe/iron/prosthetic //These are the inexpensive alternatives
-	name = "Iron Prosthetic (+1 Plank) (+1 Cog)"
+	name = "铁制义肢 (+1 木板) (+1 Cog)"
 	created_item = /obj/item/contraption/ironprosthetic
 	required_item = /obj/item/ingot/iron
 	additional_items = list(/obj/item/natural/wood/plank = 1, /obj/item/roguegear/bronze = 1)
@@ -596,7 +596,7 @@
 
 
 /datum/artificer_recipe/general/tntbomb
-	name = "Gun powder sticks"
+	name = "火药棒"
 	required_item = /obj/item/rogueore/coal
 	additional_items = list(/obj/item/paper/scroll,
 							/obj/item/alch/coaldust,
@@ -610,7 +610,7 @@
 	skill_level = 4
 
 /datum/artificer_recipe/general/satchelbomb
-	name = "Gun powder satchel"
+	name = "火药包"
 	required_item = /obj/item/storage/backpack/rogue/satchel
 	additional_items = list(/obj/item/tntstick,
 							/obj/item/tntstick,
@@ -620,7 +620,7 @@
 	skill_level = 5
 
 /datum/artificer_recipe/general/smokebomb
-	name = "Gas belcher shells (x3) (+Cog)"
+	name = "喷气枪弹 (x3) (+Cog)"
 	required_item = /obj/item/ingot/bronze
 	additional_items = list(/obj/item/roguegear)
 	created_item = list(/obj/item/smokeshell,
