From 6b4bd6676ce6dcaee2b580a50e6ebb956ce1b78d Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Sat, 30 May 2026 13:00:00 +0800
Subject: [PATCH] 翻译 code/modules/mob/living/bloodcrawl.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
diff --git a/code/modules/mob/living/bloodcrawl.dm b/code/modules/mob/living/bloodcrawl.dm
--- a/code/modules/mob/living/bloodcrawl.dm
+++ b/code/modules/mob/living/bloodcrawl.dm
@@ -24,7 +24,7 @@
 			//TODO make it toggleable to either forcedrop the items, or deny
 			//entry when holding them
 			// literally only an option for carbons though
-			to_chat(C, span_warning("I may not hold items while blood crawling!"))
+			to_chat(C, span_warning("血行时我不能拿着物品！"))
 			return FALSE
 		var/obj/item/bloodcrawl/B1 = new(C)
 		var/obj/item/bloodcrawl/B2 = new(C)
@@ -42,7 +42,7 @@
 /mob/living/proc/bloodpool_sink(obj/effect/decal/cleanable/B)
 	var/turf/mobloc = get_turf(loc)
 
-	visible_message(span_warning("[src] sinks into the pool of blood!"))
+	visible_message(span_warning("[src]沉入了血泊之中！"))
 	playsound(get_turf(src), 'sound/blank.ogg', 50, TRUE, -1)
 	// Extinguish, unbuckle, stop being pulled, set our location into the
 	// dummy object
@@ -70,23 +70,23 @@
 	var/kidnapped = FALSE
 
 	if(victim.stat == CONSCIOUS)
-		visible_message(span_warning("[victim] kicks free of the blood pool just before entering it!"), null, span_notice("I hear splashing and struggling."))
+		visible_message(span_warning("[victim]在被拖入血泊前猛地挣脱了出来！"), null, span_notice("我听见了扑腾与挣扎声。"))
 	else
 		victim.forceMove(src)
 		victim.emote("scream")
-		visible_message(span_warning("<b>[src] drags [victim] into the pool of blood!</b>"), null, span_notice("I hear a splash."))
+		visible_message(span_warning("<b>[src]将[victim]拖进了血泊之中！</b>"), null, span_notice("我听见了一阵水花声。"))
 		kidnapped = TRUE
 
 	if(kidnapped)
 		var/success = bloodcrawl_consume(victim)
 		if(!success)
-			to_chat(src, span_danger("I happily devour... nothing? Your meal vanished at some point!"))
+			to_chat(src, span_danger("我愉快地吞噬着……什么也没有？你的猎物不知何时消失了！"))
 
 	notransform = FALSE
 	return TRUE
 
 /mob/living/proc/bloodcrawl_consume(mob/living/victim)
-	to_chat(src, span_danger("I begin to feast on [victim]... You can not move while you are doing this."))
+	to_chat(src, span_danger("我开始吞食[victim]……进行此事时我无法移动。"))
 
 	var/sound = 'sound/blank.ogg'
 
@@ -98,24 +98,24 @@
 		return FALSE
 
 	if(victim.reagents)
-		to_chat(src, span_warning("<b>AAH! THEIR FLESH! IT BURNS!</b>"))
+		to_chat(src, span_warning("<b>啊啊！祂的血肉在灼烧我！</b>"))
 		adjustBruteLoss(25) //I can't use adjustHealth() here because bloodcrawl affects /mob/living and adjustHealth() only affects simple mobs
 		var/found_bloodpool = FALSE
 		for(var/obj/effect/decal/cleanable/target in range(1,get_turf(victim)))
 			if(target.can_bloodcrawl_in())
 				victim.forceMove(get_turf(target))
-				victim.visible_message(span_warning("[target] violently expels [victim]!"))
+				victim.visible_message(span_warning("[target]猛地将[victim]吐了出来！"))
 				victim.exit_blood_effect(target)
 				found_bloodpool = TRUE
 
 		if(!found_bloodpool)
 			// Fuck it, just eject them, thanks to some split second cleaning
 			victim.forceMove(get_turf(victim))
-			victim.visible_message(span_warning("[victim] appears from nowhere, covered in blood!"))
+			victim.visible_message(span_warning("[victim]浑身浴血，凭空现身！"))
 			victim.exit_blood_effect()
 		return TRUE
 
-	to_chat(src, span_danger("I devour [victim]. Your health is fully restored."))
+	to_chat(src, span_danger("我吞噬了[victim]。我的生命已完全恢复。"))
 	revive(full_heal = TRUE, admin_revive = FALSE)
 
 	// No defib possible after laughter
@@ -128,7 +128,7 @@
 	qdel(victim)
 
 /obj/item/bloodcrawl
-	name = "blood crawl"
+	name = "血行"
 	desc = ""
 	icon = 'icons/effects/blood.dmi'
 	item_flags = ABSTRACT | DROPDEL
@@ -149,16 +149,16 @@
 
 /mob/living/proc/phasein(obj/effect/decal/cleanable/B)
 	if(notransform)
-		to_chat(src, span_warning("Finish eating first!"))
+		to_chat(src, span_warning("先把进食完成再说！"))
 		return FALSE
-	B.visible_message(span_warning("[B] starts to bubble..."))
+	B.visible_message(span_warning("[B]开始冒泡……"))
 	if(!do_after(src, 20, target = B))
 		return
 	if(!B)
 		return
 	forceMove(B.loc)
 	client.eye = src
-	visible_message(span_boldwarning("[src] rises out of the pool of blood!"))
+	visible_message(span_boldwarning("[src]自血泊中升起！"))
 	exit_blood_effect(B)
 	if(iscarbon(src))
 		var/mob/living/carbon/C = src
-- 
Translation Patch Toolkit
