From 7503b5c3ea0a19e7182a750357944ec11e1a0297 Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Sat, 30 May 2026 13:00:00 +0800
Subject: [PATCH] 翻译 code/modules/mob/living/death.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
diff --git a/code/modules/mob/living/death.dm b/code/modules/mob/living/death.dm
--- a/code/modules/mob/living/death.dm
+++ b/code/modules/mob/living/death.dm
@@ -167,9 +167,9 @@
 					continue
 				if (HAS_TRAIT(player, TRAIT_DEATHSIGHT))
 					if (HAS_TRAIT(player, TRAIT_CABAL))
-						to_chat(player, span_warning("I feel the faint passage of disjointed life essence as it flees [locale]."))
+						to_chat(player, span_warning("我感到支离破碎的生命精华正从[locale]淡淡流逝。"))
 					else
-						to_chat(player, span_warning("Veiled whispers herald the Undermaiden's gaze in my mind's eye as it turn towards [locale] for but a brief, singular moment."))
+						to_chat(player, span_warning("朦胧低语在我心眼中预示着冥下侍女的目光，曾于转瞬之间投向[locale]。"))
 	// AZURE EDIT END
 
 	return TRUE
@@ -177,5 +177,5 @@
 /mob/living/proc/prepare_deathsight_message()
 	var/area/A = get_area(src)
 	if(!A)
-		return "a locale wreathed in enigmatic fog" // fallback if we can't find the area somehow??
+		return "一处被神秘迷雾笼罩之地" // fallback if we can't find the area somehow??
 	return A.deathsight_message
-- 
Translation Patch Toolkit
