From 21a0d86575ef72211cae4657a6fc105a369e8b6e Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Sat, 30 May 2026 13:00:00 +0800
Subject: [PATCH] 翻译 code/modules/mob/living/emote.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
diff --git a/code/modules/mob/living/emote.dm b/code/modules/mob/living/emote.dm
--- a/code/modules/mob/living/emote.dm
+++ b/code/modules/mob/living/emote.dm
@@ -18,7 +18,7 @@
 	stat_allowed = list(CONSCIOUS, UNCONSCIOUS)
 
 /mob/living/carbon/human/verb/emote_pray()
-	set name = "Pray"
+	set name = "祈祷"
 	set category = "Emotes"
 
 	emote("pray", intentional = TRUE)
@@ -51,7 +51,7 @@
 					continue
 				//Do they even have the boon/trait? If so, send it and heal them.
 				if (HAS_TRAIT(player, TRAIT_CONVICTION))
-					to_chat(player, span_dead("I hear the passing of whispers, knowledge forbidden to share: <br>[span_info(prayer)]"))
+					to_chat(player, span_dead("我听见耳畔低语流过，那是不可外传的知识：<br>[span_info(prayer)]"))
 					player.apply_status_effect(/datum/status_effect/buff/healing/prayer_power, 6)//GET IT?
 
 	/* admin stuff - tells you the followers name, key, and what patron they follow */
@@ -73,7 +73,7 @@
 	emote_type = EMOTE_VISIBLE
 
 /mob/living/carbon/human/verb/emote_meditate()
-	set name = "Meditate"
+	set name = "冥想"
 	set category = "Emotes"
 
 	emote("meditate", intentional = TRUE)
@@ -82,7 +82,7 @@
 	. = ..()
 	if(do_after(user, 1 MINUTES))
 		user.add_stress(/datum/stressevent/meditation)
-		to_chat(user, span_green("My meditations were rewarding."))
+		to_chat(user, span_green("我的冥想颇有收获。"))
 
 /datum/emote/living/bow
 	key = "bow"
@@ -106,7 +106,7 @@
 			L.add_stress(/datum/stressevent/bowedasnoble)
 
 /mob/living/carbon/human/verb/emote_bow()
-	set name = "Bow"
+	set name = "鞠躬"
 	set category = "Emotes"
 
 	emote("bow", intentional = TRUE, targetted = TRUE)
@@ -120,7 +120,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_burp()
-	set name = "Burp"
+	set name = "打嗝"
 	set category = "Noises"
 
 	emote("burp", intentional = TRUE)
@@ -134,7 +134,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_choke()
-	set name = "Choke"
+	set name = "呛咳"
 	set category = "Noises"
 
 	emote("choke", intentional = TRUE)
@@ -147,7 +147,7 @@
 	emote_type = EMOTE_VISIBLE
 
 /mob/living/carbon/human/verb/emote_crossarms()
-	set name = "Cross Arms"
+	set name = "抱臂"
 	set category = "Emotes"
 
 	emote("crossarms", intentional = TRUE)
@@ -190,7 +190,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_cough()
-	set name = "Cough"
+	set name = "咳嗽"
 	set category = "Noises"
 
 	emote("cough", intentional = TRUE)
@@ -204,7 +204,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_clearthroat()
-	set name = "Clear Throat"
+	set name = "清嗓"
 	set category = "Noises"
 
 	emote("clearthroat", intentional = TRUE)
@@ -217,7 +217,7 @@
 	emote_type = EMOTE_VISIBLE
 
 /mob/living/carbon/human/verb/emote_dance()
-	set name = "Dance"
+	set name = "跳舞"
 	set category = "Emotes"
 
 	emote("dance", intentional = TRUE)
@@ -250,7 +250,7 @@
 	emote_type = EMOTE_VISIBLE
 
 /mob/living/carbon/human/verb/emote_drool()
-	set name = "Drool"
+	set name = "流口水"
 	set category = "Emotes"
 
 	emote("drool", intentional = TRUE)
@@ -262,7 +262,7 @@
 	emote_type = EMOTE_VISIBLE
 
 /mob/living/carbon/human/verb/emote_faint()
-	set name = "Faint"
+	set name = "晕倒"
 	set category = "Emotes"
 
 	emote("faint", intentional = TRUE)
@@ -303,7 +303,7 @@
 	message = "frowns."
 	emote_type = EMOTE_VISIBLE
 /mob/living/carbon/human/verb/emote_frown()
-	set name = "Frown"
+	set name = "皱眉"
 	set category = "Emotes"
 
 	emote("frown", intentional = TRUE)
@@ -317,7 +317,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_gag()
-	set name = "Gag"
+	set name = "干呕"
 	set category = "Noises"
 
 	emote("gag", intentional = TRUE)
@@ -332,7 +332,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_gasp()
-	set name = "Gasp"
+	set name = "倒吸气"
 	set category = "Noises"
 
 	emote("gasp", intentional = TRUE)
@@ -363,10 +363,10 @@
 				continue
 			if(HAS_TRAIT(H, TRAIT_XYLIX) && !H.has_status_effect(/datum/status_effect/buff/xylix_joy))
 				H.apply_status_effect(/datum/status_effect/buff/xylix_joy)
-				to_chat(H, span_info("The giggling brings a smile to my face, and fortune to my steps!"))
+				to_chat(H, span_info("那咯咯笑声让我面露笑意，连步伐都多了几分好运！"))
 
 /mob/living/carbon/human/verb/emote_giggle()
-	set name = "Giggle"
+	set name = "咯咯笑"
 	set category = "Noises"
 
 	emote("giggle", intentional = TRUE)
@@ -389,10 +389,10 @@
 				continue
 			if(HAS_TRAIT(H, TRAIT_XYLIX) && !H.has_status_effect(/datum/status_effect/buff/xylix_joy))
 				H.apply_status_effect(/datum/status_effect/buff/xylix_joy)
-				to_chat(H, span_info("The chuckling brings a smile to my face, and fortune to my steps!"))
+				to_chat(H, span_info("那轻笑让我面露笑意，连步伐都多了几分好运！"))
 
 /mob/living/carbon/human/verb/emote_chuckle()
-	set name = "Chuckle"
+	set name = "轻笑"
 	set category = "Noises"
 
 	emote("chuckle", intentional = TRUE)
@@ -405,7 +405,7 @@
 	emote_type = EMOTE_VISIBLE
 
 /mob/living/carbon/human/verb/emote_glare()
-	set name = "Glare"
+	set name = "怒视"
 	set category = "Emotes"
 
 	emote("glare", intentional = TRUE)
@@ -416,7 +416,7 @@
 	message = "grins."
 	emote_type = EMOTE_VISIBLE
 /mob/living/carbon/human/verb/emote_grin()
-	set name = "Grin"
+	set name = "咧嘴笑"
 	set category = "Emotes"
 
 	emote("grin", intentional = TRUE)
@@ -431,7 +431,7 @@
 	needs_emotion = TRUE
 
 /mob/living/carbon/human/verb/emote_groan()
-	set name = "Groan"
+	set name = "呻吟"
 	set category = "Noises"
 
 	emote("groan", intentional = TRUE)
@@ -442,7 +442,7 @@
 	message = "grimaces."
 	emote_type = EMOTE_VISIBLE
 /mob/living/carbon/human/verb/emote_grimace()
-	set name = "Grimace"
+	set name = "做鬼脸"
 	set category = "Emotes"
 
 	emote("grimace", intentional = TRUE)
@@ -469,7 +469,7 @@
 	use_params_for_runechat = TRUE
 
 /mob/living/carbon/human/verb/emote_kiss()
-	set name = "Kiss"
+	set name = "亲吻"
 	set category = "Emotes"
 
 	emote("kiss", intentional = TRUE, targetted = TRUE)
@@ -495,7 +495,7 @@
 				var/mob/living/carbon/human/E = target
 				if(iself(E) || ishalfelf(E) || isdarkelf(E))
 					if(!E.cmode)
-						to_chat(target, span_love("It tickles..."))
+						to_chat(target, span_love("好痒……"))
 			else if(H.zone_selected == BODY_ZONE_PRECISE_R_EYE || H.zone_selected == BODY_ZONE_PRECISE_L_EYE)
 				message_param = "kisses %t on the brow."
 			else if(H.zone_selected == BODY_ZONE_PRECISE_SKULL)
@@ -507,7 +507,7 @@
 				var/mob/living/carbon/human/L = target
 				if(isliving(L))
 					if(!L.cmode)
-						to_chat(target, span_love("It's somewhat stimulating..."))
+						to_chat(target, span_love("还有点让人心痒……"))
 			else
 				message_param = "kisses %t on \the [parse_zone(H.zone_selected)]."
 	playsound(target.loc, pick('sound/vo/kiss (1).ogg','sound/vo/kiss (2).ogg'), 100, FALSE, -1)
@@ -526,7 +526,7 @@
 	use_params_for_runechat = TRUE
 
 /mob/living/carbon/human/verb/emote_lick()
-	set name = "Lick"
+	set name = "舔舐"
 	set category = "Emotes"
 	emote("lick", intentional = TRUE, targetted = TRUE)
 
@@ -551,13 +551,13 @@
 				var/mob/living/carbon/human/O = target
 				if(iself(O) || ishalfelf(O) || isdarkelf(O))
 					if(!O.cmode)
-						to_chat(target, span_love("It tickles..."))
+						to_chat(target, span_love("好痒……"))
 			else if(J.zone_selected == BODY_ZONE_PRECISE_GROIN)
 				message_param = "licks %t between the legs."
 				var/mob/living/carbon/human/M = target
 				if(isliving(M))
 					if(!M.cmode)
-						to_chat(target, span_love("It's somewhat stimulating..."))
+						to_chat(target, span_love("还有点让人心痒……"))
 			else if(J.zone_selected == BODY_ZONE_HEAD)
 				message_param = "licks %t on the cheek."
 			else
@@ -572,7 +572,7 @@
 	emote_type = EMOTE_VISIBLE
 
 /mob/living/carbon/human/verb/emote_spit()
-	set name = "Spit"
+	set name = "吐口水"
 	set category = "Emotes"
 
 	emote("spit", intentional = TRUE, targetted = TRUE)
@@ -584,7 +584,7 @@
 		var/mob/living/carbon/human/H = user
 		if(H.mouth)
 			if(H.mouth.spitoutmouth)
-				H.visible_message(span_warning("[H] spits out [H.mouth]."))
+				H.visible_message(span_warning("[H]把[H.mouth]吐了出来。"))
 				H.dropItemToGround(H.mouth, silent = FALSE)
 			return
 	..()
@@ -608,7 +608,7 @@
 	restraint_check = TRUE
 
 /mob/living/carbon/human/verb/emote_hug()
-	set name = "Hug"
+	set name = "拥抱"
 	set category = "Emotes"
 
 	emote("hug", intentional = TRUE, targetted = TRUE)
@@ -629,7 +629,7 @@
 	message = null
 
 /mob/living/carbon/human/verb/emote_hold()
-	set name = "Hold Breath"
+	set name = "屏息"
 	set category = "Emotes"
 	emote("hold", intentional = TRUE)
 
@@ -649,8 +649,8 @@
 	if(is_holding)
 		REMOVE_TRAIT(H, TRAIT_HOLDBREATH, "[type]")
 		H.visible_message(
-			span_notice("[H] stops holding [H.p_their()] breath."),
-			span_notice("You stop holding your breath.")
+			span_notice("[H]停止了屏息。"),
+			span_notice("你停止了屏息。")
 		)
 		H.log_message("stopped holding their breath.", LOG_ATTACK)
 	else
@@ -659,8 +659,8 @@
 			return FALSE
 		ADD_TRAIT(H, TRAIT_HOLDBREATH, "[type]")
 		H.visible_message(
-			span_notice("[H] begins to hold [H.p_their()] breath."),
-			span_notice("You begin to hold your breath.")
+			span_notice("[H]开始屏住呼吸。"),
+			span_notice("你开始屏住呼吸。")
 		)
 		H.log_message("started holding their breath.", LOG_ATTACK)
 
@@ -675,7 +675,7 @@
 	restraint_check = TRUE
 
 /mob/living/carbon/human/verb/emote_pat()
-	set name = "Pat"
+	set name = "拍一拍"
 	set category = "Emotes"
 
 	emote("pat", intentional = TRUE, targetted = TRUE)
@@ -732,7 +732,7 @@
 	..()
 
 /mob/living/carbon/human/verb/emote_slap()
-	set name = "Slap"
+	set name = "掌掴"
 	set category = "Emotes"
 
 	emote("slap", intentional = TRUE, targetted = TRUE)
@@ -764,7 +764,7 @@
 		H.flash_fullscreen("redflash1")
 
 /mob/living/carbon/human/verb/emote_pinch()
-	set name = "Pinch"
+	set name = "掐"
 	set category = "Emotes"
 
 	emote("pinch", intentional = TRUE, targetted = TRUE)
@@ -800,10 +800,10 @@
 				continue
 			if(HAS_TRAIT(H, TRAIT_XYLIX) && !H.has_status_effect(/datum/status_effect/buff/xylix_joy))
 				H.apply_status_effect(/datum/status_effect/buff/xylix_joy)
-				to_chat(H, span_info("The laughter brings a smile to my face, and fortune to my steps!"))
+				to_chat(H, span_info("那笑声让我面露笑意，连步伐都多了几分好运！"))
 
 /mob/living/carbon/human/verb/emote_laugh()
-	set name = "Laugh"
+	set name = "大笑"
 	set category = "Noises"
 
 	emote("laugh", intentional = TRUE)
@@ -821,7 +821,7 @@
 	message_param = "nods at %t."
 	emote_type = EMOTE_VISIBLE
 /mob/living/carbon/human/verb/emote_nod()
-	set name = "Nod"
+	set name = "点头"
 	set category = "Emotes"
 
 	emote("nod", intentional = TRUE)
@@ -864,7 +864,7 @@
 	needs_emotion = TRUE
 
 /mob/living/carbon/human/verb/emote_scream()
-	set name = "Scream"
+	set name = "尖叫"
 	set category = "Noises"
 
 	emote("scream", intentional = TRUE)
@@ -875,7 +875,7 @@
 		var/mob/living/carbon/C = user
 		if(intentional)
 			if(!C.stamina_add(10))
-				to_chat(C, span_warning("I try to scream but my voice fails me."))
+				to_chat(C, span_warning("我试图尖叫，可嗓音背叛了我。"))
 				. = FALSE
 
 /datum/emote/living/scream/run_emote(mob/user, params, type_override, intentional, targetted)
@@ -1057,7 +1057,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_rage()
-	set name = "Rage"
+	set name = "狂怒"
 	set category = "Noises"
 
 	emote("rage", intentional = TRUE)
@@ -1075,7 +1075,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_attnwhistle()
-	set name = "Attnwhistle"
+	set name = "示警口哨"
 	set category = "Noises"
 
 	emote("attnwhistle", intentional = TRUE)
@@ -1101,7 +1101,7 @@
 	emote_type = EMOTE_VISIBLE
 
 /mob/living/carbon/human/verb/emote_shakehead()
-	set name = "Shake Head"
+	set name = "摇头"
 	set category = "Emotes"
 
 	emote("shakehead", intentional = TRUE)
@@ -1115,7 +1115,7 @@
 	show_runechat = TRUE
 
 /mob/living/carbon/human/verb/emote_shiver()
-	set name = "Shiver"
+	set name = "发抖"
 	set category = "Noises"
 
 	emote("shiver", intentional = TRUE)
@@ -1139,7 +1139,7 @@
 	mute_time = 5 SECONDS
 
 /mob/living/carbon/human/verb/emote_spin()
-	set name = "Spin"
+	set name = "转圈"
 	set category = "Emotes"
 	emote("spin", intentional = TRUE)
 
@@ -1165,7 +1165,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_sigh()
-	set name = "Sigh"
+	set name = "叹气"
 	set category = "Noises"
 
 	emote("sigh", intentional = TRUE)
@@ -1179,7 +1179,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_whistle()
-	set name = "Whistle"
+	set name = "吹口哨"
 	set category = "Noises"
 
 	emote("whistle", intentional = TRUE)
@@ -1193,7 +1193,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_hmm()
-	set name = "Hmm"
+	set name = "唔"
 	set category = "Noises"
 
 	emote("hmm", intentional = TRUE)
@@ -1207,7 +1207,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_huh()
-	set name = "Huh"
+	set name = "啊？"
 	set category = "Noises"
 
 	emote("huh", intentional = TRUE)
@@ -1221,7 +1221,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_hum()
-	set name = "Hum"
+	set name = "哼唱"
 	set category = "Noises"
 
 	emote("hum", intentional = TRUE)
@@ -1232,7 +1232,7 @@
 	message = "smiles."
 	emote_type = EMOTE_VISIBLE
 /mob/living/carbon/human/verb/emote_smile()
-	set name = "Smile"
+	set name = "微笑"
 	set category = "Emotes"
 
 	emote("smile", intentional = TRUE)
@@ -1262,7 +1262,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_shh()
-	set name = "Shh"
+	set name = "嘘声"
 	set category = "Noises"
 
 	emote("shh", intentional = TRUE)
@@ -1335,7 +1335,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_warcry()
-	set name = "Warcry"
+	set name = "战吼"
 	set category = "Noises"
 
 	emote("warcry", intentional = TRUE)
@@ -1355,7 +1355,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_whimper()
-	set name = "Whimper"
+	set name = "呜咽"
 	set category = "Noises"
 
 	emote("whimper", intentional = TRUE)
@@ -1374,7 +1374,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_yawn()
-	set name = "Yawn"
+	set name = "打哈欠"
 	set category = "Noises"
 
 	emote("yawn", intentional = TRUE)
@@ -1394,13 +1394,13 @@
 /datum/emote/living/custom/proc/check_invalid(mob/user, input)
 	. = TRUE
 	if(copytext(input,1,5) == "says")
-		to_chat(user, span_danger("Invalid emote."))
+		to_chat(user, span_danger("无效表情动作。"))
 	else if(copytext(input,1,9) == "exclaims")
-		to_chat(user, span_danger("Invalid emote."))
+		to_chat(user, span_danger("无效表情动作。"))
 	else if(copytext(input,1,6) == "yells")
-		to_chat(user, span_danger("Invalid emote."))
+		to_chat(user, span_danger("无效表情动作。"))
 	else if(copytext(input,1,5) == "asks")
-		to_chat(user, span_danger("Invalid emote."))
+		to_chat(user, span_danger("无效表情动作。"))
 	else
 		. = FALSE
 
@@ -1408,12 +1408,12 @@
 	if(!can_run_emote(user, TRUE, intentional))
 		return FALSE
 	if(is_banned_from(user.ckey, "Emote"))
-		to_chat(user, span_boldwarning("I cannot send custom emotes (banned)."))
+		to_chat(user, span_boldwarning("我无法发送自定义表情动作（已被禁用）。"))
 		return FALSE
 	else if(QDELETED(user))
 		return FALSE
 	else if(user.client && user.client.prefs.muted & MUTE_IC)
-		to_chat(user, span_boldwarning("I cannot send IC messages (muted)."))
+		to_chat(user, span_boldwarning("我无法发送 IC 消息（已被禁言）。"))
 		return FALSE
 	else if(!params)
 		var/custom_emote = copytext(sanitize(input("What does your character do?") as text|null), 1, MAX_MESSAGE_LEN)
@@ -1485,10 +1485,10 @@
 	. = ..()
 	var/obj/item/circlegame/N = new(user)
 	if(user.put_in_hands(N))
-		to_chat(user, span_notice("I make a circle with your hand."))
+		to_chat(user, span_notice("我用手比了一个圈。"))
 	else
 		qdel(N)
-		to_chat(user, span_warning("I don't have any free hands to make a circle with."))
+		to_chat(user, span_warning("我没有空闲的手来比圈。"))
 
 /datum/emote/living/slap
 	key = "slap"
@@ -1501,9 +1501,9 @@
 		return
 	var/obj/item/slapper/N = new(user)
 	if(user.put_in_hands(N))
-		to_chat(user, span_notice("I ready your slapping hand."))
-	else
-		to_chat(user, span_warning("You're incapable of slapping in your current state."))
+		to_chat(user, span_notice("我摆好了自己的扇打姿势。"))
+	else
+		to_chat(user, span_warning("以我现在的状态无法扇打。"))
 */
 
 /datum/emote/living/shake
@@ -1513,7 +1513,7 @@
 	emote_type = EMOTE_VISIBLE
 
 /mob/living/carbon/human/verb/emote_shake()
-	set name = "Shake Head"
+	set name = "摇头"
 	set category = "Emotes"
 
 	emote("shake", intentional = TRUE)
@@ -1525,7 +1525,7 @@
 	emote_type = EMOTE_VISIBLE
 
 /mob/living/carbon/human/verb/emote_squint()
-	set name = "Squint"
+	set name = "眯眼"
 	set category = "Emotes"
 
 	emote("squint", intentional = TRUE)
@@ -1657,11 +1657,11 @@
 		verbs += allowed_verbs
 
 /mob/living/carbon/human/proc/show_tongue_noise_warning()
-	to_chat(src, span_warning("Your tongue doesn't do that"))
+	to_chat(src, span_warning("你的舌头做不出那种动作。"))
 
 /mob/living/carbon/human/verb/emote_meow()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue))
-		set name = "Meow"
+		set name = "喵叫"
 		set category = "Noises"
 		emote("meow", intentional = TRUE, animal = TRUE)
 	else
@@ -1680,7 +1680,7 @@
 
 /mob/living/carbon/human/verb/emote_caw()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue) || istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/harpy))
-		set name = "Caw"
+		set name = "乌鸦叫"
 		set category = "Noises"
 		emote("caw", intentional = TRUE, animal = TRUE)
 	else
@@ -1699,7 +1699,7 @@
 
 /mob/living/carbon/human/verb/emote_peep()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue) || istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/harpy))
-		set name = "Peep"
+		set name = "啾啾叫"
 		set category = "Noises"
 		emote("peep", intentional = TRUE, animal = TRUE)
 	else
@@ -1718,7 +1718,7 @@
 
 /mob/living/carbon/human/verb/emote_hoot()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue) || istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/harpy))
-		set name = "Hoot"
+		set name = "咕咕叫"
 		set category = "Noises"
 		emote("hoot", intentional = TRUE, animal = TRUE)
 	else
@@ -1737,7 +1737,7 @@
 
 /mob/living/carbon/human/verb/emote_squeak()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue) || istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/harpy))
-		set name = "Squeak"
+		set name = "吱叫"
 		set category = "Noises"
 		emote("squeak", intentional = TRUE, animal = TRUE)
 	else
@@ -1756,7 +1756,7 @@
 
 /mob/living/carbon/human/verb/emote_chirp()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue) || istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/harpy))
-		set name = "Chirp"
+		set name = "啁啾"
 		set category = "Noises"
 		emote("chirp", intentional = TRUE, animal = TRUE)
 	else
@@ -1775,7 +1775,7 @@
 
 /mob/living/carbon/human/verb/emote_warble()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue) || istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/harpy))
-		set name = "Warble"
+		set name = "啭鸣"
 		set category = "Noises"
 		emote("warble", intentional = TRUE, animal = TRUE)
 	else
@@ -1794,7 +1794,7 @@
 
 /mob/living/carbon/human/verb/emote_dove()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue) || istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/harpy))
-		set name = "Dove"
+		set name = "咕咕"
 		set category = "Noises"
 		emote("dove", intentional = TRUE, animal = TRUE)
 	else
@@ -1813,7 +1813,7 @@
 
 /mob/living/carbon/human/verb/emote_loudcaw()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue) || istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/harpy))
-		set name = "Loudcaw"
+		set name = "大声乌鸦叫"
 		set category = "Noises"
 		emote("loudcaw", intentional = TRUE, animal = TRUE)
 	else
@@ -1833,7 +1833,7 @@
 
 /mob/living/carbon/human/verb/emote_raptor()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue) || istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/harpy))
-		set name = "Raptor"
+		set name = "猛禽吼叫"
 		set category = "Noises"
 		emote("raptor", intentional = TRUE, animal = TRUE)
 	else
@@ -1852,11 +1852,11 @@
 
 /mob/living/carbon/human/verb/emote_hiss()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue))
-		set name = "Hiss"
+		set name = "嘶声"
 		set category = "Noises"
 		emote("hiss", intentional = TRUE, animal = TRUE)
 	else if (istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/lizard))
-		set name = "Hiss"
+		set name = "嘶声"
 		set category = "Noises"
 		emote("hiss", intentional = TRUE, animal = TRUE)
 	else
@@ -1875,11 +1875,11 @@
 
 /mob/living/carbon/human/verb/emote_phiss()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue))
-		set name = "PHiss"
+		set name = "喷气嘶声"
 		set category = "Noises"
 		emote("phiss", intentional = TRUE, animal = TRUE)
 	else if (istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/lizard))
-		set name = "PHiss"
+		set name = "喷气嘶声"
 		set category = "Noises"
 		emote("hiss", intentional = TRUE, animal = TRUE)
 	else
@@ -1897,7 +1897,7 @@
 
 /mob/living/carbon/human/verb/emote_roar()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue))
-		set name = "Roar"
+		set name = "咆哮"
 		set category = "Noises"
 		emote("roar", intentional = TRUE)
 	else
@@ -1916,7 +1916,7 @@
 
 /mob/living/carbon/human/verb/emote_howl()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue))
-		set name = "Howl"
+		set name = "嚎叫"
 		set category = "Noises"
 		emote("howl", intentional = TRUE, animal = TRUE)
 	else
@@ -1935,7 +1935,7 @@
 
 /mob/living/carbon/human/verb/emote_cackle()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue))
-		set name = "Cackle"
+		set name = "怪笑"
 		set category = "Noises"
 		emote("cackle", intentional = TRUE, animal = TRUE)
 	else
@@ -1954,7 +1954,7 @@
 
 /mob/living/carbon/human/verb/emote_whine()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue))
-		set name = "Whine"
+		set name = "呜呜叫"
 		set category = "Noises"
 		emote("whine", intentional = TRUE, animal = TRUE)
 	else
@@ -1973,7 +1973,7 @@
 
 /mob/living/carbon/human/verb/emote_trill()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue) || istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/harpy))
-		set name = "Trill"
+		set name = "颤鸣"
 		set category = "Noises"
 		emote("trill", intentional = TRUE, animal = TRUE)
 	else
@@ -1988,7 +1988,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_snap()
-	set name = "Snap"
+	set name = "啪嗒"
 	set category = "Noises"
 
 	emote("snap", intentional = TRUE)
@@ -2001,7 +2001,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_blink()
-	set name = "Blink"
+	set name = "眨眼"
 	set category = "Noises"
 
 	emote("blink", intentional = TRUE)
@@ -2014,7 +2014,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_stomp()
-	set name = "Stomp"
+	set name = "跺脚"
 	set category = "Noises"
 
 	emote("stomp", intentional = TRUE)
@@ -2027,7 +2027,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_snap2()
-	set name = "Snap2"
+	set name = "啪嗒2"
 	set category = "Noises"
 
 	emote("snap2", intentional = TRUE)
@@ -2040,7 +2040,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_snap3()
-	set name = "Snap3"
+	set name = "啪嗒3"
 	set category = "Noises"
 
 	emote("snap3", intentional = TRUE)
@@ -2057,7 +2057,7 @@
 
 /mob/living/carbon/human/verb/emote_purr()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue) || istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/harpy))
-		set name = "Purr"
+		set name = "呼噜"
 		set category = "Noises"
 		emote("purr", intentional = TRUE, animal = TRUE)
 	else
@@ -2076,7 +2076,7 @@
 
 /mob/living/carbon/human/verb/emote_moo()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue))
-		set name = "Moo"
+		set name = "哞叫"
 		set category = "Noises"
 		emote("moo", intentional = TRUE, animal = TRUE)
 	else
@@ -2095,7 +2095,7 @@
 
 /mob/living/carbon/human/verb/emote_bark()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue))
-		set name = "Bark"
+		set name = "吠叫"
 		set category = "Noises"
 		emote("bark", intentional = TRUE, animal = TRUE)
 	else
@@ -2114,7 +2114,7 @@
 
 /mob/living/carbon/human/verb/emote_growl()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue) || istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/harpy))
-		set name = "Growl"
+		set name = "低吼"
 		set category = "Noises"
 		emote("growl", intentional = TRUE, animal = TRUE)
 	else
@@ -2133,7 +2133,7 @@
 
 /mob/living/carbon/human/verb/emote_bleat()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue))
-		set name = "Bleat"
+		set name = "咩叫"
 		set category = "Noises"
 		emote("bleat", intentional = TRUE, animal = TRUE)
 	else
@@ -2152,11 +2152,11 @@
 
 /mob/living/carbon/human/verb/emote_chitter()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/moth))
-		set name = "Chitter"
+		set name = "吱喳"
 		set category = "Noises"
 		emote("chitter", intentional = TRUE, animal = TRUE)
 	else if (istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue))
-		set name = "Chitter"
+		set name = "吱喳"
 		set category = "Noises"
 		emote("chitter", intentional = TRUE, animal = TRUE)
 	else
@@ -2172,11 +2172,11 @@
 
 /mob/living/carbon/human/verb/emote_flutter()
 	if(istype(usr.getorganslot(ORGAN_SLOT_WINGS), /obj/item/organ/wings/moth))
-		set name = "Flutter"
+		set name = "扇翅"
 		set category = "Noises"
 		emote("flutter", intentional = TRUE)
 	else
-		to_chat(usr, span_warning("Your back doesn't do that"))
+		to_chat(usr, span_warning("你的背部做不出那种动作。"))
 		return
 
 /datum/emote/living/fsalute
@@ -2192,7 +2192,7 @@
 		user.play_overhead_indicator('icons/mob/overhead_effects.dmi', "stress", 15, MUTATIONS_LAYER, private = user.patron.type, soundin = 'sound/magic/holyshield.ogg', y_offset = 32)
 
 /mob/living/carbon/human/verb/emote_fsalute()
-	set name = "Faith Salute"
+	set name = "信仰敬礼"
 	set category = "Emotes"
 
 	emote("fsalute", intentional =  TRUE)
@@ -2209,7 +2209,7 @@
 		. = ..()
 
 /mob/living/carbon/human/proc/emote_ffsalute()
-	set name = "Fake Faith Salute"
+	set name = "假信仰敬礼"
 	set category = "Emotes"
 
 	emote("ffsalute", intentional =  TRUE)
@@ -2224,7 +2224,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/yip()
-	set name = "Yip"
+	set name = "汪呜"
 	set category = "Noises"
 
 	emote("yip", intentional = TRUE)
@@ -2238,13 +2238,13 @@
 	is_animal = TRUE
 	show_runechat = FALSE
 /mob/living/carbon/human/verb/yap()
-	set name = "Yap"
+	set name = "汪汪"
 	set category = "Noises"
 	emote("yap", intentional = TRUE)
 
 /* Vomit emote */
 /mob/living/carbon/human/verb/emote_vomit()
-	set name = "Vomit"
+	set name = "呕吐"
 	set category = "Emotes"
 
 	emote("vomit", intentional = TRUE)
@@ -2267,7 +2267,7 @@
 	show_runechat = TRUE
 
 /mob/living/carbon/human/verb/gulp()
-	set name = "Gulp"
+	set name = "咕噜吞咽"
 	set category = "Noises"
 
 	emote("gulp", intentional = TRUE)
@@ -2280,7 +2280,7 @@
 	show_runechat = TRUE
 
 /mob/living/carbon/human/verb/crack()
-	set name = "Knuckles"
+	set name = "掰指节"
 	set category = "Noises"
 
 	emote("crack", intentional = TRUE)
@@ -2293,7 +2293,7 @@
 	show_runechat = TRUE
 
 /mob/living/carbon/human/verb/facepalm()
-	set name = "Facepalm"
+	set name = "捂脸"
 	set category = "Noises"
 
 	emote("facepalms", intentional = TRUE)
@@ -2306,7 +2306,7 @@
 	show_runechat = TRUE
 
 /mob/living/carbon/human/verb/eye_roll()
-	set name = "Eye Roll"
+	set name = "翻白眼"
 	set category = "Emotes"
 
 	emote("eye_roll", intentional = TRUE)
@@ -2319,7 +2319,7 @@
 	show_runechat = TRUE
 
 /mob/living/carbon/human/verb/salute()
-	set name = "Salute"
+	set name = "敬礼"
 	set category = "Noises"
 
 	emote("salute", intentional = TRUE)
@@ -2332,7 +2332,7 @@
 	show_runechat = TRUE
 
 /mob/living/carbon/human/verb/sniff()
-	set name = "Sniff"
+	set name = "嗅闻"
 	set category = "Noises"
 
 	emote("sniff", intentional = TRUE)
@@ -2378,7 +2378,7 @@
 
 		chance *= chance_per_point
 
-		var/msg = success ? span_green("SUCCEEDS and [pick(success_message_list)]") : span_danger("FAILS and [pick(failure_message_list)] [chance]%")
+		var/msg = success ? span_green("成功，并且[pick(success_message_list)]") : span_danger("失败，并且[pick(failure_message_list)] [chance]%")
 
 		msg = replace_pronoun(user, msg)
 
@@ -2446,7 +2446,7 @@
 	)
 
 /mob/living/carbon/human/verb/emote_strength_roll()
-	set name = "Roll Strength"
+	set name = "力量检定"
 	set category = "Emotes"
 
 	emote("strength", intentional = TRUE)
@@ -2473,7 +2473,7 @@
 	)
 
 /mob/living/carbon/human/verb/emote_perception_roll()
-	set name = "Roll Perception"
+	set name = "感知检定"
 	set category = "Emotes"
 
 	emote("perception", intentional = TRUE)
@@ -2501,7 +2501,7 @@
 	)
 
 /mob/living/carbon/human/verb/emote_intelligence_roll()
-	set name = "Roll Intelligence"
+	set name = "智力检定"
 	set category = "Emotes"
 
 	emote("intelligence", intentional = TRUE)
@@ -2528,7 +2528,7 @@
 	)
 
 /mob/living/carbon/human/verb/emote_constitution_roll()
-	set name = "Roll Constitution"
+	set name = "体质检定"
 	set category = "Emotes"
 
 	emote("constitution", intentional = TRUE)
@@ -2555,7 +2555,7 @@
 	)
 
 /mob/living/carbon/human/verb/emote_willpower_roll()
-	set name = "Roll Willpower"
+	set name = "意志检定"
 	set category = "Emotes"
 
 	emote("willpower", intentional = TRUE)
@@ -2582,7 +2582,7 @@
 	)
 
 /mob/living/carbon/human/verb/emote_speed_roll()
-	set name = "Roll Speed"
+	set name = "速度检定"
 	set category = "Emotes"
 
 	emote("speed", intentional = TRUE)
@@ -2609,7 +2609,7 @@
 	)
 
 /mob/living/carbon/human/verb/emote_fortune_roll()
-	set name = "Roll Fortune"
+	set name = "幸运检定"
 	set category = "Emotes"
 
 	emote("fortune", intentional = TRUE)
@@ -2623,7 +2623,7 @@
 	show_runechat = FALSE
 
 /mob/living/carbon/human/verb/emote_praysuicide()
-	set name = "Suicidal pray"
+	set name = "自尽祈祷"
 	set category = "Emotes"
 	emote("praysuicide", intentional = TRUE)
 
@@ -2632,7 +2632,7 @@
 		return FALSE
 
 	var/mob/living/L = user
-	to_chat(L, span_danger("I pray to my patron for my death... and I am heard."))
+	to_chat(L, span_danger("我向我的主神祈求死亡……而祂听见了。"))
 	var/lastmsg = params
 	if(!lastmsg)
 		lastmsg = input("Whisper your final words:", "Last Words") as text|null
@@ -2657,7 +2657,7 @@
 
 /mob/living/carbon/human/verb/emote_arf()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue) || istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/harpy))
-		set name = "Arf"
+		set name = "汪"
 		set category = "Noises"
 		emote("arf", intentional = TRUE, animal = TRUE)
 	else
@@ -2676,7 +2676,7 @@
 
 /mob/living/carbon/human/verb/emote_awuff()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue) || istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/harpy))
-		set name = "Awuff"
+		set name = "嗷呜"
 		set category = "Noises"
 		emote("awuff", intentional = TRUE, animal = TRUE)
 	else
@@ -2695,7 +2695,7 @@
 
 /mob/living/carbon/human/verb/emote_dcomplain()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue) || istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/harpy))
-		set name = "Dog Complain"
+		set name = "狗狗抱怨"
 		set category = "Noises"
 		emote("dcomplain", intentional = TRUE, animal = TRUE)
 	else
@@ -2714,7 +2714,7 @@
 
 /mob/living/carbon/human/verb/emote_dgrowl()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue) || istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/harpy))
-		set name = "Dog Growl"
+		set name = "狗狗低吼"
 		set category = "Noises"
 		emote("dgrowl", intentional = TRUE, animal = TRUE)
 	else
@@ -2733,7 +2733,7 @@
 
 /mob/living/carbon/human/verb/emote_dwhine()
 	if(istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/wild_tongue) || istype(usr.getorganslot(ORGAN_SLOT_TONGUE), /obj/item/organ/tongue/harpy))
-		set name = "Dog Whine"
+		set name = "狗狗呜咽"
 		set category = "Noises"
 		emote("dwhine", intentional = TRUE, animal = TRUE)
 	else
-- 
Translation Patch Toolkit
