From 619caec76be41bd46777cb9f57d1709d646e5578 Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Sat, 30 May 2026 13:00:00 +0800
Subject: [PATCH] 翻译 code/modules/mob/living/life.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
diff --git a/code/modules/mob/living/life.dm b/code/modules/mob/living/life.dm
--- a/code/modules/mob/living/life.dm
+++ b/code/modules/mob/living/life.dm
@@ -129,8 +129,8 @@
 					mob_timers["painstun"] = world.time
 					Immobilize(10)
 					emote("painscream")
-					visible_message(span_warning("[src] freezes in pain!"),
-								span_warning("I'm frozen in pain!"))
+					visible_message(span_warning("[src]因剧痛而僵住了！"),
+								span_warning("我被疼得僵住了！"))
 					sleep(10)
 					Stun(110)
 					Knockdown(110)
@@ -160,11 +160,11 @@
 			if(embedded.is_silver && HAS_TRAIT(src, TRAIT_SILVER_WEAK) && !has_status_effect(STATUS_EFFECT_ANTIMAGIC))
 				var/datum/component/silverbless/psyblessed = embedded.GetComponent(/datum/component/silverbless)
 				adjust_fire_stacks(1, psyblessed?.is_blessed ? /datum/status_effect/fire_handler/fire_stacks/sunder/blessed : /datum/status_effect/fire_handler/fire_stacks/sunder)
-			to_chat(src, span_danger("[embedded] in me hurts!"))
+			to_chat(src, span_danger("插在我体内的[embedded]好痛！"))
 
 		if(prob(embedded.embedding.embedded_fall_chance))
 			simple_remove_embedded_object(embedded)
-			to_chat(src,span_danger("[embedded] falls out of me!"))
+			to_chat(src,span_danger("[embedded]从我体内掉了出来！"))
 
 //this updates all special effects: knockdown, druggy, stuttering, etc..
 /mob/living/proc/handle_status_effects(additional)
-- 
Translation Patch Toolkit
