diff --git a/code/modules/mob/living/say.dm b/code/modules/mob/living/say.dm
--- a/code/modules/mob/living/say.dm
+++ b/code/modules/mob/living/say.dm
@@ -100,7 +100,7 @@
 
 	if(ic_blocked)
 		//The filter warning message shows the sanitized message though.
-		to_chat(src, span_warning("That message contained a word prohibited in IC chat! Consider reviewing the server rules.\n<span replaceRegex='show_filtered_ic_chat'>\"[message]\"</span>"))
+		to_chat(src, span_warning("?????? IC ????????????????????????\n<span replaceRegex='show_filtered_ic_chat'>\"[message]\"</span>"))
 		SSblackbox.record_feedback("tally", "ic_blocked_words", 1, LOWER_TEXT(config.ic_filter_regex.match))
 		return
 
@@ -186,8 +186,8 @@
 	// Allow sign languages and other tongueless speech to bypass the vocal speech check
 	var/using_tongueless_speech = language && (initial(language.flags) & TONGUELESS_SPEECH)
 	if(!can_speak_vocal(message) && !using_tongueless_speech)
-		emote("custom", message = "makes a muffled noise")
-		to_chat(src, span_warning("I can't talk."))
+		emote("custom", message = "??????????")
+		to_chat(src, span_warning("??????"))
 		return
 
 	var/message_range = 7
@@ -387,10 +387,10 @@
 	var/deaf_type
 	if(speaker != src)
 		if(!radio_freq) //These checks have to be seperate, else people talking on the radio will make "You can't hear yourself!" appear when hearing people over the radio while deaf.
-			deaf_message = "<span class='name'>[speaker]</span> [speaker.verb_say] something but you cannot hear [speaker.p_them()]."
+			deaf_message = "<span class='name'>[speaker]</span>?????????????[speaker.p_them()]?"
 			deaf_type = 1
 	else
-		deaf_message = span_notice("I can't hear myself!")
+		deaf_message = span_notice("????????????")
 		deaf_type = 2 // Since you should be able to hear myself without looking
 
 	// Create map text prior to modifying message for goonchat
@@ -444,9 +444,9 @@
 		message_range += (5 + buff.potency) // maximum 12 tiles extra, which is a lot!
 		for(var/obj/structure/roguemachine/scomm/S in SSroguemachine.scomm_machines)
 			if (prob(buff.potency * 3) && S.speaking) // 3% chance per holy level, per SCOM for it to shriek your message in town wherever you are
-				S.verb_say = "shrieks in terror"
-				S.verb_exclaim = "shrieks in terror"
-				S.verb_yell = "shrieks in terror"
+				S.verb_say = "?????"
+				S.verb_exclaim = "?????"
+				S.verb_yell = "?????"
 				S.say(message, spans = list("info", "reallybig"))
 				S.verb_say = initial(S.verb_say)
 				S.verb_exclaim = initial(S.verb_exclaim)
@@ -606,7 +606,7 @@
 /mob/living/proc/can_speak_basic(message, ignore_spam = FALSE, forced = FALSE) //Check BEFORE handling of xeno and ling channels
 	if(client)
 		if(client.prefs.muted & MUTE_IC)
-			to_chat(src, span_danger("I cannot speak in IC (muted)."))
+			to_chat(src, span_danger("???? IC ???????????"))
 			return FALSE
 		if(!(ignore_spam || forced) && client.handle_spam_prevention(message,MUTE_IC))
 			return FALSE
@@ -693,9 +693,9 @@
 	else if(message_mode == MODE_WHISPER_CRIT)
 		. = "[verb_whisper] in [p_their()] last breath"
 	else if(stuttering)
-		. = "stammers"
+		. = "??????"
 	else if(derpspeech)
-		. = "gibbers"
+		. = "???????"
 	else if(message_mode == MODE_SING)
 		. = verb_sing
 	else
