From 54cb0c2d19da4c4705f055f95e29ee7db6b1f955 Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Sat, 30 May 2026 13:00:00 +0800
Subject: [PATCH] 翻译 code/modules/mob/living/taste.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
diff --git a/code/modules/mob/living/taste.dm b/code/modules/mob/living/taste.dm
--- a/code/modules/mob/living/taste.dm
+++ b/code/modules/mob/living/taste.dm
@@ -22,11 +22,11 @@
 		// We dont want to spam the same message over and over again at the
 		// person. Give it a bit of a buffer.
 		if(hallucination > 50 && prob(25))
-			text_output = pick("spiders","dreams","nightmares","the future","the past","victory",\
-			"defeat","pain","bliss","revenge","poison","time","space","death","life","truth","lies","justice","memory",\
-			"regrets","your soul","suffering","music","noise","blood","hunger","the american way")
+			text_output = pick("蜘蛛","梦境","噩梦","未来","过去","胜利",\
+			"失败","痛苦","极乐","复仇","毒药","时间","空间","死亡","生命","真相","谎言","正义","记忆",\
+			"遗憾","你的灵魂","苦难","音乐","噪音","鲜血","饥饿","美式作风")
 		if(text_output != last_taste_text || last_taste_time + 100 < world.time)
-			to_chat(src, span_info("I can taste [text_output]."))
+			to_chat(src, span_info("我能尝到[text_output]的滋味。"))
 			// "something indescribable" -> too many tastes, not enough flavor.
 
 			last_taste_time = world.time
-- 
Translation Patch Toolkit
