diff --git a/code/modules/roguetown/roguecrafting/crafting/clothes.dm b/code/modules/roguetown/roguecrafting/crafting/clothes.dm
--- a/code/modules/roguetown/roguecrafting/crafting/clothes.dm
+++ b/code/modules/roguetown/roguecrafting/crafting/clothes.dm
@@ -1,5 +1,5 @@
 /datum/crafting_recipe/roguetown/survival/skullmask
-	name = "skull mask"
+	name = "头骨面具"
 	category = "Clothes"
 	result = /obj/item/clothing/mask/rogue/skullmask
 	reqs = list(
@@ -13,7 +13,7 @@
 
 
 /datum/crafting_recipe/roguetown/survival/antlerhood
-	name = "antlerhood"
+	name = "鹿角兜帽"
 	category = "Clothes"
 	result = /obj/item/clothing/head/roguetown/antlerhood
 	reqs = list(
@@ -28,7 +28,7 @@
 	craftdiff = 0
 
 /datum/crafting_recipe/roguetown/survival/tribalrags
-	name = "tribal rags"
+	name = "部族破衣"
 	category = "Clothes"
 	result = /obj/item/clothing/suit/roguetown/shirt/tribalrag
 	reqs = list(
@@ -43,7 +43,7 @@
 	craftdiff = 0
 
 /datum/crafting_recipe/roguetown/leather/neck/leather_collar
-	name = "leather collar"
+	name = "皮项圈"
 	category = "Clothes"
 	result = /obj/item/clothing/neck/roguetown/collar/leather
 	reqs = list(/obj/item/natural/hide/cured = 1)
@@ -54,7 +54,7 @@
 	always_availible = TRUE
 
 /datum/crafting_recipe/roguetown/leather/neck/catbell_collar
-	name = "catbell collar"
+	name = "猫铃项圈"
 	result = /obj/item/clothing/neck/roguetown/collar/catbell
 	reqs = list(/obj/item/natural/hide/cured = 1, /obj/item/catbell = 1)
 	tools = list(/obj/item/needle)
@@ -64,7 +64,7 @@
 	always_availible = TRUE
 
 /datum/crafting_recipe/roguetown/leather/neck/cowbell_collar
-	name = "cowbell collar"
+	name = "牛铃项圈"
 	result = /obj/item/clothing/neck/roguetown/collar/cowbell
 	reqs = list(/obj/item/natural/hide/cured = 1, /obj/item/catbell/cow = 1)
 	tools = list(/obj/item/needle)
@@ -74,7 +74,7 @@
 	always_availible = TRUE
 
 /datum/crafting_recipe/roguetown/leather/neck/leather_leash
-	name = "leather leash"
+	name = "皮制牵引绳"
 	result = /obj/item/leash/leather
 	reqs = list(/obj/item/natural/hide/cured = 1)
 	tools = list(/obj/item/needle)
@@ -84,7 +84,7 @@
 	always_availible = TRUE
 
 /datum/crafting_recipe/roguetown/survival/goodluckcharm
-	name = "cabbit's foot luck charm"
+	name = "卡比特脚幸运符"
 	category = "Clothes"
 	result = /obj/item/clothing/neck/roguetown/luckcharm // +1 fortune when worn
 	reqs = list(
@@ -96,7 +96,7 @@
 // BOUQUETS & CROWNS
 
 /datum/crafting_recipe/roguetown/survival/bouquet_rosa
-	name = "rosa bouquet"
+	name = "Rosa 花束"
 	category = "Clothes"
 	result = /obj/item/bouquet/rosa
 	reqs = list(
@@ -109,7 +109,7 @@
 	verbage = "arranges"
 
 /datum/crafting_recipe/roguetown/survival/bouquet_salvia
-	name = "salvia bouquet"
+	name = "Salvia 花束"
 	category = "Clothes"
 	result = /obj/item/bouquet/salvia
 	reqs = list(
@@ -122,7 +122,7 @@
 	verbage = "arranges"
 
 /datum/crafting_recipe/roguetown/survival/bouquet_matricaria
-	name = "matricaria bouquet"
+	name = "Matricaria 花束"
 	category = "Clothes"
 	result = /obj/item/bouquet/matricaria
 	reqs = list(
@@ -135,7 +135,7 @@
 	verbage = "arranges"
 
 /datum/crafting_recipe/roguetown/survival/bouquet_calendula
-	name = "calendula bouquet"
+	name = "Calendula 花束"
 	category = "Clothes"
 	result = /obj/item/bouquet/calendula
 	reqs = list(
@@ -148,7 +148,7 @@
 	verbage = "arranges"
 
 /datum/crafting_recipe/roguetown/survival/flowercrown_rosa
-	name = "rosa crown"
+	name = "Rosa 花冠"
 	category = "Clothes"
 	result = /obj/item/flowercrown/rosa
 	reqs = list(
@@ -160,7 +160,7 @@
 	verbage = "ties"
 
 /datum/crafting_recipe/roguetown/survival/flowercrown_salvia
-	name = "salvia crown"
+	name = "Salvia 花冠"
 	category = "Clothes"
 	result = /obj/item/flowercrown/salvia
 	reqs = list(
@@ -173,7 +173,7 @@
 
 // Amulet
 /datum/crafting_recipe/roguetown/survival/pearlcross
-	name = "amulet (pearls)"
+	name = "护符（珍珠）"
 	category = "Clothes"
 	result = /obj/item/clothing/neck/roguetown/psicross/pearl
 	reqs = list(
@@ -183,7 +183,7 @@
 	craftdiff = 2
 
 /datum/crafting_recipe/roguetown/survival/bpearlcross
-	name = "amulet (blue pearls) "
+	name = "护符（蓝珍珠） "
 	category = "Clothes"
 	result = /obj/item/clothing/neck/roguetown/psicross/bpearl
 	reqs = list(
@@ -193,7 +193,7 @@
 	craftdiff = 2
 
 /datum/crafting_recipe/roguetown/survival/shellnecklace
-	name = "shell necklace"
+	name = "贝壳项链"
 	category = "Clothes"
 	result = /obj/item/clothing/neck/roguetown/psicross/shell
 	reqs = list(
@@ -202,7 +202,7 @@
 		)
 
 /datum/crafting_recipe/roguetown/survival/shellbracelet
-	name = "shell bracelet"
+	name = "贝壳手环"
 	category = "Clothes"
 	result = /obj/item/clothing/neck/roguetown/psicross/shell/bracelet
 	reqs = list(
@@ -211,7 +211,7 @@
 		)
 
 /datum/crafting_recipe/roguetown/survival/abyssoramulet
-	name = "amulet of abyssor"
+	name = "Abyssor 护符"
 	category = "Clothes"
 	result = /obj/item/clothing/neck/roguetown/psicross/abyssor
 	reqs = list(
@@ -220,7 +220,7 @@
 		)
 
 /datum/crafting_recipe/roguetown/survival/woodcross
-	name = "wooden psycross"
+	name = "木制 Psydon 十字架"
 	category = "Clothes"
 	result = /obj/item/clothing/neck/roguetown/psicross/wood
 	reqs = list(
@@ -229,7 +229,7 @@
 		)
 
 /datum/crafting_recipe/roguetown/survival/wickercloak
-	name = "wicker cloak"
+	name = "柳条披风"
 	category = "Clothes"
 	result = /obj/item/clothing/cloak/wickercloak
 	reqs = list(
@@ -240,7 +240,7 @@
 	craftdiff = 0
 
 /datum/crafting_recipe/roguetown/survival/mentorhat
-	name = "worn bamboo hat"
+	name = "旧竹帽"
 	category = "Clothes"
 	result = /obj/item/clothing/head/roguetown/mentorhat
 	reqs = list(
