diff --git a/code/modules/roguetown/roguecrafting/alchemy/alch_grind_recipes.dm b/code/modules/roguetown/roguecrafting/alchemy/alch_grind_recipes.dm
--- a/code/modules/roguetown/roguecrafting/alchemy/alch_grind_recipes.dm
+++ b/code/modules/roguetown/roguecrafting/alchemy/alch_grind_recipes.dm
@@ -32,19 +32,19 @@
 
 //Objects -> dusts
 /datum/alch_grind_recipe/crow
-	name = "Crow"
+	name = "乌鸦"
 	valid_input = /obj/item/reagent_containers/food/snacks/crow
 	valid_outputs = list(/obj/item/alch/airdust = 1)
 	bonus_chance_outputs = list(/obj/item/alch/airdust = 33)
 
 /datum/alch_grind_recipe/bone
-	name = "Bone"
+	name = "骨头"
 	valid_input = /obj/item/alch/bone
 	valid_outputs = list( /obj/item/alch/bonemeal = 2)
 	bonus_chance_outputs = list(/obj/item/alch/bonemeal = 50)
 
 /datum/alch_grind_recipe/natural_bone
-	name = "Natural Bone"
+	name = "天然骨骼"
 	valid_input = /obj/item/natural/bone
 	valid_outputs = list(/obj/item/alch/bonemeal = 1)
 	bonus_chance_outputs = list(/obj/item/alch/bonemeal = 2)
@@ -64,7 +64,7 @@
 	bonus_chance_outputs = list(/obj/item/alch/earthdust = 66)
 
 /datum/alch_grind_recipe/fish
-	name = "Fish"
+	name = "鱼"
 	picky = FALSE
 	valid_input = /obj/item/reagent_containers/food/snacks/fish
 	valid_outputs = list(/obj/item/alch/waterdust = 1)
@@ -77,7 +77,7 @@
 	bonus_chance_outputs = list(/obj/item/alch/earthdust = 33)
 
 /datum/alch_grind_recipe/swampweed_dried
-	name = "Dried Swampweed"
+	name = "干燥 Swampweed"
 	valid_input = /obj/item/reagent_containers/food/snacks/grown/rogue/swampweeddry
 	valid_outputs = list(/obj/item/alch/swampdust = 1)
 	bonus_chance_outputs = list(/obj/item/alch/earthdust = 50,/obj/item/alch/swampdust = 50)
@@ -89,7 +89,7 @@
 	bonus_chance_outputs = list(/obj/item/alch/airdust = 33)
 
 /datum/alch_grind_recipe/dry_westleach
-	name = "Dried Westleach"
+	name = "干燥 Westleach"
 	valid_input = /obj/item/reagent_containers/food/snacks/grown/rogue/pipeweeddry
 	valid_outputs = list(/obj/item/alch/tobaccodust = 1)
 	bonus_chance_outputs = list(/obj/item/alch/airdust = 50,/obj/item/alch/tobaccodust = 50)
@@ -101,111 +101,111 @@
 	bonus_chance_outputs = list(/obj/item/alch/solardust = 50)
 
 /datum/alch_grind_recipe/poppy
-	name = "Poppy"
+	name = "罂粟"
 	valid_input = /obj/item/reagent_containers/food/snacks/grown/rogue/poppy
 	valid_outputs = list(/obj/item/reagent_containers/powder/ozium = 1)
 	bonus_chance_outputs = list(/obj/item/alch/airdust =33,/obj/item/alch/earthdust = 33)
 
 /datum/alch_grind_recipe/seeds
-	name = "Seeds"
+	name = "种子"
 	picky = FALSE
 	valid_input = /obj/item/seeds
 	valid_outputs = list(/obj/item/alch/seeddust = 1)
 	bonus_chance_outputs = list(/obj/item/alch/airdust =25,/obj/item/alch/earthdust = 25)
 
 /datum/alch_grind_recipe/stone_dust
-	name = "Stone"
+	name = "石料"
 	valid_input = /obj/item/natural/stone
 	valid_outputs = list(/obj/item/alch/stonedust = 1)
 	bonus_chance_outputs = list(/obj/item/alch/stonedust = 50)
 
 /datum/alch_grind_recipe/seedsherb
-	name = "Herb Seeds"
+	name = "草药种子"
 	picky = FALSE
 	valid_input = /obj/item/herbseed
 	valid_outputs = list(/obj/item/alch/seeddust = 1)
 	bonus_chance_outputs = list(/obj/item/alch/airdust =25,/obj/item/alch/earthdust = 25)
 
 /datum/alch_grind_recipe/ozium
-	name = "Ozium"
+	name = "奥兹姆"
 	valid_input = /obj/item/reagent_containers/powder/ozium
 	valid_outputs = list(/obj/item/alch/ozium = 1)
 	bonus_chance_outputs = list(/obj/item/alch/airdust =25,/obj/item/alch/ozium = 25)
 
 /datum/alch_grind_recipe/sunflower
-	name = "Sunflower"
+	name = "向日葵"
 	valid_input = /obj/item/reagent_containers/food/snacks/grown/sunflower
 	valid_outputs = list(/obj/item/alch/solardust = 1)
 	bonus_chance_outputs = list(/obj/item/alch/firedust = 25)
 
 //Ores -> dust
 /datum/alch_grind_recipe/gold_ore
-	name = "Gold Ore"
+	name = "金矿石"
 	valid_input = /obj/item/rogueore/gold
 	valid_outputs = list(/obj/item/alch/golddust = 1)
 	bonus_chance_outputs = list(/obj/item/alch/golddust = 33)
 
 /datum/alch_grind_recipe/silver_ore
-	name = "Silver Ore"
+	name = "银矿石"
 	valid_input = /obj/item/rogueore/silver
 	valid_outputs = list(/obj/item/alch/silverdust = 1)
 	bonus_chance_outputs = list(/obj/item/alch/silverdust = 33)
 
 /datum/alch_grind_recipe/iron_ore
-	name = "Iron Ore"
+	name = "铁矿石"
 	valid_input = /obj/item/rogueore/iron
 	valid_outputs = list(/obj/item/alch/irondust = 1)
 	bonus_chance_outputs = list(/obj/item/alch/irondust = 33, /obj/item/alch/runedust = 25)
 
 /datum/alch_grind_recipe/coal_ore
-	name = "Coal Ore"
+	name = "煤矿石"
 	valid_input = /obj/item/rogueore/coal
 	valid_outputs = list(/obj/item/alch/coaldust = 1)
 	bonus_chance_outputs = list(/obj/item/alch/coaldust = 33, /obj/item/alch/firedust = 25)
 
 /datum/alch_grind_recipe/gold_bar
-	name = "Gold Bar"
+	name = "金锭"
 	valid_input = /obj/item/ingot/gold
 	valid_outputs = list(/obj/item/alch/golddust = 1)
 	bonus_chance_outputs = list(/obj/item/alch/golddust = 33, /obj/item/alch/firedust = 25)
 
 /datum/alch_grind_recipe/silver_bar
-	name = "Silver Bar"
+	name = "银锭"
 	valid_input = /obj/item/ingot/silver
 	valid_outputs = list(/obj/item/alch/silverdust = 1)
 	bonus_chance_outputs = list(/obj/item/alch/silverdust = 33, /obj/item/alch/firedust = 25)
 
 /datum/alch_grind_recipe/blessed_silver_bar
-	name = "Blessed Silver Bar"
+	name = "祝圣银锭"
 	valid_input = /obj/item/ingot/silverblessed
 	valid_outputs = list(/obj/item/alch/silverdust = 1)
 	bonus_chance_outputs = list(/obj/item/alch/silverdust = 33, /obj/item/alch/firedust = 25)
 
 /datum/alch_grind_recipe/iron_bar
-	name = "Iron Bar"
+	name = "铁锭"
 	valid_input = /obj/item/ingot/iron
 	valid_outputs = list(/obj/item/alch/irondust = 1)
 	bonus_chance_outputs = list(/obj/item/alch/irondust = 33, /obj/item/alch/runedust = 25, /obj/item/alch/firedust = 25)
 
 /datum/alch_grind_recipe/puresalt
-	name = "Pure Salt"
+	name = "纯盐"
 	valid_input = /obj/item/reagent_containers/powder/salt
 	valid_outputs = list(/obj/item/alch/puresalt = 1)
 
 /datum/alch_grind_recipe/berrypowder
-	name = "Berry Powder"
+	name = "浆果粉"
 	valid_input = /obj/item/reagent_containers/food/snacks/grown/berries/rogue
 	valid_outputs = list(/obj/item/alch/berrypowder = 1)
 	bonus_chance_outputs = list(/obj/item/alch/waterdust = 25)
 
 /datum/alch_grind_recipe/manabloompowder
-	name = "Manabloom Powder"
+	name = "法力花粉"
 	valid_input = /obj/item/reagent_containers/food/snacks/grown/manabloom
 	valid_outputs = list(/obj/item/alch/manabloompowder = 1)
 	bonus_chance_outputs = list(/obj/item/alch/manabloompowder = 25)
 
 /datum/alch_grind_recipe/infernaldust
-	name = "Infernal Dust"
+	name = "地狱尘"
 	valid_input = /obj/item/magic/infernal/fang
 	valid_outputs = list(/obj/item/alch/infernaldust = 1)
 	bonus_chance_outputs = list(/obj/item/alch/firedust = 25)
@@ -213,37 +213,37 @@
 // Start of gem dust section - I've included gold dust as an additional product because of lesser alchemy, grinding up a gem should give you a bit extra (I mean come on it's a gem)
 
 /datum/alch_grind_recipe/mineraldustyellow  // costs two gold to make
-	name = "Mineral Dust (Toper)"
+	name = "矿物粉（Toper）"
 	valid_input = /obj/item/roguegem/yellow
 	valid_outputs = list(/obj/item/alch/mineraldust = 1, /obj/item/alch/golddust = 1)
 	bonus_chance_outputs = list(/obj/item/alch/golddust = 66)
 
 /datum/alch_grind_recipe/mineraldustgreen  // costs 4 gold to make
-	name = "Mineral Dust (Gemerald)"
+	name = "矿物粉（Gemerald）"
 	valid_input = /obj/item/roguegem/green
 	valid_outputs = list(/obj/item/alch/mineraldust = 1, /obj/item/alch/earthdust = 1, /obj/item/alch/golddust = 2)
 	bonus_chance_outputs = list(/obj/item/alch/earthdust = 66)
 
 /datum/alch_grind_recipe/mineraldustviolet // costs 6 gold to make
-	name = "Mineral Dust (Saffira)"
+	name = "矿物粉（Saffira）"
 	valid_input = /obj/item/roguegem/violet
 	valid_outputs = list(/obj/item/alch/mineraldust = 1, /obj/item/alch/magicdust = 1, /obj/item/alch/golddust = 3)
 	bonus_chance_outputs = list(/obj/item/alch/magicdust = 66)
 
 /datum/alch_grind_recipe/mineraldustblue // costs 8 gold to make
-	name = "Mineral Dust (Blortz)"
+	name = "矿物粉（Blortz）"
 	valid_input = /obj/item/roguegem/blue
 	valid_outputs = list(/obj/item/alch/mineraldust = 1, /obj/item/alch/waterdust = 2, /obj/item/alch/golddust = 4)
 	bonus_chance_outputs = list(/obj/item/alch/waterdust = 66)
 
 /datum/alch_grind_recipe/mineraldustdiamond // costs a whopping 18 gold to make, why are you doing this
-	name = "Mineral Dust (Diamond)"
+	name = "矿物粉（钻石）"
 	valid_input = /obj/item/roguegem/diamond
 	valid_outputs = list(/obj/item/alch/mineraldust = 1, /obj/item/alch/golddust = 6)
 	bonus_chance_outputs = list(/obj/item/alch/golddust = 66)
 
 /datum/alch_grind_recipe/mineraldustriddle //why are you doing this...
-	name = "Mineral Dust (Riddle of Steel)"
+	name = "矿物粉（钢铁之谜）"
 	valid_input = /obj/item/riddleofsteel
 	valid_outputs = list(/obj/item/alch/mineraldust = 2, /obj/item/alch/airdust = 1, /obj/item/alch/irondust = 1, /obj/item/alch/firedust = 1, /obj/item/alch/magicdust = 1, /obj/item/alch/silverdust = 1, /obj/item/alch/coaldust = 1, /obj/item/alch/runedust = 1, /obj/item/alch/waterdust = 1)  // if you're crazy enough to grind a riddle you should get at LEAST one of every dust.
 	bonus_chance_outputs = list(/obj/item/alch/mineraldust = 25, /obj/item/alch/airdust = 25, /obj/item/alch/irondust = 25, /obj/item/alch/firedust = 25, /obj/item/alch/magicdust = 25, /obj/item/alch/silverdust = 25, /obj/item/alch/coaldust = 25, /obj/item/alch/runedust = 25, /obj/item/alch/waterdust = 25)
@@ -253,76 +253,76 @@
 
 //Herb -> Herbseed
 /datum/alch_grind_recipe/atropa_seed
-	name = "Herb Seed (Atropa)"
+	name = "草药种子（Atropa）"
 	valid_input = /obj/item/alch/atropa
 	valid_outputs = list(/obj/item/herbseed/atropa = 1)
 
 /datum/alch_grind_recipe/matricaria_seed
-	name = "Herb Seed (Matricaria)"
+	name = "草药种子（Matricaria）"
 	valid_input = /obj/item/alch/matricaria
 	valid_outputs = list(/obj/item/herbseed/matricaria = 1)
 
 /datum/alch_grind_recipe/symphitum_seed
-	name = "Herb Seed (Symphitum)"
+	name = "草药种子（Symphitum）"
 	valid_input = /obj/item/alch/symphitum
 	valid_outputs = list(/obj/item/herbseed/symphitum = 1)
 
 /datum/alch_grind_recipe/taraxacum_seed
-	name = "Herb Seed (Taraxacum)"
+	name = "草药种子（Taraxacum）"
 	valid_input = /obj/item/alch/taraxacum
 	valid_outputs = list(/obj/item/herbseed/taraxacum = 1)
 
 /datum/alch_grind_recipe/euphrasia_seed
-	name = "Herb Seed (Euphrasia)"
+	name = "草药种子（Euphrasia）"
 	valid_input = /obj/item/alch/euphrasia
 	valid_outputs = list(/obj/item/herbseed/euphrasia = 1)
 
 /datum/alch_grind_recipe/paris_seed
-	name = "Herb Seed (Paris)"
+	name = "草药种子（Paris）"
 	valid_input = /obj/item/alch/paris
 	valid_outputs = list(/obj/item/herbseed/paris = 1)
 
 /datum/alch_grind_recipe/calendula_seed
-	name = "Herb Seed (Calendula)"
+	name = "草药种子（Calendula）"
 	valid_input = /obj/item/alch/calendula
 	valid_outputs = list(/obj/item/herbseed/calendula = 1)
 
 /datum/alch_grind_recipe/mentha_seed
-	name = "Herb Seed (Mentha)"
+	name = "草药种子（Mentha）"
 	valid_input = /obj/item/alch/mentha
 	valid_outputs = list(/obj/item/herbseed/mentha = 1)
 
 /datum/alch_grind_recipe/urtica_seed
-	name = "Herb Seed (Urtica)"
+	name = "草药种子（Urtica）"
 	valid_input = /obj/item/alch/urtica
 	valid_outputs = list(/obj/item/herbseed/urtica = 1)
 
 /datum/alch_grind_recipe/salvia_seed
-	name = "Herb Seed (Salvia)"
+	name = "草药种子（Salvia）"
 	valid_input = /obj/item/alch/salvia
 	valid_outputs = list(/obj/item/herbseed/salvia = 1)
 
 /datum/alch_grind_recipe/hypericum_seed
-	name = "Herb Seed (Hypericum)"
+	name = "草药种子（Hypericum）"
 	valid_input = /obj/item/alch/hypericum
 	valid_outputs = list(/obj/item/herbseed/hypericum = 1)
 
 /datum/alch_grind_recipe/benedictus_seed
-	name = "Herb Seed (Benedictus)"
+	name = "草药种子（Benedictus）"
 	valid_input = /obj/item/alch/benedictus
 	valid_outputs = list(/obj/item/herbseed/benedictus = 1)
 
 /datum/alch_grind_recipe/valeriana_seed
-	name = "Herb Seed (Valeriana)"
+	name = "草药种子（Valeriana）"
 	valid_input = /obj/item/alch/valeriana
 	valid_outputs = list(/obj/item/herbseed/valeriana = 1)
 
 /datum/alch_grind_recipe/artemisia_seed
-	name = "Herb Seed (Artemisia)"
+	name = "草药种子（Artemisia）"
 	valid_input = /obj/item/alch/artemisia
 	valid_outputs = list(/obj/item/herbseed/artemisia = 1)
 
 /datum/alch_grind_recipe/rosa_seed
-	name = "Herb Seed (Rosa)"
+	name = "草药种子（Rosa）"
 	valid_input = /obj/item/alch/rosa
 	valid_outputs = list(/obj/item/herbseed/rosa = 1)
