diff --git a/code/modules/roguetown/roguecrafting/alchemy/alch_cauldron_recipes.dm b/code/modules/roguetown/roguecrafting/alchemy/alch_cauldron_recipes.dm
--- a/code/modules/roguetown/roguecrafting/alchemy/alch_cauldron_recipes.dm
+++ b/code/modules/roguetown/roguecrafting/alchemy/alch_cauldron_recipes.dm
@@ -1,128 +1,128 @@
 //vial = 30, bottle = 50
 /datum/alch_cauldron_recipe/antidote
-	name = "Antidote"
+	name = "解毒剂"
 	smells_like = "wet moss"
 	output_reagents = list(/datum/reagent/medicine/antidote = 100)
 
 /datum/alch_cauldron_recipe/strong_antidote
-	name = "Antidote (Strong)"
+	name = "强效解毒剂"
 	smells_like = "purity"
 	output_reagents = list(/datum/reagent/medicine/strong_antidote = 100)
 
 /datum/alch_cauldron_recipe/berrypoison
-	name = "Poison (Berry)"
+	name = "莓果毒药"
 	smells_like = "death"
 	skill_required = SKILL_LEVEL_JOURNEYMAN // Basic poison should be harder to handle
 	output_reagents = list(/datum/reagent/berrypoison = 100)
 
 /datum/alch_cauldron_recipe/doompoison
-	name = "Poison (Doom)"
+	name = "毁灭毒药"
 	smells_like = "doom"
 	skill_required = SKILL_LEVEL_EXPERT // Strong poison should be more difficult to make
 	output_reagents = list(/datum/reagent/strongpoison = 100)
 
 /datum/alch_cauldron_recipe/stam_poison
-	name = "Stamina Poison"
+	name = "耐力毒药"
 	smells_like = "a slow breeze"
 	skill_required = SKILL_LEVEL_JOURNEYMAN // Basic poison should be harder to handle
 	output_reagents = list(/datum/reagent/stampoison = 100)
 
 /datum/alch_cauldron_recipe/big_stam_poison
-	name = "Stamina Poison (Strong)"
+	name = "强效耐力毒药"
 	smells_like = "stagnant air"
 	skill_required = SKILL_LEVEL_EXPERT // Strong poison should be more difficult to make
 	output_reagents = list(/datum/reagent/strongstampoison = 100)
 
 //Healing potions
 /datum/alch_cauldron_recipe/health_potion
-	name = "Elixir of Health"
+	name = "生命灵药"
 	smells_like = "sweet berries"
 	output_reagents = list(/datum/reagent/medicine/healthpot = 100)
 
 /datum/alch_cauldron_recipe/big_health_potion
-	name = "Elixir of Health (Strong)"
+	name = "强效生命灵药"
 	smells_like = "berry pie"
 	skill_required = SKILL_LEVEL_EXPERT // If it has "Strong", lock it roundstart for Apothecary or above
 	output_reagents = list(/datum/reagent/medicine/stronghealth = 50)//it has an extremely easy recipe compared to regular red
 
 /datum/alch_cauldron_recipe/mana_potion
-	name = "Elixir of Mana"
+	name = "法力灵药"
 	smells_like = "power"
 	output_reagents = list(/datum/reagent/medicine/manapot = 100)//recipe is harder to make than regular blue
 
 /datum/alch_cauldron_recipe/big_mana_potion
-	name = "Elixir of Mana (Strong)"
+	name = "强效法力灵药"
 	smells_like = "fear"
 	skill_required = SKILL_LEVEL_EXPERT
 	output_reagents = list(/datum/reagent/medicine/strongmana = 100)
 
 /datum/alch_cauldron_recipe/stamina_potion
-	name = "Elixir of Stamina"
+	name = "耐力灵药"
 	smells_like = "fresh air"
 	output_reagents = list(/datum/reagent/medicine/stampot = 100)
 
 /datum/alch_cauldron_recipe/big_stamina_potion
-	name = "Elixir of Stamina (Strong)"
+	name = "强效耐力灵药"
 	smells_like = "clean winds"
 	skill_required = SKILL_LEVEL_JOURNEYMAN
 	output_reagents = list(/datum/reagent/medicine/strongstam = 100)
 
 //S.P.E.C.I.A.L. potions - Expert or above (roundstart Witch etc.)
 /datum/alch_cauldron_recipe/temp_potion
-	name = "Potion of Equilibrium"
+	name = "平衡药剂"
 	smells_like = "spring time"
 	skill_required = SKILL_LEVEL_EXPERT
 	output_reagents = list(/datum/reagent/buff/temperature_normalize = 30)
 
 /datum/alch_cauldron_recipe/str_potion
-	name = "Potion of Mountain Muscles"
+	name = "山岳肌力药剂"
 	smells_like = "petrichor"
 	skill_required = SKILL_LEVEL_EXPERT
 	output_reagents = list(/datum/reagent/buff/strength = 30)
 
 /datum/alch_cauldron_recipe/per_potion
-	name = "Potion of Keen Eye"
+	name = "锐目药剂"
 	smells_like = "fire"
 	skill_required = SKILL_LEVEL_EXPERT
 	output_reagents = list(/datum/reagent/buff/perception = 30)
 
 /datum/alch_cauldron_recipe/end_potion
-	name = "Potion of Enduring Fortitude"
+	name = "坚忍药剂"
 	smells_like = "mountain air"
 	skill_required = SKILL_LEVEL_EXPERT
 	output_reagents = list(/datum/reagent/buff/endurance = 30)
 
 /datum/alch_cauldron_recipe/con_potion
-	name = "Potion of Stone Flesh"
+	name = "石肤药剂"
 	smells_like = "earth"
 	skill_required = SKILL_LEVEL_EXPERT
 	output_reagents = list(/datum/reagent/buff/constitution = 30)
 
 /datum/alch_cauldron_recipe/int_potion
-	name = "Potion of Keen Mind"
+	name = "锐思药剂"
 	smells_like = "water"
 	skill_required = SKILL_LEVEL_EXPERT
 	output_reagents = list(/datum/reagent/buff/intelligence = 30)
 
 /datum/alch_cauldron_recipe/spd_potion
-	name = "Potion of Fleet Foot"
+	name = "疾足药剂"
 	smells_like = "clean air"
 	skill_required = SKILL_LEVEL_EXPERT
 	output_reagents = list(/datum/reagent/buff/speed = 30)
 
 /datum/alch_cauldron_recipe/lck_potion
-	name = "Potion of Seven Clovers"
+	name = "七叶草药剂"
 	smells_like = "calming"
 	skill_required = SKILL_LEVEL_EXPERT
 	output_reagents = list(/datum/reagent/buff/fortune = 30)
 
 /datum/alch_cauldron_recipe/aphrodisiac
-	name = "Aphrodisiac Wine"
+	name = "催情葡萄酒"
 	smells_like = "ardent sweetness"
 	output_reagents = list(/datum/reagent/consumable/ethanol/beer/emberwine = 60)
 
 /datum/alch_cauldron_recipe/fire_potion
-	name = "Potion of Fire Warding"
+	name = "抗火药剂"
 	smells_like = "authority"
 	skill_required = SKILL_LEVEL_MASTER
 	output_reagents =list(/datum/reagent/fire_resist = 30)
