From 02b6c1821fdd47a613c1afc7c67a99c39ec18d0e Mon Sep 17 00:00:00 2001
From: Translation Patch Toolkit <translation@local>
Date: Sat, 30 May 2026 13:00:00 +0800
Subject: [PATCH] 翻译 code/modules/mob/living/brain/emote.dm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
diff --git a/code/modules/mob/living/brain/emote.dm b/code/modules/mob/living/brain/emote.dm
--- a/code/modules/mob/living/brain/emote.dm
+++ b/code/modules/mob/living/brain/emote.dm
@@ -10,25 +10,25 @@
 
 /datum/emote/brain/alarm
 	key = "alarm"
-	message = "sounds an alarm."
+	message = "发出警报声。"
 	emote_type = EMOTE_AUDIBLE
 
 /datum/emote/brain/alert
 	key = "alert"
-	message = "lets out a distressed noise."
+	message = "发出惊慌失措的噪音。"
 	emote_type = EMOTE_AUDIBLE
 
 /datum/emote/brain/flash
 	key = "flash"
-	message = "blinks their lights."
+	message = "闪烁起自己的灯光。"
 
 /datum/emote/brain/notice
 	key = "notice"
-	message = "plays a loud tone."
+	message = "发出一声高亢音调。"
 	emote_type = EMOTE_AUDIBLE
 
 /datum/emote/brain/whistle
 	key = "whistle"
 	key_third_person = "whistles"
-	message = "whistles."
+	message = "吹起口哨。"
 	emote_type = EMOTE_AUDIBLE
-- 
Translation Patch Toolkit
