diff --git a/code/game/objects/structures/crates_lockers/crates.dm b/code/game/objects/structures/crates_lockers/crates.dm
--- a/code/game/objects/structures/crates_lockers/crates.dm
+++ b/code/game/objects/structures/crates_lockers/crates.dm
@@ -1,5 +1,5 @@
 /obj/structure/closet/crate
-	name = "crate"
+	name = "木箱"
 	desc = ""
 	icon_state = "crate"
 	req_access = null
@@ -50,7 +50,7 @@
 	. = ..()
 
 /obj/structure/closet/crate/coffin
-	name = "casket"
+	name = "棺匣"
 	desc = "Death basket."
 	icon_state = "casket"
 	resistance_flags = FLAMMABLE
