diff --git a/code/modules/roguetown/roguemachine/questing/types/quest_retrieval.dm b/code/modules/roguetown/roguemachine/questing/types/quest_retrieval.dm
--- a/code/modules/roguetown/roguemachine/questing/types/quest_retrieval.dm
+++ b/code/modules/roguetown/roguemachine/questing/types/quest_retrieval.dm
@@ -9,10 +9,10 @@
 /datum/quest/retrieval/get_title()
 	if(title)
 		return title
-	return "Retrieve [pick("an ancient", "a rare", "a stolen", "a magical")] [pick("artifact", "relic", "doohickey", "treasure")]"
+	return "寻回[pick("一件古老的", "一件稀有的", "一件失窃的", "一件魔法的")][pick("器物", "遗物", "玩意", "宝藏")]"
 
 /datum/quest/retrieval/get_objective_text()
-	return "Retrieve [progress_required] [initial(target_item_type.name)]."
+	return "寻回 [progress_required] 个 [initial(target_item_type.name)]。"
 
 
 /datum/quest/retrieval/get_additional_reward(target_turf)
