diff --git a/code/modules/jobs/job_types/roguetown/peasants/soilson.dm b/code/modules/jobs/job_types/roguetown/peasants/soilson.dm
--- a/code/modules/jobs/job_types/roguetown/peasants/soilson.dm
+++ b/code/modules/jobs/job_types/roguetown/peasants/soilson.dm
@@ -1,5 +1,6 @@
 /datum/job/roguetown/farmer
 	title = "Soilson"
+	display_title = "农人"
 	flag = FARMER
 	department_flag = PEASANTS
 	faction = "Station"
@@ -11,10 +12,10 @@
 	allowed_races = ACCEPTED_RACES
 	cmode_music = 'sound/music/cmode/towner/combat_towner2.ogg'
 
-	tutorial = "It is a simple life you live, your basic understanding of life is something many would be envious of if they knew just how perfect it was. You know a good day's work, the sweat on your brow is yours: Famines and plague may take their toll, but you know how to celebrate life well. Till the soil and produce fresh food for those around you, and maybe you'll be more than an unsung hero someday."
+	tutorial = "你过着一种朴素的生活，而你对日子的那点朴实理解，若让旁人知道其中有多圆满，怕是许多人都会心生羡慕。你知道怎样才算一整天扎实的劳作，也明白额上的汗水都是替自己流的。饥荒与瘟疫也许会降临，但你依旧懂得如何好好庆贺活着这件事。翻耕土地，为周围的人种出新鲜粮食，说不定有朝一日，你就不只是个无人称颂的英雄。"
 
 
 	f_title = "Soilbride"
 	outfit = /datum/outfit/job/roguetown/farmer
 	display_order = 24
 	give_bank_account = 17
@@ -31,8 +32,8 @@
 	spells = list(/obj/effect/proc_holder/spell/invoked/takeapprentice)
 
 /datum/advclass/soilson
-	name = "Soilson"
-	tutorial = "It is a simple life you live, your basic understanding of life is something many would be envious of if they knew just how perfect it was. You know a good day's work, the sweat on your brow is yours: Famines and plague may take their toll, but you know how to celebrate life well. Till the soil and produce fresh food for those around you, and maybe you'll be more than an unsung hero someday."
+	name = "农人"
+	tutorial = "你过着一种朴素的生活，而你对日子的那点朴实理解，若让旁人知道其中有多圆满，怕是许多人都会心生羡慕。你知道怎样才算一整天扎实的劳作，也明白额上的汗水都是替自己流的。饥荒与瘟疫也许会降临，但你依旧懂得如何好好庆贺活着这件事。翻耕土地，为周围的人种出新鲜粮食，说不定有朝一日，你就不只是个无人称颂的英雄。"
 	outfit = /datum/outfit/job/roguetown/farmer/basic
 	category_tags = list(CTAG_SOILBRIDE)
 	subclass_stats = list(
